<!doctype html>
<meta charset=utf-8>
<meta name=viewport content=width=device-width,initial-scale=1.0>
<style>
body{background:#2f2f2f;color:white}
body,td,th{font:14.5px/1.33 Arial,Helvetica,'Liberation Sans',sans-serif}
p{max-width:530px;margin:1em auto}
dl,pre{margin:1em auto;width:fit-content;max-width:min(100%, 1100px);box-sizing:border-box}
hr{border:none;border-top: 2px solid #bbb;margin:1em auto;max-width:1100px}
ul,ol{width:min(100%, 530px);margin:1em auto;box-sizing:border-box}
ul ul,ul ol,ol ol,ol ul{margin-top:0;margin-bottom:0}
div:has(img){width:fit-content;margin:0 auto}
h1,h2,h3,h2 a,h3 a,h2 a:visited,h3 a:visited{color:white;font-weight:normal}
h1{font-size:2.1em;margin-top:.7em;text-align:center}
h2{font-size:1.75em;text-align:center}
h3{font-size:1.35em;text-align:center}
blockquote h2,blockquote h3{margin:.5em 0}
blockquote ul,blockquote ol{margin:.4em 0}
a,a .quote{color:hsl(220,100%,75%)}
a:visited,a .quote{color:hsl(270,100%,75%)}
pre,code{white-space:pre-wrap;word-wrap:break-word;font:13.7px Menlo,'DejaVu Sans Mono','Liberation Mono',Consolas,monospace;background:#4a4a4a;color:white}
pre{padding:4px}
code{padding:1px}
table{border-collapse:collapse;margin:0 auto}
td,th{border:1px solid #555;padding:1px 3px}
th{background:#444;text-align:left;font-weight:bold}
img{max-width:100%;margin:0;vertical-align:middle}
dt{font-size:1.3em;margin:.4em 0}
body>blockquote{margin:1em auto;padding-left:45px;width:485px}
blockquote blockquote{width:calc(100% - 45px);margin-left:45px;box-sizing:border-box}
blockquote>:first-child{margin-top:0}
blockquote>:last-child{margin-bottom:0}
blockquote,.quote,blockquote,blockquote{color:hsl(100,100%,88%)}
blockquote .quote,blockquote blockquote,.quote .quote{color:hsl(100,50%,67%)}
blockquote blockquote blockquote,blockquote blockquote .quote,blockquote .quote .quote,.quote .quote .quote{color:hsl(100,35%,45%)}
blockquote h3{color:inherit}
blockquote h2,blockquote h3{text-align:left}
.spoiler{max-width:1100px;margin:0 auto;text-align:center}
.spoiler button{width:90px;border:0;font-family:inherit;font-size:inherit;color:inherit;background:#4a4a4a;padding:4px 0px;outline:none;cursor:pointer;margin:0 0 -.3em 0}
.spoiler pre{display:none;margin-top:0;text-align:left}
.red,.red .quote{color:hsl(0,100%,77%)!important}
.comment{color:hsl(0,100%,80%)!important}
.toc{width:calc(50% + 265px);margin:1em 0 1em max(0px, 50% - 265px)}
.toc ul{width:min(100%, 50% + 265px - 40px);margin:0}
.images{display:inline-block;width:content-width;margin:0 auto}
@media print{
  body{background:white;color:black;margin:0px;zoom:1;print-color-adjust:exact;-webkit-print-color-adjust:exact}
  a{color:#0000EE}a:visited{color:#551A8B}
  h1,h2,h2 a,h3 a,h2 a:visited,h3 a:visited{color:black}
  pre,code{background:#ccc;color:black;font-size:13px}
  th{background:#ddd}
  blockquote,.quote{color:hsl(30,65%,30%)}
  blockquote .quote,blockquote blockquote,.quote .quote{color:hsl(30,45%,50%)}
  blockquote blockquote blockquote,blockquote .quote .quote,.quote .quote .quote{color:hsl(30,70%,60%)}
  .spoiler{display:none}
  .red,.red .quote{color:hsl(0,50%,60%)!important}
}
</style>
<title>Richard Fleming is a fraud</title>
<h1>Richard Fleming is a fraud - <a href=index.html>sars2.net</a></h1>
<h2 id=Contents><a href=#Contents>Contents</a></h2>
<ul class=toc>
<li><a href="#Flemings_history_of_lies_and_fraud">Fleming&#39;s history of lies and fraud</a></li>
<ul class=toc>
<li><a href="#Research_by_Cheshire_and_LogarithmicDis">Research by Cheshire and LogarithmicDis</a></li>
<li><a href="#Fake_PhD_diploma_in_physics">Fake PhD diploma in physics</a></li>
<li><a href="#Twitter_accounts_ExperiencedStud_and_ImBodu">Twitter accounts ExperiencedStud and ImBodu</a></li>
<li><a href="#Incorrect_chi_squared_comparison_of_Fleming_FMTVDM_method_and_mammography">Incorrect chi-squared comparison of Fleming FMTVDM method and mammography</a></li>
<li><a href="#Claim_that_his_Twitter_account_was_hacked_to_add_a_photo_of_David_Icke">Claim that his Twitter account was hacked to add a photo of David Icke</a></li>
</ul>
<li><a href="#Fake_clinical_trial_comparing_COVID_treatments_in_2020">Fake clinical trial comparing COVID treatments in 2020</a></li>
<ul class=toc>
<li><a href="#Comments_at_PubPeer">Comments at PubPeer</a></li>
<li><a href="#Zinc_treatment_in_outpatients">Zinc treatment in outpatients</a></li>
<li><a href="#Duration_of_hospitalization_and_99_83_effectiveness">Duration of hospitalization and 99.83% effectiveness</a></li>
<li><a href="#All_treatment_arms_had_the_same_baseline_levels_of_ferritin_and_IL_6">All treatment arms had the same baseline levels of ferritin and IL-6</a></li>
<li><a href="#Comments_at_Retraction_Watch">Comments at Retraction Watch</a></li>
<li><a href="#The_paper_about_the_trial_was_published_in_a_fake_journal">The paper about the trial was published in a fake journal</a></li>
</ul>
<li><a href="#Crimes_Against_Humanity_affidavit">Crimes Against Humanity affidavit</a></li>
<ul class=toc>
<li><a href="#Chi_squared_calculation_of_COVID_cases_in_vaccine_trials">Chi-squared calculation of COVID cases in vaccine trials</a></li>
<li><a href="#Claim_that_no_other_coronavirus_has_a_furin_cleavage_site">Claim that no other coronavirus has a furin cleavage site</a></li>
<li><a href="#Claim_that_the_likelihood_of_an_FCS_occurring_spontaneously_is_3_21e_11">Claim that the likelihood of an FCS occurring spontaneously is 3.21e-11</a></li>
<li><a href="#Alex_Washburnes_endonuclease_fingerprint_paper">Alex Washburne&#39;s endonuclease fingerprint paper</a></li>
</ul>
<li><a href="#Crimes_Against_Humanity_Tour">Crimes Against Humanity Tour</a></li>
<ul class=toc>
<li><a href="#Matches_to_Corman_Drostens_primers_in_SHC014_MA15_and_Rs3367">Matches to Corman-Drosten&#39;s primers in SHC014, MA15, and Rs3367</a></li>
<li><a href="#Chromosomes_of_viruses">Chromosomes of viruses</a></li>
<li><a href="#Description_of_genetic_sequencing_and_PCR_testing">Description of genetic sequencing and PCR testing</a></li>
</ul>
<li><a href="#Book_Is_COVID_19_a_Bioweapon">Book Is COVID-19 a Bioweapon</a></li>
<ul class=toc>
<li><a href="#Chimeric_sequence_from_2006_with_a_segment_labeled_SARS_CoV2">Chimeric sequence from 2006 with a segment labeled SARS-CoV2</a></li>
<li><a href="#Was_the_WIV_database_wiped_in_the_summer_of_2019">Was the WIV database wiped in the summer of 2019?</a></li>
<li><a href="#Claim_that_Mullis_did_not_use_more_than_15_to_20_cycles_with_PCR">Claim that Mullis did not use more than 15 to 20 cycles with PCR</a></li>
<li><a href="#Claim_that_SARS_CoV_is_unable_to_bind_human_ACE2">Claim that SARS-CoV is unable to bind human ACE2</a></li>
<li><a href="#Claim_that_the_S1_subunit_is_more_stable_than_the_S2_subunit">Claim that the S1 subunit is more stable than the S2 subunit</a></li>
<li><a href="#Critique_of_Pradhan_by_Zhang_et_al">Critique of Pradhan by Zhang et al.</a></li>
<li><a href="#Furin_cleavage_motif_in_HIV_is_not_PRRA">Furin cleavage motif in HIV is not PRRA</a></li>
<li><a href="#Perez_and_Montagniers_paper_about_BLAST_matches_to_HIV">Perez and Montagnier&#39;s paper about BLAST matches to HIV</a></li>
</ul>
<li><a href="#Collaboration_with_Slovakian_commission">Collaboration with Slovakian commission</a></li>
<ul class=toc>
<li><a href="#So_a_Pekov_and_ALLATRA">Soňa Peková and ALLATRA</a></li>
<li><a href="#Statement_by_Slovak_Academy_of_Sciences">Statement by Slovak Academy of Sciences</a></li>
</ul>
</ul>


<h2>
<a href=#Flemings_history_of_lies_and_fraud id=Flemings_history_of_lies_and_fraud>Fleming's
history of lies and fraud</a>
</h2>
<h3>
<a href=#Research_by_Cheshire_and_LogarithmicDis id=Research_by_Cheshire_and_LogarithmicDis>Research
by Cheshire and LogarithmicDis</a>
</h3>
<p>See here:
<a href=https://forbetterscience.com/2021/09/27/cheshire-vs-dr-who/>https://forbetterscience.com/2021/09/27/cheshire-vs-dr-who/</a>,
<a href=https://x.com/search?q=from%3Athatsregrettab1+%28fleming+OR+%22dr.+who%22+OR+Doctor_I_am_The%29&f=live>https://x.com/search?q=from%3Athatsregrettab1+%28fleming+OR+%22dr.+who%22+OR+Doctor_I_am_The%29&amp;f=live</a>,
<a href=https://x.com/search?q=from%3Alogarithmicdis+%28fleming+OR+Doctor_I_am_The%29&f=live>https://x.com/search?q=from%3Alogarithmicdis+%28fleming+OR+Doctor_I_am_The%29&amp;f=live</a>,
<a href=https://x.com/LogarithmicDis/status/1883240497206169971.><a
href="https://x.com/LogarithmicDis/status/1883240497206169971"
class="uri">https://x.com/LogarithmicDis/status/1883240497206169971</a>.</a></p>
<h3>
<a href=#Fake_PhD_diploma_in_physics id=Fake_PhD_diploma_in_physics>Fake
PhD diploma in physics</a>
</h3>
<p>Fleming wrote: <span class="quote">"Yes, <span class="citation"
data-cites="kalashnikity">@kalashnikity</span> I received my PhD two
days after my HS diploma. That was the requirement from the work begun
in 1968 when our group, following the MK Ultra group, was initiated. I
owe much to many including respect and confidentiality of our
work."</span>
[<a href=https://x.com/Doctor_I_am_The/status/1876680552952004630>https://x.com/Doctor_I_am_The/status/1876680552952004630</a>,
<a href=https://archive.is/IZdz7>https://archive.is/IZdz7</a>]</p>
<div>
<img src=i/fleming-diploma-1.png width=500>
</div>
<p>When someone asked Fleming to post his PhD thesis to show it wasn't
fake, Fleming replied simply <span class="quote">"Q-clearance"</span>
(which I presume meant he couldn't publish his thesis because it
contained classified information).
[<a href=https://x.com/Doctor_I_am_The/status/1876722383400083499>https://x.com/Doctor_I_am_The/status/1876722383400083499</a>]
Fleming also told me: <span class="quote">"As far as my research for the
thesis, this was classified as Q-clearance in 1974. I have reached out
to Trump and others to work with him using this research."</span>
[<a href=https://x.com/Doctor_I_am_The/status/1882153774636028052>https://x.com/Doctor_I_am_The/status/1882153774636028052</a>]
And later Fleming said: <span class="quote">"I myself agreed to make my
doctorate research Q-clearance because I could not envision humanity
using it for the peaceful purposes I had intended it to be used for. I
could only envision it being used for destructive purposes."</span>
[<a href=https://x.com/Doctor_I_am_The/status/1893677446081269778>https://x.com/Doctor_I_am_The/status/1893677446081269778</a>]</p>
<p>The Twitter user LogarithmicDis found many problems with Fleming's
diploma:
[<a href=https://x.com/LogarithmicDis/status/1882133468500660348>https://x.com/LogarithmicDis/status/1882133468500660348</a>]</p>
<blockquote>
<p>On his website he has posted his <span class="quote">"highly
unusual"</span> PhD diploma that he started on supposedly at age 12, if
you subtract his birth year (posted online by him) from a 1968
matriculating date = 12 years old.</p>
<p>But the diploma has MANY problems.</p>
<div>
<img src=i/fleming-dis-1.jpg width=600>
</div>
<p>First, the doctor is a convicted felon and was sentenced for his
criminal acts and debarred by the FDA.</p>
<p><a href=https://archives.fbi.gov/archives/omaha/press-releases/2009/om082009.htm>https://archives.fbi.gov/archives/omaha/press-releases/2009/om082009.htm</a></p>
<div>
<img src=i/fleming-dis-2.jpg width=400>
</div>
<p>So his honesty isn't exactly batting a thousand here. Upon closer
look at his CV posted on his website:</p>
<div>
<img src=i/fleming-dis-3.jpg width=500>
</div>
<p>And closer look at the unusual PhD diploma with contrast adjusted, we
see a copy and paste left hand signature rectangle and very unusual
seals.</p>
<p>But there are 12 total problems with this <span
class="quote">"diploma"</span> that I see:</p>
<div>
<img src=i/fleming-dis-4.jpg width=600>
</div>
<p>1. There is no such thing as <span class="quote">"The State
Universities of Iowa"</span>.</p>
<p>University of Iowa was in fact called <span class="quote">"State
University of Iowa"</span>.</p>
<p>Singular, not plural.</p>
<p><a href=https://iowaregents.edu/the-board/regents-history>https://iowaregents.edu/the-board/regents-history</a></p>
<div>
<img src=i/fleming-dis-5.jpg width=500>
</div>
<p>2. The copy &amp; paste left hand signatures (contrast adj).</p>
<p>3. The president of the Board of Regents was Mary Peterson in 1974.
So wrong name here.</p>
<p>4. The proper title is President, Board of Regents (not Board of
Education, a term that ceased in 1955).</p>
<p><a href=https://iowaregents.edu/the-board/regents-history>https://iowaregents.edu/the-board/regents-history</a></p>
<div>
<img src=i/fleming-dis-6.jpg width=300>
</div>
<p>5. The seals appear to not be original to the background</p>
<p>6. LT and RT seals do not fade w/the document (along w/the square
patch on left and e-signatures in right, addressed later).</p>
<p>7. The middle seal appears curved and uneven, like a poor patch
job.</p>
<p>8. Right seal is what?</p>
<div>
<img src=i/fleming-dis-7.jpg width=600>
</div>
<p>9. Two electronic signatures are exactly the same font and
uncharacteristic of 1974.</p>
<p>10. Weird fine print at bottom with misspelling.</p>
<p>11. Executive order reference to JFK is strange on a diploma. Here
are JFK's EOs:</p>
<p><a href=https://en.wikipedia.org/wiki/List_of_executive_actions_by_John_F%2e_Kennedy>https://en.wikipedia.org/wiki/List_of_executive_actions_by_John_F%2e_Kennedy</a></p>
<div>
<img src=i/fleming-dis-9.jpg width=300>
</div>
<p>12. UNI does not confer PhD degrees. It was formerly a teachers
college and confers only EdD degrees.</p>
<div>
<img src=i/fleming-dis-10.jpg width=500>
</div>
<p>Here's his CV from <span class="citation"
data-cites="Doctor_I_am_The">@Doctor_I_am_The</span> federal criminal
court case in 2009.</p>
<p>He failed to update it.</p>
<p>Certification Board in Nuclear Cardiology reveals no certifications
currently.</p>
<p><a href=https://apca.org/certifications-examinations/cbnc-and-cbcct/cbnc/>https://apca.org/certifications-examinations/cbnc-and-cbcct/cbnc/</a></p>
<p>ASNC says expired in 2002.</p>
<div>
<img src=i/fleming-dis-11.jpg width=400><img src=i/fleming-dis-12.jpg width=300>
</div>
<p>I verified the diploma was fake with the regent universities. I
actually called (like that was necessary? lol 😂)</p>
<p>[...]</p>
<p>And the ridiculous statement at the bottom of Fleming's confirmed
fake degree is a complete joke.</p>
<p>Here are JFK's EO's that day.</p>
<p><a href=https://federalregister.gov/presidential-documents/executive-orders/john-f-kennedy/1963>https://federalregister.gov/presidential-documents/executive-orders/john-f-kennedy/1963</a></p>
<div>
<img src=i/fleming-dis-13.jpg width=300>
</div>
</blockquote>
<p>The line at the bottom of the diploma says: <span class="quote">"This
Doctorate is issued for the recipient's work on plasma and positrons
pursuant to Executive Order APP#43 issued January 29, 1963 by John
Fitzgerald Kennedy, President of the Unisted [sic] States of
America."</span> But the list of executive orders issued by JFK is
public, and it doesn't include any order number <span
class="quote">"APP#43"</span>.</p>
<p>I tried to check if the diploma featured fonts which had not yet been
published in 1974, but I believe the two fonts I was able to identify
had both been published before 1974 (even though it's possible that
Fleming would have used newer near-identical variants of the old fonts I
identified, because there's newer copycat versions of both fonts, but I
didn't bother doing a detailed investigation of the fonts).</p>
<p>The signatures of Francis F Chen and CD Anderson seem to have been
written in some version of the Brush Script font which was designed in
1942:</p>
<div>
<img src=i/fleming-dis-brush.png width=400>
</div>
<p>At first I thought the blackletter font in the diploma was Monotype
Old English Text which was only published in 1990.
[<a href=https://www.myfonts.com/collections/old-english-text-font-monotype-imaging>https://www.myfonts.com/collections/old-english-text-font-monotype-imaging</a>]
However Monotype Old English Text was a clone of an older font called
Cloister Back that was designed around 1904, and based on the angle of
the upper left arm in the capital letter U, Fleming's font actually
seems closer to the old Cloister Black font:
[<a href=http://identifont.com/show?249>http://identifont.com/show?249</a>]</p>
<div>
<img src=i/fleming-dis-fonts.jpg width=600>
</div>
<p>When I linked to the thread by LogarithmicDis, Fleming told me: <span
class="quote">"This image of my doctoral diploma is clearly altered and
you are either guilty of altering it or guilty of presenting an altered
diploma. It's a new country. :-)"</span>.
[<a href=https://x.com/Doctor_I_am_The/status/1882159478986170631>https://x.com/Doctor_I_am_The/status/1882159478986170631</a>]
Fleming also told me: <span class="quote">"This doctorate was
manipulated by people on line. People who spend a lot of time
manipulating images including published research."</span>
[<a href=https://x.com/Doctor_I_am_The/status/1882153774636028052>https://x.com/Doctor_I_am_The/status/1882153774636028052</a>]
However I didn't find any way the images in the thread by LogarithmicDis
were manipulated apart from the adjustment of contrast which she
mentioned in her thread.</p>
<p>But perhaps Fleming was projecting his own sins onto his adversaries,
because at some point after LogarithmicDis posted her thread, Fleming
altered the image of his diploma on his website so that the background
of the signatures on the left and the two seals on the middle was
modified to match the background of the rest of the paper:
[<a href=https://www.flemingmethod.com/publications>https://www.flemingmethod.com/publications</a>]</p>
<div>
<img src=i/fleming-diploma-new.png width=700>
</div>
<p>The old version of the image where the backgrounds didn't match is
still available at the Wayback Machine:
[<a href=http://web.archive.org/web/20230206233241/https://www.flemingmethod.com/publications>http://web.archive.org/web/20230206233241/https://www.flemingmethod.com/publications</a>,
<a href=https://archive.is/Z1Ja4>https://archive.is/Z1Ja4</a>]</p>
<div>
<img src=i/fleming-diploma-old.png width=700>
</div>
<p>When I asked Fleming about his PhD thesis, he told me to read his
book <span class="quote">"Are We the Next Endangered Species"</span>.
[<a href=https://x.com/Doctor_I_am_The/status/1878458620402151869>https://x.com/Doctor_I_am_The/status/1878458620402151869</a>]
I believe he wanted me to read the appendix in his book called <span
class="quote">"Fleming Doctorate Research and Training"</span>. However
I didn't find the book on Libgen or Anna's Archive, and it costs 27 USD
on Amazon but I don't want to pay that much for a single book
chapter.</p>
<h3>
<a href=#Twitter_accounts_ExperiencedStud_and_ImBodu id=Twitter_accounts_ExperiencedStud_and_ImBodu>Twitter
accounts ExperiencedStud and ImBodu</a>
</h3>
<p>Fleming used to have a Twitter account called ExperiencedStud where
he advertised his work as an actor and he tried to get casted in adult
films. There's no snapshots of the account at the Wayback Machine, but
Cheshire managed to take these screenshots of the account:
[<a href=https://x.com/Thatsregrettab1/status/1559607005848883200/photo/1>https://x.com/Thatsregrettab1/status/1559607005848883200/photo/1</a>]</p>
<div>
<img src=i/fleming-stud-1.jpg width=1500>
</div>
<p>I don't know if Fleming actually managed to appear in any adult
movie, because one of his tweets from 2016 said: <span
class="quote">"I've been acting a couple years. Just started in porn a
few weeks ago after people told me for years I'm a natural. Looking to
be your talent"</span>.
[<a href=https://x.com/Thatsregrettab1/status/1386706893788258313>https://x.com/Thatsregrettab1/status/1386706893788258313</a>]</p>
<p>Fleming said that the account was hacked, and that when people said
he was an adult actor, it just meant that he was not a child actor:
[<a href=https://x.com/Doctor_I_am_The/status/1386712519436349440>https://x.com/Doctor_I_am_The/status/1386712519436349440</a>,
<a href=https://x.com/Doctor_I_am_The/status/1788273755619242277>https://x.com/Doctor_I_am_The/status/1788273755619242277</a>]</p>
<div>
<img src=i/fleming-stud-2.png width=600>
</div>
<p>Fleming also claimed that the account was called <span
class="quote">"Experienced Student that got truncated and then
hacked."</span>
[<a href=https://x.com/Doctor_I_am_The/status/1386723883647131658>https://x.com/Doctor_I_am_The/status/1386723883647131658</a>]
(The truncation theory is actually somewhat plausible, because when you
register a new account on Twitter, it first asks you to enter a display
name, and then it automatically suggests a username based on the display
name which is truncated to at most 15 characters, and <span
class="quote">"ExperiencedStud"</span> happens to be 15 characters long.
But why would Fleming not have edited the username manually?)</p>
<p>However as evidence against Fleming's hacking theory, the court
documents of the Fleming v. Sims case said: <span
class="quote">"According to Larry Sims, the parties first came into
contact after Defendants placed a job posting on an adult entertainment
job listing website. (See, e.g., ECF Nos. 90 at 1-2, 124 at 1, 188 at 1,
234 at 1, 244 at 1.) Plaintiff allegedly responded to the advertisement
and represented himself as an adult film actor and model. (Id.)"</span>
[<a href=https://www.courtlistener.com/docket/6282193/247/fleming-v-sims/>https://www.courtlistener.com/docket/6282193/247/fleming-v-sims/</a>]</p>
<p>Fleming's current Twitter account used to be called MedicinesDarkK1
until October 2020.
[<a href=https://x.com/search?q=medicinesdarkk1&f=live>https://x.com/search?q=medicinesdarkk1&amp;f=live</a>]
After that he briefly changed his username to Lys_og_sannhet (which
means <span class="quote">"light and truth"</span> in Norwegian) until
he settled with his current username Doctor_I_am_The.
[<a href=https://x.com/Thatsregrettab1/status/1331705114659229696>https://x.com/Thatsregrettab1/status/1331705114659229696</a>]
His old username was probably a reference to how Batman is called the
Dark Knight:
[<a href=https://x.com/Thatsregrettab1/status/1876688818805350811>https://x.com/Thatsregrettab1/status/1876688818805350811</a>]</p>
<div>
<img src=i/fleming-stud-3.png width=500>
</div>
<p>Fleming's IMDB profile has a photo where he poses as Batman and a
video where he poses as Joker:
[<a href=https://www.imdb.com/name/nm6222842/mediaviewer/rm57834240/>https://www.imdb.com/name/nm6222842/mediaviewer/rm57834240/</a>,
<a href=https://www.imdb.com/video/vi1525398297/>https://www.imdb.com/video/vi1525398297/</a>]</p>
<div>
<img src=i/fleming-stud-6.jpg width=800>
</div>
<p>An account with a Batman avatar called <span
class="quote">"ImBodu"</span> was created in May 2021 (where the <span
class="quote">"Im"</span> seems to match <span
class="quote">"I_Am"</span> in the username of Fleming's main account).
ImBodu currently only has 12 tweets. Its display name is <span
class="quote">"im back for richards ass ANND his bodu"</span> and its
bio says <span class="quote">"He thought he could block me. But imma get
him"</span>. Cheshire suspected the account might have been created by
Fleming himself.
[<a href=https://x.com/Thatsregrettab1/status/1397312313531650050>https://x.com/Thatsregrettab1/status/1397312313531650050</a>]
The account seemed familiar with several of Fleming's short films, IMDB
videos and YouTube videos, which likely few people besides Fleming
himself would know about. They supposedly included a Joker video which
would match the Batman theme, an audition video, a penis pump video, and
a video where Fleming exposed himself in a speedo. The Batman account
seems to have wanted Fleming to post his penis on the internet again
(which likely few people besides Fleming himself would be interested
in):
[<a href=https://x.com/imbodu/with_replies>https://x.com/imbodu/with_replies</a>,
<a href=https://archive.is/molua>https://archive.is/molua</a>]</p>
<div>
<img src=i/fleming-stud-4.png width=550>
</div>
<h3>
<a href=#Incorrect_chi_squared_comparison_of_Fleming_FMTVDM_method_and_mammography id=Incorrect_chi_squared_comparison_of_Fleming_FMTVDM_method_and_mammography>Incorrect
chi-squared comparison of Fleming FMTVDM method and mammography</a>
</h3>
<p>In 2019 Fleming published a paper titled <span
class="quote">"Statistical Demonstration that FMTVDM is Superior to
Mammography"</span>.
[<a href=https://www.remedypublications.com/open-access/statistical-demonstration-that-fmtvdm-is-superior-to-mammography-5186.pdf>https://www.remedypublications.com/open-access/statistical-demonstration-that-fmtvdm-is-superior-to-mammography-5186.pdf</a>]
FMTVDM stands for the <span class="quote">"Fleming Method for Tissue and
Vascular Differentiation and Metabolism"</span>, which Fleming claims
had a perfect accuracy of detecting breast cancer 1000 times out of 1000
when mammography detected breast cancer correctly only 828 times out of
1000:</p>
<div>
<img src=i/fleming-breast-1.png width=1000>
</div>
<p>In the table above the sum on the second row is wrong, because
<code>142+747</code> adds up to 889 and not 781. But that's the least of
Fleming's problems. The formula he used to calculate the chi-squared
test is also wrong, and it has two divisions by zero:</p>
<div>
<img src=i/fleming-breast-2.png width=500>
</div>
<p>In his formula <span class="quote">"χ<sup>2</sup> = E
(O-E)2/E"</span>, the first <span class="quote">"E"</span> should've
been <span class="quote">"∑"</span> and the number 2 should've been
superscripted: <span class="quote">"χ<sup>2</sup> =
∑((O-E)<sup>2</sup>/E)"</span>.</p>
<p>Fleming also misunderstood the meaning of the expected counts E in
the formula. He treated the results of FMTVDM as the expected counts and
he compared it to the results of mammography which he treated as the
observed counts. But his observed counts should've consisted of a
contingency matrix with one row for mammography and another row for his
Fleming method. And then he should've calculated the expected matrix
based on the observed matrix by doing a matrix multiplication of the row
sums and column sums and then dividing the result with the sum of all
values in the contingency matrix:</p>
<pre>&gt; m=rbind(mammography=c(77,34,747,142),Fleming_FMTVDM=c(111,0,889,0))
&gt; colnames(m)=c(&quot;present_correct&quot;,&quot;present_incorrect&quot;,&quot;absent_correct&quot;,&quot;absent_incorrect&quot;)
&gt; m
               present_correct present_incorrect absent_correct absent_incorrect
mammography                 77                34            747              142
Fleming_FMTVDM             111                 0            889                0
&gt; expected=rowSums(m)%*%t(colSums(m))/sum(m)
&gt; expected
     present_correct present_incorrect absent_correct absent_incorrect
[1,]              94                17            818               71
[2,]              94                17            818               71</pre>
<p>And only after that he should've calculated
<code>sum((observed-expected)^2/expected)</code>:</p>
<pre>&gt; sum((m-expected)^2/expected)
194.4741
&gt; chisq.test(m,correct=F) # same result
  Pearson&#39;s Chi-squared test

data:  m
X-squared = 194.47, df = 3, p-value &lt; 2.2e-16</pre>
<p>Here the procedure of calculating the expected matrix is written out
using basic arithmetic operators:</p>
<pre>&gt; rowsums=c(77+34+747+142,111+0+889+0)
&gt; colsums=c(77+111,34+0,747+889,142+0)
&gt; sum=c(77+111+34+0+747+889+142+0)
&gt; rowsums
[1] 1000 1000
&gt; colsums
[1]  188   34 1636  142
&gt; sum
[1] 2000
&gt; mult=matrix(c(1000*188,1000*34,1000*1636,1000*142,1000*188,1000*34,1000*1636,1000*142),byrow=T,ncol=4)
&gt; mult
       [,1]  [,2]    [,3]   [,4]
[1,] 188000 34000 1636000 142000
[2,] 188000 34000 1636000 142000
&gt; mult/sum # expected matrix
     [,1] [,2] [,3] [,4]
[1,]   94   17  818   71
[2,]   94   17  818   71</pre>
<p>I don't know if it would've been better to just use a simple 2 by 2
contingency matrix with one column for the number of correct detections
and another column for the number of incorrect detections, but it gave
me close to the same chi-squared value as the test with the 4 by 2
matrix:</p>
<pre>&gt; m=rbind(mammography=c(77+747,34+142),Fleming_FMTVDM=c(111+889,0+0))
&gt; colnames(m)=c(&quot;correct&quot;,&quot;incorrect&quot;)
&gt; m
               correct incorrect
mammography        824       176
Fleming_FMTVDM    1000         0
&gt; chisq.test(m,correct=F)
    Pearson&#39;s Chi-squared test

data:  m
X-squared = 192.98, df = 1, p-value &lt; 2.2e-16
&gt; expected=rowSums(m)%*%t(colSums(m))/sum(m)
&gt; sum((m-expected)^2/expected)
[1] 192.9825</pre>
<h3>
<a href=#Claim_that_his_Twitter_account_was_hacked_to_add_a_photo_of_David_Icke id=Claim_that_his_Twitter_account_was_hacked_to_add_a_photo_of_David_Icke>Claim
that his Twitter account was hacked to add a photo of David Icke</a>
</h3>
<p>In this tweet from January 2025, I believe Fleming suggested that his
Twitter account had been hacked so that someone had added a photo of
David Icke to a poster for the Crimes Against Humanity Tour:
[<a href=https://x.com/Doctor_I_am_The/status/1537897704830840836>https://x.com/Doctor_I_am_The/status/1537897704830840836</a>,
<a href=https://archive.is/2BM0D>https://archive.is/2BM0D</a>]</p>
<div>
<img src=i/fleming-icke.png width=500>
</div>
<p>But then was Fleming's Facebook account also hacked? The Facebook
account of the tour also has a poster that features David Icke:
[<a href=https://www.facebook.com/103054418992983/photos/142901295008295/>https://www.facebook.com/103054418992983/photos/142901295008295/</a>,
<a href=https://archive.is/qGiFS>https://archive.is/qGiFS</a>]</p>
<div>
<img src=i/fleming-icke-2.jpg width=500>
</div>
<p>Fleming's tweet that featured Icke was posted on June 17th 2022 UTC
and the Facebook post that featured Icke was posted on June 19th 2022
UTC.</p>
<p>David Icke's blog also has a post dated May 24th 2022 which says:
<span class="quote">"I am currently traveling the country with Drs. Judy
Mikovits, Richard Fleming and Reiner Fuellmich. Our one-day conference
topic in nine cities focuses on the case for crimes against humanity
having been committed by leaders of Big Pharma and the biosecurity
cartel."</span>
[<a href=https://davidicke.com/2022/05/24/monkeypox-technocracys-next-wave-of-crimes-against-humanity/>https://davidicke.com/2022/05/24/monkeypox-technocracys-next-wave-of-crimes-against-humanity/</a>]</p>
<p>Kevin McCairn who was also featured on the tour told me he asked
Fleming to remove Icke, but even after that Fleming still denied that
Icke was ever included on the tour and said it was just <span
class="quote">"more photoshopping and hacking"</span>:
[<a href=https://x.com/NestCommander/status/1882872073883976179>https://x.com/NestCommander/status/1882872073883976179</a>]</p>
<div>
<img src=i/fleming-icke-3.png width=600>
</div>
<p>In 2022 Fleming even quoted the poster with Icke himself and referred
to Icke in his tweet. So if the image file of the poster was
photoshopped by the hackers, then did the hackers also edit Fleming's
the text of Fleming's tweet so that it referred to the photoshopped
poster?
[<a href=https://x.com/Doctor_I_am_The/status/1537962322576125953>https://x.com/Doctor_I_am_The/status/1537962322576125953</a>]</p>
<div>
<img src=i/fleming-icke-4.jpg width=500>
</div>
<h2>
<a href=#Fake_clinical_trial_comparing_COVID_treatments_in_2020 id=Fake_clinical_trial_comparing_COVID_treatments_in_2020>Fake
clinical trial comparing COVID treatments in 2020</a>
</h2>
<p>See this post by Cheshire:
<a href=https://retractionwatch.com/2020/05/29/a-convicted-felon-wants-people-to-enroll-in-a-covid-19-clinical-trial-what-could-go-wrong/.><a
href="https://retractionwatch.com/2020/05/29/a-convicted-felon-wants-people-to-enroll-in-a-covid-19-clinical-trial-what-could-go-wrong/"
class="uri">https://retractionwatch.com/2020/05/29/a-convicted-felon-wants-people-to-enroll-in-a-covid-19-clinical-trial-what-could-go-wrong/</a>.</a></p>
<p>Before the paper about Fleming's fake trial was published in a
predatory journal, he published two preprint versions of the paper as
well as two accompanying short reports, so here's links to them all for
the sake of convenience:</p>
<ul>
<li>April 14th 2020: Fleming submitted a trial to ClinicalTrials.gov
with the title <span class="quote">"The Fleming [FMTVDM] Directed
CoVid-19 Treatment Protocol (FMTVDM)"</span>.
[<a href=https://clinicaltrials.gov/study/NCT04349410>https://clinicaltrials.gov/study/NCT04349410</a>]</li>
<li>October 20th 2020: Fleming published a preprint about the trial
titled <span class="quote">"Is there a treatment for
SARS-CoV-2?"</span>.
[<a href=https://www.researchsquare.com/article/rs-93307>https://www.researchsquare.com/article/rs-93307</a>]</li>
<li>October 30th 2020: Results of two supposed treatment arms were
described in a preprint titled <span class="quote">"Preliminary Results
of Tocilizumab and Interferon a- 2b Treatment of SARS-CoV-2"</span>.
[<a href=https://www.researchsquare.com/article/rs-98566>https://www.researchsquare.com/article/rs-98566</a>]</li>
<li>October 30th 2020: Fleming published a short case report of a
supposed Cuban patient titled <span class="quote">"Initial COVID
Hydroxychloroquine Failure Responds to Interferon and
Tocilizumab."</span>
[<a href=https://www.researchsquare.com/article/rs-98569>https://www.researchsquare.com/article/rs-98569</a>]</li>
<li>November 12th 2020: An updated version of the preprint from October
20th was published under the title <span class="quote">"Is there a
treatment for SARS-CoV-2? Quantitative Nuclear Imaging finds Treatments
for SARS-CoV-2."</span>
[<a href=https://www.researchsquare.com/article/rs-106988>https://www.researchsquare.com/article/rs-106988</a>]</li>
<li>February 18th 2021: The paper was published in a fake journal with
the title <span class="quote">"FMTVDM Quantitative Nuclear Imaging finds
Three Treatments for SARS-CoV-2"</span>.
[<a href=https://biomedres.us/fulltexts/BJSTR.MS.ID.005443.php>https://biomedres.us/fulltexts/BJSTR.MS.ID.005443.php</a>]</li>
</ul>
<h3>
<a href=#Comments_at_PubPeer id=Comments_at_PubPeer>Comments at
PubPeer</a>
</h3>
<p>Cheshire posted this comment about the paper at PubPeer:
[<a href=https://pubpeer.com/publications/618E865FABF6345892579A16F2AFEB>https://pubpeer.com/publications/618E865FABF6345892579A16F2AFEB</a>]</p>
<blockquote>
<p>Highlights: 23 sites, 7 countries, 1,800 patients, 10 treatment arms.
No other authors other than Fleming and son. None of the sites were
identified, although the ill-formatted tables (apparently due to
unfamiliarity with Acrobat) include Table 3 showing the country of the
sites. No funding was required for this study (?).</p>
<p>Interestingly, the Clinicaltrials.gov site shows that on October 8,
the first author/sponsor changed the <span class="quote">"treatment arms
and interventions"</span> details on all of the tested protocols. This
is 3 weeks <em>after</em> the trial was marked concluded. An example of
the changes:</p>
<p>Experimental: Treatment 1 <strong>ORIGINAL</strong></p>
<blockquote>
<p>Hydroxychloroquine 200 mg po q 8 hrs (600 mg qD) for a total of
10-days, and Azithromycin 500 mg IV on day 1, followed by 250 mg IV on
days 2-5 (to prevent bacterial superinfection).</p>
</blockquote>
<p>Experimental: Treatment 1 <strong>REVISED</strong></p>
<blockquote>
<p>Hydroxychloroquine 200 mg po q 8 hrs (600 mg qD) for a total of
10-days , <strong>OR Hydroxychloroquine 155 mg IV every 8-hours (600 mg
qD) for 10-days if patient is intubated</strong> and Azithromycin 500 mg
IV on day 1, followed by 250 mg IV on days 2-5 (to prevent bacterial
superinfection). (emphasis added)</p>
</blockquote>
<p>Source:
<a href=https://clinicaltrials.gov/ct2/history/NCT04349410?A=6&B=7&C=merged#StudyPageTop>https://clinicaltrials.gov/ct2/history/NCT04349410?A=6&amp;B=7&amp;C=merged#StudyPageTop</a></p>
<p>Other remarks:</p>
<ul>
<li>While there is mention of centralized institutional board review
there is no mention of informed consent by patients or authorized
representatives</li>
<li>There does not seem to be any data provided as to the racial
demographics of the patients</li>
<li>Although 8 different Figures are referenced in the text, none appear
in the preprint</li>
<li>The author acknowledges 6 individuals (by initials only) for their
assistance with the <span class="quote">"individual patient
centers."</span> In comparison, a NEJM-published study on Remdesivir for
COVID-19 with 35% fewer patients than this one, acknowledged over 200
physicians <em>by name</em>.
<a href=https://www.nejm.org/doi/full/10.1056/NEJMoa2007764>https://www.nejm.org/doi/full/10.1056/NEJMoa2007764</a></li>
</ul>
</blockquote>
<p>Cheshire also pointed out that after the trial is supposed to have
already ended, Fleming changed the number of enrollees from 500 to 501
to 1,800:</p>
<blockquote>
<p>Trial ended on September 14; at which time clinical trials.gov showed
500 enrollees.</p>
<p>Dr. Fleming posted this on Twitter on September 24, showing 501
enrollees:</p>
<div>
<img src=i/fleming-trial-1.jpg width=500>
</div>
<p>On October 2, the sponsor (Dr. Fleming) changed the trial enrollment
number to 1,800 at clinicaltrials.gov and 1,800 is the figure used in
this preprint.</p>
<div>
<img src=i/fleming-trial-2.jpg width=500>
</div>
</blockquote>
<p>However the discrepancy seems to be because Table 3 shows that the
total number of patients was 1,800, but 1,299 of them were outpatients
and the number of inpatients enrolled to phases I and II was 501:</p>
<div>
<img src=i/fleming-trial-7.png width=500>
</div>
<p>Fleming wrote that <span class="quote">"NCT04349410 required
identification of a seven-person SARS-CoV-2 treatment team"</span>
(where presumably each site would've had its own <span
class="quote">"treatment team"</span> with at least seven persons, but
the study is supposed to have included 23 sites so 23 times 7 would be
161). Cheshire wrote:</p>
<blockquote>
<p>Given the extraordinary <em>requirement</em> that a 7-person
treatment team (see below) be formed to treat the 501 hospitalized
patients, how is it possible that all this additional care was provided
with no funding from the sponsor? Why was this <span
class="quote">"required identification"</span> not included as part of
the pre-registered trial protocols? What evidence does the sponsor have
that this <span class="quote">"required identification"</span> was
followed in every case?</p>
<div>
<img src=i/fleming-trial-3.jpg width=600>
</div>
</blockquote>
<p>Someone also posted this comment at PubPeer:</p>
<blockquote>
<p>It is noteworthy that these ambiguities about the study's feasibility
and protocols are reminiscent of earlier skepticism, expressed 18 years
ago, about another purported clinical trial from the same author:</p>
<blockquote>
<p>...clinical trials are expensive, difficult and time-consuming. Even
small dietary trials can easily cost several hundred thousand dollars
and require entire research teams. The DPP estimated a cost of $1,075
just to recruit each participant. Fleming reports on a one-year trial of
100 participants and four diets with extensive follow-up. His paper,
however, has no co-authors; it acknowledges no source of funding, nor
any nurses, dietitians or technicians who might have helped. Fleming
identifies himself as Medical Director of Preventive Cardiology, the
Camelot Foundation at the Fleming Heart &amp; Health Institute, but if
his Web site or receptionist are any indication, he is the sole member
of each of those.</p>
<p>As for the issue of peer-review, Fleming states that his patients
<span class="quote">"were randomly assigned to one of the four dietary
regimens based upon dietary preferences."</span> This protocol is
pivotal to interpretation of the findings, yet oxymoronic: If patients
were assigned to diets based on their dietary preferences, then they
weren't randomly assigned. If they were randomly assigned, then their
preferences must be irrelevant. The two methods are incompatible. If
this paper was peer-reviewed, it was done poorly. If this constitutes
high-quality research in this field, then I suggest even more skepticism
is necessary.</p>
</blockquote>
<p>source:
<a href=https://www.washingtonpost.com/archive/lifestyle/wellness/2002/09/24/interactions/dfcd1470-eedb-474f-8ae8-714836e810d9/>https://www.washingtonpost.com/archive/lifestyle/wellness/2002/09/24/interactions/dfcd1470-eedb-474f-8ae8-714836e810d9/</a>
[not behind paywall here:
<a href=https://www.askbjoernhansen.com/2002/09/02/maybe_it_wasnt_a_big_fat_lie.html>https://www.askbjoernhansen.com/2002/09/02/maybe_it_wasnt_a_big_fat_lie.html</a>]</p>
</blockquote>
<p>Cheshire also wrote:</p>
<blockquote>
<p>This trial does not seem to have been registered in India
<a href=http://ctri.nic.in/Clinicaltrials/faq.php#6a>as appears to be
required</a>:</p>
<blockquote>
<p><strong>My trial is already registered in another Primary Register,
then why do I need to register again with the CTRI?</strong> A clinical
trial being conducted in India, is also required to be registered in the
CTRI as the CTRI captures data specific for the Indian arm of a trial,
e.g. Site and PI details, Name of Ethics Committee and approval status,
target sample size in India, start date in India etc.</p>
</blockquote>
<p>Search here:
<a href=http://ctri.nic.in/Clinicaltrials/advancesearchmain.php>http://ctri.nic.in/Clinicaltrials/advancesearchmain.php</a></p>
</blockquote>
<h3>
<a href=#Zinc_treatment_in_outpatients id=Zinc_treatment_in_outpatients>Zinc
treatment in outpatients</a>
</h3>
<p>Cheshire wrote:
[<a href=https://pubpeer.com/publications/618E865FABF6345892579A16F2AFEB>https://pubpeer.com/publications/618E865FABF6345892579A16F2AFEB</a>]</p>
<blockquote>
<p>I'm confused again.</p>
<p>From the Methods section of this preprint (posted 10/27/20):</p>
<blockquote>
<p>Outpatient Treatment: Patient recruitment for each outpatient
treatment site is shown in Tables 3, 4 and Figure 2. Outpatient
treatment was by definition provided by clinicians prior to hospital
admission. Outpatient sites included private offices, physician and
hospital clinics. Decision to treat (Treatments 1-4; Tables 2A and 2B)
was made solely by the physician and patient. <strong>All outpatients
received a minimum of 200 mg of elemental zinc daily</strong> while
taking aminoquinolines. Following initial evaluation including PCR
testing and initiation of treatment or the decision to provide no
treatment, patients returned 3-5 days later for re-evaluation."
(Emphasis added)</p>
</blockquote>
<p>I'm puzzled how the treating physicians would have known to treat
patients with 200 mg of zinc daily? In the protocol document dated April
2020 (still available at clinicaltrials.gov), there is no mention of
pre-hospitalization or outpatient treatment. On July 4, 2020, an
Appendix G was added that added a prehospitalization step which
includes, <span class="quote">"<strong>Begin immune supportive Rx,
including Zn.</strong>"</span> No <em>quantity</em> of Zn is
specified.</p>
<p>Further in a late September YouTube video by the first author he
recommended <strong>10 mg of daily zinc</strong> for prehospitalized
COVID-19 patients.</p>
<div>
<img src=i/fleming-trial-4.jpg width=500>
</div>
<p>If the protocol provided to physicians at the outset of the trial did
not include prehospitalization instructions, but zinc (of unknown
quantity) was included beginning in July, and in late September (after
the conclusion of this study) the first author was recommending 10 mg
daily Zn... how can the author make the statement that <strong>all
outpatients received a minimum of 200 mg of elemental zinc
daily?</strong></p>
</blockquote>
<p>Cheshire pointed out that a table in Fleming's preprint indicated
that in all treatment groups patients received either 10 mg of zinc
orally or 4 mg of zinc intravenously:</p>
<div>
<img src=i/fleming-trial-5.png width=600>
</div>
<p>But Fleming responded by saying that his table showed the treatment
regime in hospitalized patients and not outpatients:</p>
<div>
<img src=i/fleming-trial-6.jpg width=400>
</div>
<p>However in case zinc was suspected to be helpful in treating COVID,
why was the dosage of zinc reduced from 200 mg to 10 mg after a patient
was hospitalized? In Fleming's YouTube video, he also recommended a
dosage of only 10 mg of zinc to outpatients.</p>
<h3>
<a href=#Duration_of_hospitalization_and_99_83_effectiveness id=Duration_of_hospitalization_and_99_83_effectiveness>Duration
of hospitalization and 99.83% effectiveness</a>
</h3>
<p>The abstract of Fleming's paper said: <span class="quote">"The three
successful treatment regimens include (1) Tocilizumab &amp; Interferon
a-2b, (2) Primaquine, Clindamycin, Tocilizumab &amp; Interferon a-2b,
and (3) Methylprednisolone. These three regimens were effective 99.83%
of the time and shortened hospital stays from 40 ± 3 days to 1-2
weeks."</span></p>
<p>It's not clear how he determined the baseline hospital stay of 40 ± 3
days, because his study did not have any control group. I found no other
reference to the figure of 40 ± 3 days apart from the abstract. The
average duration of 40 days seems far too high, and the standard
deviation of 3 days seems too narrow relative to the average
duration.</p>
<p>A meta-analysis of COVID hospitalization studies from 2020 said:
<span class="quote">"We identified 52 studies, the majority from China
(46/52). Median hospital LoS [length of stay] ranged from 4 to 53 days
within China, and 4 to 21 days outside of China, across 45
studies."</span>
[<a href=https://link.springer.com/article/10.1186/s12916-020-01726-3>https://link.springer.com/article/10.1186/s12916-020-01726-3</a>]
A meta-analysis from 2021 said: <span class="quote">"The mean length of
hospital stay due to SARS-CoV-2 infection was 12.5 days (SD
6.8)."</span>
[<a href=https://pmc.ncbi.nlm.nih.gov/articles/PMC8206636/>https://pmc.ncbi.nlm.nih.gov/articles/PMC8206636/</a>]
A study from a hospital in Brazil said that the mean duration of
hospital stay was 10 ± 8 days.
[<a href=https://www.sciencedirect.com/science/article/pii/S1876034122001563>https://www.sciencedirect.com/science/article/pii/S1876034122001563</a>]</p>
<p>The conclusion of Fleming's paper said: <span
class="quote">"Successful treatment interventions focused on (1)
avoiding intubation or extubating the patient within a matter of days -
less than one week - to minimize the ARDS associated ventilator
complications associated with the immunologic ITR to SARS-CoV-2, in
addition to (2) using a combination of treatments within the first few
days of admission including Interferon a-2b, Tocilizumab, and
Methylprednisolone. These combinations were most effective if the
patient had already received an aminoquinoline as an outpatient, or
Primaquine as an inpatient. When provided the administration of
convalescent plasma proved effective; however, given the limited supply
of convalescent plasma, the potential consequences of a blood product
transfusion including increased potential for thrombosis as a plasma
product, and the availability of effective ITR treatments, convalescent
plasma should be reserved for cases not responding to Interferon a-2b,
Tocilizumab, Methylprednisolone, or the combination of Tocilizumab with
Interferon a-2b. These ITR drugs proved most promising when initiated
upon admission and when used in combination, reducing hospitalization
time from 30-45 days to as little as 18-25 days with 0.17%
mortality."</span></p>
<p>However 18 to 25 days is about 2.6 to 3.6 weeks, and not 1 to 2 weeks
like Fleming wrote in the abstract.</p>
<p>The mortality rate of 0.17% seems to match the figure of 99.83%
effectiveness Fleming mentioned in the abstract, so at first I thought
that by effectiveness Fleming simply meant the percentage of subjects
who did not die. Hovever then I thought I was wrong because I noticed
that his paper said: <span class="quote">"Successful treatment outcomes
were defined using the quantitative measurements of FMTVDM with a
reduction of ≥ 25, or a level of &lt; 150, Ferritin levels &lt; 270
ng/ml for men and &lt; 160 ng/ml for women, and an IL-6 level of &lt; 5
pg/ml."</span> But next I noticed that Fleming also wrote: <span
class="quote">"Three hundred and forty patients entered Phase I and
received sequentially added medical Treatment(s) until the patient
demonstrated treatment success or expired."</span> So apparently all of
his imaginary subjects are supposed to have been treated until they
either died or they recovered as measured by one of his three
biomarkers, which explains why his efficacy against death is the same as
his efficacy determined based on the biomarkers.</p>
<p>A mortality rate of 0.17% would correspond to 1 death per 570 to 606
people when rounded to the nearest digit. But I think the study didn't
even have 570 subjects who received any of Fleming's so-called <span
class="quote">"three successful treatment regimes"</span>. There were
only 501 inpatient subjects, and the section for outpatient treatment
said: <span class="quote">"Outpatient treatment was by definition
provided by clinicians prior to hospital admission. Outpatient sites
included private offices, physician and hospital clinics. Decision to
treat (Treatments 1-4; Tables 3A &amp; 3B) was made solely by the
physician and patient."</span> And Table 3A shows that none of Fleming's
<span class="quote">"three succesful treatment regimes"</span> were
given to outpatients, even though clindamycin alone was given to some
outpatients (in the imaginary realm where Fleming conducted his
trial):</p>
<div>
<img src=i/fleming-trial-10.png width=800>
</div>
<h3>
<a href=#All_treatment_arms_had_the_same_baseline_levels_of_ferritin_and_IL_6 id=All_treatment_arms_had_the_same_baseline_levels_of_ferritin_and_IL_6>All
treatment arms had the same baseline levels of ferritin and IL-6</a>
</h3>
<p>Cheshire pointed out that people in all 11 treatment arms had the
same baseline levels of ferritin and IL-6 at admission:</p>
<div>
<img src=i/fleming-trial-8.jpg width=600>
</div>
<p>When I asked Fleming to explain it, he replied: <span
class="quote">"As explained in the paper, although clearly not read or
comprehended by others, all participants began at baseline. This was
baseline. They were then randomly assigned to treatments and then based
upon measured FMTVDM outcomes, additional treatments were added or not.
Statistical analysis was by multiple ANOVA."</span>
[<a href=https://x.com/Doctor_I_am_The/status/1878259706700984586>https://x.com/Doctor_I_am_The/status/1878259706700984586</a>]
Then I asked him: <span class="quote">"Ok, so did you first measure the
ferritin level in each subject and then divided the subjects into
treatment arms so that each arm happened to have the same average
ferritin level with the same CI? It wouldn't be possible."</span> (Where
I should've said SD and not CI.) But Fleming replied: <span
class="quote">"You obviously are not very good at this and obviously
have not read the paper. The sequence was laid out in the paper. You act
like the typical student who has a reading assignment, shows up for
class not having read the assignment and then try to blame the professor
for your lack of understanding. Thus wasting your time, my time and
everyone else's time."</span></p>
<p>But I still don't understand how it's possible for each arm to have
the same levels of ferritin and IL-6 at admission.</p>
<h3>
<a href=#Comments_at_Retraction_Watch id=Comments_at_Retraction_Watch>Comments
at Retraction Watch</a>
</h3>
<p>Someone posted this comment:
[<a href=https://retractionwatch.com/2020/05/29/a-convicted-felon-wants-people-to-enroll-in-a-covid-19-clinical-trial-what-could-go-wrong/>https://retractionwatch.com/2020/05/29/a-convicted-felon-wants-people-to-enroll-in-a-covid-19-clinical-trial-what-could-go-wrong/</a>]</p>
<blockquote>
<p>According to the trial registration, <span class="quote">"study
chair"</span> Fleming is conducting a clinical trial testing FDA
regulated drugs for diagnosis and treatment in patients with Covid-19.
Since there are no approved treatments for Covid-19, and IND application
to FDA is required.</p>
<p>Per the FDA debarment notice, Fleming is prohibited <span
class="quote">"for 10 years from providing services in any capacity to a
person that has an approved or pending drug product
application."</span></p>
<p><a href=https://www.govinfo.gov/app/details/FR-2018-09-28/2018-21210>https://www.govinfo.gov/app/details/FR-2018-09-28/2018-21210</a></p>
<p>Perhaps Retraction Watch could clarify with FDA that the debarment
includes provision of services in FDA regulated research.</p>
</blockquote>
<p>Cheshire posted this reply:</p>
<blockquote>
<p>Source:
<a href=https://www.healthgrades.com/media/english/pdf/sanctions/HGPYA59E0F8B7C744ECBB05212010.pdf>https://www.healthgrades.com/media/english/pdf/sanctions/HGPYA59E0F8B7C744ECBB05212010.pdf</a></p>
<p>According to this State of Nebraska <span class="quote">"Petition for
Disciplinary Action,"</span> Fleming is <span class="quote">"permanently
excluded from Medicare, Medicaid, Tricare, and all other federal
healthcare programs."</span></p>
<p>I verified that he is on the exclusion list accessible here:
<a href=https://oig.hhs.gov/exclusions/index.asp>https://oig.hhs.gov/exclusions/index.asp</a>
and that site says:</p>
<blockquote>
<p>OIG has the authority to exclude individuals and entities from
Federally funded health care programs for a variety of reasons,
including a conviction for Medicare or Medicaid fraud. Those that are
excluded can receive no payment from Federal healthcare programs for any
items or services they furnish, order, or prescribe. This includes those
that provide health benefits funded directly or indirectly by the United
States (other than the Federal Employees Health Benefits Plan).</p>
<p>OIG maintains a list of all currently excluded individuals and
entities called the List of Excluded Individuals/Entities (LEIE). Anyone
who hires an individual or entity on the LEIE may be subject to civil
monetary penalties (CMP). To avoid CMP liability, health care entities
should routinely check the list to ensure that new hires and current
employees are not on it.</p>
</blockquote>
<p>Based on this, it sounds unlikely that Fleming/Dr. who? would be in a
position to legally run a clinical trial.</p>
</blockquote>
<p>However I don't know to what extent US regulations would've applied
to Fleming's trial since none of his imaginary trial sites were located
in the United States. There were 5 sites in India, 4 in Germany, 4 in
Brazil, 3 in Philippines, 3 in Cuba, 3 in Belgium, and 1 in South
Africa.</p>
<h3>
<a href=#The_paper_about_the_trial_was_published_in_a_fake_journal id=The_paper_about_the_trial_was_published_in_a_fake_journal>The
paper about the trial was published in a fake journal</a>
</h3>
<p>A blog post about the journal where Fleming's paper was published
said the following:
[<a href=http://flakyj.blogspot.com/2019/11/biomedical-journal-of-scientific.html>http://flakyj.blogspot.com/2019/11/biomedical-journal-of-scientific.html</a>]</p>
<blockquote>
<p>Although the ISSN entry for the Biomedical Journal of Scientific
&amp; Technical Research identifies it as a US publication, its mangled
management of the English language suggests otherwise. The website
describes the goal of this journal's <span
class="quote">"publishers"</span> as follows:</p>
<blockquote>
<p>The only motto of Biomedical Journal of Scientific &amp; Technical
Research (BJSTR) Publishers is accelerating the scientific and technical
research papers, considering the importance of technology and the human
health in the advanced levels and several emergency medical and clinical
issues associated with it, the key attention is given towards biomedical
research. <span class="red">Thus, asserting the requirement of a common
evoked and enriched information sharing platform for the craving
readers.</span></p>
<p>BJSTR is such a unique platform to accumulate and publicize
scientific knowledge on science and related discipline. This
multidisciplinary open access publisher is <span class="red">rendering a
global podium</span> for the professors, academicians, researchers and
students of the relevant disciplines to share their scientific
excellence in the form of an original research article, review article,
case reports, short communication, e-books, video articles, etc.</p>
<p>BJSTR Publishers are self supporting, with no dependency on any other
external sources (like universities, centers) for funds and <span
class="red">strives for the best and enhanced quality publications
competes the world wide open access publishing</span> market.</p>
<p>We always rely on the support from the members of our BJSTR family
that is relevantly our Authors, Editorial Committee members, advisory
board, Reviewers Board and all the technical support teams all over the
globe. <span class="red">We trust in the reciprocated coordination and
cooperation</span> in terms of sharing the scientific knowledge of
individuals and Groups of Research centers/areas <span class="red">will
in turn educates and provokes in advanced researches</span>. In this
case we would like to act as a media that anchors in the transformation
of information in the form of global online publication.</p>
</blockquote>
<p>With writing like this, the absence of <span
class="quote">"dependency on any ... external sources (like universities,
centers) for fund"</span> is understandable. So is the fact that the
journal is <a href=https://www.ncbi.nlm.nih.gov/nlmcatalog/101723284>not
in the National Library of Medicine collection and not indexed for
Medline</a>.</p>
<p>When
<a href=https://twitter.com/StefanLuschnig/status/997823306551250944>one
researcher</a> <span class="quote">"submitted a string of
machine-generated nonsense"</span> entitled <span class="quote">"The
Expression of the Proper-Name Effect Reinforces the Disarticulation
ofCommunicative [sic] Rationality,"</span> the article was <span
class="quote">"quickly accepted after scrupulous peer review."</span>
Evidently, the editors did not notice that the author was <strong>Harold
E. Potter of the Institute of Improbabilistics, University of Bogus,
UK</strong>. An author
<a href=https://www.researchgate.net/post/Biomedical_Journal_of_Scientific_Technical_Research_fake#view=5defc2b5d7141b4a5667c3c2>posting
on Researchgate</a> reports that when he declined to pay the $600 fee,
the journal dropped the price to $99.</p>
<p>Wisely, the publishers do not reveal their identities. The domain
name biomed-res.org in the email is registered to a privacy service. The
contact information on the website is Biomedical Research Network+, LLC,
1 Westbrook Corporate Center, Suite 300, one Westchester, IL 60154, USA
+1 (720) 414-3554 Fax - (720) 367-5187 <a
href="mailto:info@biomedres.us" class="email">info@biomedres.us</a> <a
href="mailto:angelaroy@biomedres.us"
class="email">angelaroy@biomedres.us</a>. The registrant of biomedres.us
provides the same physical address along with the name Biomedical
Research Network + LLC and email URL of <a
href="mailto:openaccessnetworks@gmail.com"
class="email">openaccessnetworks@gmail.com</a>. The
<a href=https://www.regus.com/offices/united-states/illinois/westchester/office-space/illinois-westchester-westbrook-corporate-center>Westbrook
Corporate Center</a> is a property with <span class="quote">"virtual
office"</span> space on sale for as little as $2 a day.</p>
</blockquote>
<p>Someone posted this comment to the blog post: <span
class="quote">"ALL these BJSTR, IRIS, CRIMSON, LUPINE are interconnected
to SRAWIK GROUP a proprietor company in Hyderabad India"</span>. However
I'm not sure if it's correct or not because there were no other hits on
Google when I searched for
<code>srawik "biomedical journal of scientific"</code>.</p>
<p>The associate editor of the BJSTR journal is suposed to be someone
called <span class="quote">"Angela Roy"</span>. Her location at Twitter
is listed as <span class="quote">"600 Third Avenue, 2nd floor, New York
- 10016, USA"</span>.
[<a href=https://x.com/Biomedres01>https://x.com/Biomedres01</a>] Her
bio at Orcid doesn't say anything about herself or list any of her
publications, but it only describes BJSTR in broken English: <span
class="quote">"Biomedical Journal of Scientific &amp; Technical Research
(BJSTR) is a multidisciplinary, scholarly Open Access publisher focused
on Genetic, Biomedical and Remedial missions in relation with Technical
Knowledge as well. We crave to select ground-breaking research based on
modernism, aptness, scientific connotation, prospective spectator's
interests, etc. We endeavor to provide by far and liberally accessible
belvedere to researchers and practitioners in support of their novel and
valuable ideas. We already have 2000+ Editorial Board members along with
5000+ Published articles in our box. Our BJSTR maintains a scrupulous,
methodical, fair peer review System. Besides, quality control is riveted
in each step of the publication process."</span>
[<a href=https://orcid.org/0000-0002-4278-5729>https://orcid.org/0000-0002-4278-5729</a>]</p>
<h2>
<a href=#Crimes_Against_Humanity_affidavit id=Crimes_Against_Humanity_affidavit>Crimes
Against Humanity affidavit</a>
</h2>
<p>This section addresses the PDF linked at the top of this page:
<a href=https://www.flemingmethod.com/crimesagainsthumaniy.><a
href="https://www.flemingmethod.com/crimesagainsthumaniy"
class="uri">https://www.flemingmethod.com/crimesagainsthumaniy</a>.</a>
(Archive link:
<a href=http://web.archive.org/web/20240217163948/https://www.flemingmethod.com/_files/ugd/659775_6532f42cb0b24f92bb36e224b7abac81.pdf.><a
href="http://web.archive.org/web/20240217163948/https://www.flemingmethod.com/_files/ugd/659775_6532f42cb0b24f92bb36e224b7abac81.pdf"
class="uri">http://web.archive.org/web/20240217163948/https://www.flemingmethod.com/_files/ugd/659775_6532f42cb0b24f92bb36e224b7abac81.pdf</a>.</a>)</p>
<h3>
<a href=#Chi_squared_calculation_of_COVID_cases_in_vaccine_trials id=Chi_squared_calculation_of_COVID_cases_in_vaccine_trials>Chi-squared
calculation of COVID cases in vaccine trials</a>
</h3>
<p>Fleming wrote: <span class="quote">"When the EUA documents were used
for the statistical analysis of the Pfizer, Moderna, and Janssen Drug
Vaccine Biologics, and the Chi-Square analysis of the results published
in those EUA documents was analyzed for the Pfizer vaccine as shown in
the following graphic, there was no statistical difference between
vaccinated and un-vaccinated people diagnosed with having COVID-19. To
be statistically different (a benefit for people being vaccinated) the
<span class="quote">'p (probability)- value'</span> must be less than or
equal to less than 5 times per hundred people. This is the scientific
definition of statistical benefit and is written as <span
class="quote">'p&lt;0.05'</span>. In the graphic the p-value was
0.224418 [note: actually 0.244218] and is NOT statistically significant;
i.e. there is no statistical difference in the number of people
diagnosed with COVID who were vaccinated when compared with the
non-vaccinated group of people."</span> And next he showed this image,
where he made the images on the right himself but he took the table on
the left from a review memorandum for the emergency use authorization of
the Pfizer vaccine:
[<a href=https://www.fda.gov/media/144416/download>https://www.fda.gov/media/144416/download</a>]</p>
<div>
<img src=i/fleming-vax.png width=1100>
</div>
<p>There were 8 cases out of 17,411 people in the vaccine group and 162
cases out of 17,511 people in the placebo group, so common sense
should've dictated that the difference between the groups was
statistically significant, and Fleming should've noticed that there was
something wrong with his calculation because he only got a p-value of
about 0.24.</p>
<p>In his contingency matrix the <span class="quote">"observed"</span>
column shows the number of people with no case, or for example
<code>17411-8</code> = 17403 for Pfizer. However I don't understand how
he calculated the <span class="quote">"expected"</span> column. The
numbers in parentheses show the result of
<code>rowSums(m)%*%t(colSums(m))/sum(m)</code> (for example
<code>34652*34752/69503</code> = 17326.25 for the top left square).</p>
<p>However the proper way to make the contingency matrix would've been
to have one column for the number of people with a COVID case and
another column for the number of people with no COVID case. It gave me a
p-value of about 9e-32 here:</p>
<pre>&gt; cases=c(8,162);people=c(17411,17511)
&gt; m=cbind(case=cases,no_case=people-cases)
&gt; rownames(m)=c(&quot;Pfizer&quot;,&quot;placebo&quot;)
&gt; m
        case no_case
Pfizer     8   17403
placebo  162   17349
&gt; chisq.test(m) # p-value shown as `&lt;2.2e-16` due to limit of floating point precision
  Pearson&#39;s Chi-squared test with Yates&#39; continuity correction

data:  m
X-squared = 137.5, df = 1, p-value &lt; 2.2e-16

&gt; .Machine$double.eps # smallest positive float `x` where `1+x!=1`
# [1] 2.220446e-16
&gt; chisq.test(m)$p.value # p-value is much lower than 2.2e-16
[1] 9.395097e-32
&gt; expected=rowSums(m)%*%t(colSums(m))/sum(m)
&gt; expected
        case  no_case
[1,] 84.7566 17326.24
[2,] 85.2434 17425.76
&gt; sum((m-expected)^2/expected) # manual calculation without correction
[1] 139.3046
&gt; chisq.test(m,correct=F)$stat # matches manual calculation
X-squared
 139.3046</pre>
<p>Kevin McCairn also got a similar p-value when he used MATLAB to do a
chi-squared test (but the reason why McCairn said he got a lower p-value
than me was because my p-value was shown as <code>&lt; 2.2e-16</code> by
R even though it was actually about 9e-32 which was similar to McCairn's
result). McCairn wrote:
[<a href=https://x.com/NestCommander/status/1882185862257242263>https://x.com/NestCommander/status/1882185862257242263</a>]</p>
<blockquote>
<p>Richard I'm not following the logic, not that I trust Pfizer data,
but the appropriate use of the Chi-squared in the tables above would be
calculated as follows (MATLAB code below), I get a smaller P Value than
Henjin, but I did no post-hoc corrections as it's only a 2x2 contingency
table, and the result is highly significant.</p>
<pre>% Data
observed = [8, 162; 17403, 17349]; % Observed frequencies
total = sum(observed, &lt;span class=quote&gt;&#39;all&#39;&lt;/span&gt;);
row_totals = sum(observed, 2);
col_totals = sum(observed, 1);

% Expected frequencies
expected = (row_totals * col_totals) / total;

% Chi-squared calculation
chi_squared = sum((observed - expected).^2 ./ expected, &lt;span class=quote&gt;&#39;all&#39;&lt;/span&gt;);

% Degrees of freedom
df = (size(observed, 1) - 1) * (size(observed, 2) - 1);

% P-value calculation
p_value = 1 - chi2cdf(chi_squared, df);

% Critical value for p = 0.05
critical_value = chi2inv(0.95, df);

% Display results
fprintf(&lt;span class=quote&gt;&#39;Chi-squared statistic: %.2f\n&#39;&lt;/span&gt;, chi_squared);
fprintf(&lt;span class=quote&gt;&#39;Degrees of freedom: %d\n&#39;&lt;/span&gt;, df);
fprintf(&lt;span class=quote&gt;&#39;Critical value (p=0.05): %.2f\n&#39;&lt;/span&gt;, critical_value);
fprintf(&lt;span class=quote&gt;&#39;P-value: %g\n&#39;&lt;/span&gt;, p_value); % Use %g for full precision scientific notation

% Conclusion
if chi_squared &gt; critical_value
    disp(&lt;span class=quote&gt;&#39;The result is significant at p = 0.05.&#39;&lt;/span&gt;);
else
    disp(&lt;span class=quote&gt;&#39;The result is not significant at p = 0.05.&#39;&lt;/span&gt;);
end

The result is significant at p = 0.05
χ2=139.3046
P=7.89×10−32</pre>
</blockquote>
<h3>
<a href=#Claim_that_no_other_coronavirus_has_a_furin_cleavage_site id=Claim_that_no_other_coronavirus_has_a_furin_cleavage_site>Claim
that no other coronavirus has a furin cleavage site</a>
</h3>
<p>Fleming wrote <span class="quote">"No other Coronavirus contains a
Furin Cleavage site as shown in the following graphic"</span>:</p>
<div>
<img src=i/fleming-prra.png width=600>
</div>
<p>But his graphic only contained sarbecoviruses. And there's sequences
of feline infectious peritonitis virus which contain an FCS at the S1/S2
junction with <code>PRRAR</code>:</p>
<div>
<img src=i/fleming-prra-2.png width=700>
</div>
<p>Many merbecoviruses like MERS and HKU5 also have a furin cleavage
site at the S1/S2 junction. The closest relative of sarbecoviruses are
hibecoviruses, which are a small subgenus of betacoronaviruses that I
believe currently includes only 4 published virus sequences (which are
Hp-betacoronavirus/Zhejiang2013, Zaria bat coronavirus ZBCoV, CD35, and
CD36). But all of them except ZBCoV have an FCS at the S1/S2
junction:</p>
<div>
<img src=i/fleming-fcs.png width=1000>
</div>
<h3>
<a href=#Claim_that_the_likelihood_of_an_FCS_occurring_spontaneously_is_3_21e_11 id=Claim_that_the_likelihood_of_an_FCS_occurring_spontaneously_is_3_21e_11>Claim
that the likelihood of an FCS occurring spontaneously is 3.21e-11</a>
</h3>
<p>Fleming wrote: <span class="quote">"The odds of such a cleavage site
occurring spontaneously (naturally) is 3.21 x 10<sup>-11</sup>. [...]
Simply put, the Furin cleavage site <span
class="quote">'critical'</span> for the SARS-CoV-2 viruses' entry into
human cells resulting in disease and death, is astronomical. It has it
not found in any other coronavirus at the critical S1/S2 location, and
the US Government which has funded the Gain-of-Function research, owns
the patent for this Furin Protease Cleavage Site, also associated with
the HIV glycoprotein (HIV gp120) 120 (sialic acid raft receptor) and
cancer progression."</span></p>
<p>The figure of 3.21e-11 comes from a paper by Ambati et al. titled
<span class="quote">"MSH3 Homology and Potential Recombination Link to
SARS-CoV-2 Furin Cleavage Site"</span>.
[<a href=https://frontiersin.org/journals/virology/articles/10.3389/fviro.2022.834808/full>https://frontiersin.org/journals/virology/articles/10.3389/fviro.2022.834808/full</a>]
But in the paper 3.21e-11 was not calculated to be the odds of an FCS
occurring naturally but the odds that a given 19-base sequence would be
contained both within a random 30,000-base sequence and within a set of
24,712 random 3,300-base sequences.</p>
<p>The paper by Ambati et al. included the image below with this
caption: <span class="quote">"Calculations of the probability of natural
occurrence of the 19nt sequence under study. The SARS-CoV-2 genome is
~30,000 nucleotides long (P1). The patented sequence is ~3,300
nucleotides long (P2). The patented library encompasses 24'712 sequences
of varying lengths with median length being in the range of 3,300
nucleotides. Conventional probability calculations are given of the
probability of the presence of a 19-nucleotide sequence in the human
genome and in one of the patented library sequences."</span></p>
<div>
<img src=i/fleming-modernagate-1.jpg width=700>
</div>
<p>Ambati calculated the likelihood that a given 19-base segment would
be included in SARS-CoV-2, and he multiplied it with the likelihood that
the same segment would be included in the single Moderna patent happened
to match the 19-base segment. But he should've instead just calculated
the likelihood that a given 19-base segment would match the Moderna
patent by chance, like what he did in his <code>P2c</code>
probability.</p>
<p>In order to search the patent BLAST database for sequences that were
published before 2020 and that matched the keyword <span
class="quote">"Moderna"</span>, I used the Entrez query
<code>0[0:2019] Moderna</code>. It returned a total of 669,810 sequences
with a total length of about 1,011,731,339 bases. So if all sequences
are assumed to be random, then the likelihood that any of them would
match a given 19-base segment would be about 0.007:
<code>1-((1-.25^19)^(1e9*2))</code> (where the likelihood of one or more
matches is calculated by subtracting the likelihood of zero matches from
1, and <code>1-.25^19</code> is the likelihood that a single 19-base
segment segment doesn't match the target sequence, and
<code>1e9*2</code> is the approximate number of 19-base segments on both
strands in all Moderna patent sequences). But since there's 8 different
ways to choose a 19-base segment of SARS-CoV-2 that contains the FCS
insert and a total of 7 surrounding bases from either side combined, the
likelihood further increases to <code>1-((1-.25^19)^(1e9*2*8))</code>
which is about 0.06.</p>
<p>For example if you pick two sets of 10 unique random integers between
1 and 1000, then what is the likelihood that both sets include one or
more shared integer? Ambati's logic for calculating the probability of
3.21e-11 was equivalent to calculating the likelihood that a given
integer like 123 would be incluced in both sets, which is
<code>(10/1000)*(10/1000)</code>. But the real likelihood that the two
sets contain one or more shared integers is
<code>1-kcombinations(990,10)/kcombinations(1000,10)</code>, which is
about 0.10 (where <code>kcombinations(990,10)</code> is the number of
ways to choose 10 unique integers which don't match any of the 10 unique
integers in the other set).</p>
<p>Ambati's probability calculation also had other more minor issues
which were mentioned in a response to his paper that was later published
in the same journal:
[<a href=https://www.frontiersin.org/journals/virology/articles/10.3389/fviro.2022.914888/full>https://www.frontiersin.org/journals/virology/articles/10.3389/fviro.2022.914888/full</a>]</p>
<ul>
<li>Ambati's <code>P2c</code> probability indicated the likelihood that
a given 19-base segment would occur exactly once among the patent
sequences. He should've instead calculated the likelihood that the
19-base segment would occur one or more times among the patent
sequences, which is the same as one minus the likelihood of zero
occurrences: <code>1-((1-.25^19)^((3300-19+1)*24712)</code> (where
<code>1-.25^19</code> is the likelihood that a segment at a given
position of a given patent sequence does not match the 19-base target
sequence, and <code>(3300-19+1)*24712</code> is the number of 19-base
segments on a single strand of all patent sequences). The resulting
probability is about 0.000295014, which is only about 0.01% higher than
Ambati's <code>P2c</code> probability so it doesn't make much difference
though.</li>
<li>Ambati forgot to take into account that the match can occur on
either strand, so he should've multiplied the <code>P2c</code>
probability by 2.</li>
<li>Ambati's probability should've been multiplied 8 because there's 8
different ways to select a 19-base segment that contains the entire
12-base FCS insert and a total of 7 surrounding bases on either side
combined.</li>
</ul>
<p>I don't even know where Ambati et al. got their number of 24,712
sequences in the Moderna patent. Actually the patent contains a total of
33,915 sequences, out of which 4,573 are amino acid sequences and 406
are RNA sequences that contain U bases. The remaining number of
nucleotide sequences is 28,936 which doesn't math Ambati's number of
24,712 sequences. And Ambati's guess that the median length of the
sequences was about 3,300 bases was also wrong because actually the
median length is 1,250.5 bases if you exclude the amino acid and RNA
sequences.</p>
<p>But anyway, Fleming was wrong to say that <span class="quote">"the
odds of such a cleavage site occurring spontaneously (naturally) is 3.21
x 10<sup>-11</sup>"</span>. The likelihood that the FCS would occur
spontaneously is not the same as the likelihood that a specific Moderna
patent would happen to match the specific insert that happened to add
the FCS along with 7 surrounding bases. There are also many other
inserts that could've added an FCS to SARS-CoV-2 and even many
combinations of point mutations that could've added an FCS. The FCS
sequence only needs to have two arginine residues with two other
residues in between, and the sequence can be located at multiple
different spots near the S1/S2 junction.</p>
<p>There's 6 different codons for arginine so there's many different
ways to introduce mutations that produce an arginine residue. For
example in RaTG13 the codons after the spot where Wuhan-Hu-1 has
<code>RRAR</code> are <code>AGT GTA GCT AGT</code> which codes for
<code>SVAS</code>, but it would change to <code>RVAR</code> if the third
nucleotide was changed to <code>A</code> or <code>G</code> and the 12th
nucleotide was changed to <code>A</code> or <code>G</code>.</p>
<p>The codons around the S1/S2 junction in RaTG13 are
<code>T:act N:aat S:tca R:cgt S:agt V:gtg A:gcc</code>, which would match
<code>RXXR</code> if either the <code>T</code> at the start or the
<code>A</code> at the end changed to <code>R</code>. But there's 3
combinations of 2 nucleotide changes and 3 combinations of 3 nucleotide
changes which would change the <code>T</code> at the start to
<code>R</code>, and there's 1 combination of 2 nucleotide changes and 5
combinations of 3 nucleotide changes which would change the
<code>A</code> at the end to <code>R</code>.</p>
<p>The likelihood that the 12-base FCS insert would occur by chance is
not even the same as the likelihood that the 12-base FCS insert and the
7 surrounding bases would occur by chance. And the likelihood that the
specific FCS motif in Wuhan-Hu-1 would occur by chance is not the same
as the likelihood that any FCS motif would occur by chance. And the
likelihood that the FCS insert in Wuhan-Hu-1 would occur naturally is
not the same as the likelihood that it would occur by chance, because
certain mutations might be favored by natural selection. And the
likelihood that the 19-base segment happened to match one specific
Moderna patent is not the same as the likelihood that the 19-base
segment would have occurred naturally.</p>
<p>In order to estimate the likelihood that the 12-base insert and a
total of 7 surrounding bases from either side would have an exact match
in the patent database, you can go here:
<a href=https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn.><a
href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn"
class="uri">https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn</a>.</a>
Set the query sequence to <code>ACTAATTCTCCTCGGCGGGCACGTAG</code>, which
consists of the 12-base FCS insert, 7 bases before it, and 7 bases after
it. Then switch the database to patent sequences, insert <span
class="quote">"SARS-CoV-2 (taxid:2697049)"</span> as the organism and
click the exclude checkbox next to it, and under algorithm parameters
set maximum results to 1000, and then click <span
class="quote">"BLAST"</span>. When I ran the query, the first 392
matches were mostly matches to SARS-CoV-2 that were not removed by the
species filter, but the matches to the Moderna patent sequences were
listed on rank 393 to 396 (JP 2017197545-A/7090, JP 2017197545-A/7089,
JP 2015518816-A/7090, and JP 2015518816-A/7089). However the E-value of
all matches was about 0.34, which means that about 0.34 similarly close
matches are expected to occur by chance in the patent database:</p>
<div>
<img src=i/fleming-modernagate-2.png width=900>
</div>
<p>The E-value depends on the size of the database, and the patent
database is fairly small compared to other databases like the core
nucleotide database. So when I repeated the previous search but I
switched to the core nucleotide database, there were a bunch of
bacterial sequences that got a perfect match to some 19-base subsegment
of the 26-base query, but their E values were about 8.9 which means that
about 8.9 similarly close matches were expected to occur by chance.</p>
<p>There's 29,885 different 19-base segments of Wuhan-Hu-1, but 1,670 or
about 6% of them had an exact match to the GRCh38 human reference
genome:</p>
<pre>$ brew install bowtie2 seqkit
$ wget ftp://ftp.ccb.jhu.edu/pub/data/bowtie_indexes/GRCh38_no_alt.zip;unzip GRCh38_no_alt.zip
$ curl -s &#39;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&amp;rettype=fasta&amp;id=MN908947&#39;&gt;sars2.fa
$ seqkit seq -s sars2.fa|awk -F &#39;&#39; &#39;{for(i=1;i&lt;length-18;i++)print &quot;&gt;&quot;i&quot;\n&quot;substr($0,i,19)}&#39;&gt;frag.fa
$ bowtie2 -p4 --no-unal -x GRCh38_noalt_as/GRCh38_noalt_as --score-min C,0,-1 -fU frag.fa&gt;temp
29884 reads; of these:
  29884 (100.00%) were unpaired; of these:
    28214 (94.41%) aligned 0 times
    1427 (4.78%) aligned exactly 1 time
    243 (0.81%) aligned &gt;1 times
5.59% overall alignment rate</pre>
<p>Jikky's 19-base segment matches bases 23601 to 23619 of Wuhan-Hu-1,
which codes for the <code>PRRAR</code> codons along with 2 bases before
them and 2 bases after them. The 19-base segment matches the 12-base
insertion in Wuhan-Hu-1 and the next 7 bases. However if we go along
with Jikky's theory where SARS2 acquired the insert in a cell culture
from the human MSH3 gene, it's not clear if the 7-base flanking region
after the insert would've helped to facilitate accidental recombination
much better than a 6-base or 5-base flanking region. And if the last two
bases from Jikky's segment are left out so the flanking region is only 5
bases long, then the 17-base segment has a perfect match on BLAST to all
of these patent sequences published before 2020:</p>
<pre>$ curl &#39;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&amp;rettype=xml&amp;id=HC903376,HC915932,HC908933,HC905783,HB443296,GN095906,BD179519&#39;&gt;temp.xml
$ xmlstarlet fo -D temp.xml|xml sel -t -m //GBSeq -v GBSeq_accession-version -o \| -v GBSeq_definition -o \| -v GBSeq_create-date -o \| -v .//GBReference/GBReference_title -o \| -v .//GBReference/GBReference_journal -o \| -n|awk -F\| &#39;$3!~/202/&#39;
HC903376.1|Sequence 13528 from Patent EP2194140|18-JUN-2010|Process for the production of fine chemicals|EP2194140-A2 13528 09-JUN-2010 Metanomics GmbH (DE)|
HC915932.1|Sequence 26085 from Patent EP2194140|18-JUN-2010|Process for the production of fine chemicals|EP2194140-A2 26085 09-JUN-2010 Metanomics GmbH (DE)|
HC908933.1|Sequence 19086 from Patent EP2194140|18-JUN-2010|Process for the production of fine chemicals|EP2194140-A2 19086 09-JUN-2010 Metanomics GmbH (DE)|
HC905783.1|Sequence 15936 from Patent EP2194140|18-JUN-2010|Process for the production of fine chemicals|EP2194140-A2 15936 09-JUN-2010 Metanomics GmbH (DE)|
HB443296.1|Sequence 19 from Patent WO2009077406|14-JUL-2009|Lipid metabolism proteins, combinations of lipid metabolism proteins and uses thereof|WO2009077406-A1 19 25-JUN-2009 BASF Plant Science GmbH (DE)|
GN095906.1|Sequence 687 from Patent WO2009037279|16-APR-2009|Plants with increased yield|WO2009037279-A1 687 26-MAR-2009 BASF Plant Science GmbH (DE)|
BD179519.1|Highly thermophilic bacterium-derived protein and gene encoding it|15-MAY-2003|Highly thermophilic bacterium-derived protein and gene encoding it|JP2002325574-A 10 12-NOV-2002 THE INSTITUTE OF PHYSICAL AND CHEMICAL RESEARCH|</pre>
<p>To reproduce the table above, go here:
<a href=https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn.><a
href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn"
class="uri">https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn</a>.</a>
Enter <code>CTCCTCGGCGGGCACGT</code> to the field at the top, switch the
database to <span class="quote">"Patent sequences(pat)"</span>, and
click BLAST. Then copy the accession numbers up to the last result where
query coverage and identity are both 100%, and run the code above with
the accession IDs joined by commas inserted as the id parameter.</p>
<p>The Moderna patent includes a total of 33,915 sequences. It's an
unusual practice to include such a large amount of genetic material in a
single patent, so McKernan called it a <span class="quote">"kitchen sink
patent"</span>. You can download all sequences from here:
<a href=https://seqdata.uspto.gov/docdetail?docId=US09587003B2.><a
href="https://seqdata.uspto.gov/docdetail?docId=US09587003B2"
class="uri">https://seqdata.uspto.gov/docdetail?docId=US09587003B2</a>.</a>
Their average length is about 1,700 bases and their total length is
about 58 million bases:</p>
<pre>$ unzip US09587003-20170307-SUPP.zip
$ sed &#39;s/.*SEQUENCE: /&gt;/;/^[&lt; ]/d;s/  *[0-9]*\r$//;s/ //g&#39; Psips/Data/03/US/2017/870/095/B2/sequence/US09587003-20170307-S00001.TXT|seqkit stat
file  format  type  num_seqs     sum_len  min_len  avg_len  max_len
-     FASTA   DNA     33,915  57,673,267        7  1,700.5   18,787</pre>
<p>The Moderna patent contains 4,573 amino acid sequences and 406 RNA
sequences that contain U bases. And out of the remaining 28,936
sequences, 24,363 sequences don't have any stop codons in the middle if
they are translated. I believe they include all or nearly all human
protein-coding genes, because the human genome contains about 20,000
protein-coding genes and their average length is about 1,500 bases:</p>
<pre>$ wget ftp://ftp.ensembl.org/pub/release-109/fasta/homo_sapiens/cds/Homo_sapiens.GRCh38.cds.all.fa.gz
$ seqkit fx2tab Homo_sapiens.GRCh38.cds.all.fa.gz|grep transcript_biotype:protein_coding|awk &#39;!a[$7]++&#39;|seqkit tab2fx|seqkit stat
file  format  type  num_seqs     sum_len  min_len  avg_len  max_len
-     FASTA   DNA     19,562  29,733,131        3  1,519.9  100,272</pre>
<p>When I generated all 19-base segments of Wuhan-Hu-1 with the poly(A)
tail removed and I searched for them within the the Moderna patent,
there were 3 other 19-base segments besides Jikky's segment which had a
perfect match to the patent:</p>
<pre>$ tr -d \\r&lt;Psips/Data/03/US/2017/870/095/B2/sequence/US09587003-20170307-S00001.TXT|awk &#39;/TYPE: DNA/&#39; RS= &#39;ORS=\n\n&#39;|sed &#39;s/&lt;400&gt; SEQUENCE: /&gt;/;/^[&lt; ]/d;s/  *[0-9]*$//;s/ //g&#39;|seqkit grep -svp u&gt;moderna.fa
$ bowtie2-build --thread 4 moderna.fa{,}
$ seqkit replace -sip&#39;a*$&#39; sars2.fa|seqkit sliding -W19 -s1|seqkit replace -p&#39;.*:&#39;&gt;frag.fa
$ bowtie2 -p4 --no-unal -x moderna.fa --score-min C,0,-1 -fU frag.fa&gt;moderna.sam
$ (echo sars2_pos patent_sequence_number match_start_in_patent_sequence sequence;grep -v ^@ moderna.sam|cut -f1,3,4,10)|column -t
sars2_pos    patent_sequence_number  match_start_in_patent_sequence  sequence
3180-3198    18510                   967                             AAGAAGAGCAAGAAGAAGA
4459-4477    26445                   1172                            AGTTTCAACTATACAGCGT
8293-8311    30068                   1160                            CTATAACAAAGTTGAAAAC
23601-23619  11651                   2760                            CTACGTGCCCGCCGAGGAG</pre>
<p>(The last sequence above matched both the patent sequence IDs 11651
and 11652 but Bowtie2 only returned one of the matches.)</p>
<p>If the amino acid and RNA sequences are excluded, the Moderna patent
has a total of about 49 million 19-base segments. And there's 29,885
19-base segments in Wuhan-Hu-1. So if both collections of segments would
be random, the expected number of exact matches between the collections
would be <code>.25^19*49159360*29885*2</code> which is about 11.</p>
<p>Jikky wrote this about the 19-base segment: <span class="quote">"In
order for that sequence to have arisen in that virus, the virus which
was manufactured with its HIV inserts, had to have had been infected
into patented cell lines supplied by Moderna that had that unique
sequence not seen in any other virus."</span>
[<a href=https://www.arkmedic.info/p/how-to-blast-your-way-to-the-truth>https://www.arkmedic.info/p/how-to-blast-your-way-to-the-truth</a>]
However it's possible for bioweaponeers to edit the genome as a text
file and then synthesize the whole genome, so they don't have to find a
cell line which contains the 12-base segment they want to insert, then
infect that cell line with the virus a zillion times until the right
piece of the cell line's genome happens to get inserted to the right
spot of the viral genome by chance. In the comments of Jikky's Substack
post, Jennifer Smith who is a PhD virologist also said that <span
class="quote">"the <span class="quote">'accidental'</span> nature you
implied is less likely than purposeful construction"</span>.
[<a href=https://www.arkmedic.info/p/how-to-blast-your-way-to-the-truth/comment/4545048>https://www.arkmedic.info/p/how-to-blast-your-way-to-the-truth/comment/4545048</a>]</p>
<p>Jikky also stated it as a fact that SARS-CoV-2 was made using an MSH3
cell line patented by Moderna: <span class="quote">"Irrespective of
whether it was released there it distracts from the fact that it was
made in a lab using a Moderna patented MSH3_mut cell line."</span>
[<a href=https://x.com/JikkyKjj/status/1474581145979293702>https://x.com/JikkyKjj/status/1474581145979293702</a>]</p>
<p>Kevin McKernan was blocked by Jikkyleaks in 2022 after McKernan
debunked Jikky's Modernagate theory in this thread:
<a href=https://x.com/Kevin_McKernan/status/1484987210508255233.><a
href="https://x.com/Kevin_McKernan/status/1484987210508255233"
class="uri">https://x.com/Kevin_McKernan/status/1484987210508255233</a>.</a>
I recommend reading the whole thread. McKernan said that Jikkyleaks had
dumped his reputation on a Bible codes interpretation of BLAST:
[<a href=https://twitter.com/search?q=from%3Akevin_mckernan+to%3Ajikkykjj+since:2022-2-10&f=live>https://twitter.com/search?q=from%3Akevin_mckernan+to%3Ajikkykjj+since:2022-2-10&amp;f=live</a>]</p>
<div>
<img src=i/fleming-modernagate-tweets.png width=600>
</div>
<p>Added later: Modernagate was also debunked here by the Japanese
patent expert Patent_SUN:
<a href=https://x.com/Patent_SUN/status/1901917154578202855.><a
href="https://x.com/Patent_SUN/status/1901917154578202855"
class="uri">https://x.com/Patent_SUN/status/1901917154578202855</a>.</a>
For example he wrote:</p>
<blockquote>
<p>The following five patent publications ① to ⑤ each includes the above
reverse complementary sequence in its Sequence Listing, but no patent
rights have been granted to the reverse complementary sequence
itself.</p>
<p>①US9149506B2<br> ②US9216205B2<br> ③US9255129B2<br> ④US9301993B2<br>
⑤US9587003B2</p>
<p>Therefore, we must carefully explain the reverse complementary
sequence so as not to mislead the public. Regarding those patent
publications ① to ⑤, by referring to the <span class="quote">"We
claim"</span> section (scope of Claims) attached to the end of each
specification, we can understand where the patent rights exist.</p>
<p>As we can see in each <span class="quote">"We claim"</span> section
of the patent publications ① to ⑤, there is no description of <span
class="quote">"CTACGTGCCCGCCGAGGAG."</span></p>
<p>From this fact, we can understand that the patent rights have NOT
been granted to it.</p>
<p>[...]</p>
<p>The topic has strayed off course, but realistically, it is not
possible for Moderna to obtain a patent for this specific sequence,
<span class="quote">"CTACGTGCCCGCCGAGGAG,"</span> or just a part of it
(i.e., the furin cleavage site). This is because Moderna disclosed <span
class="quote">"SEQ ID 11652"</span> as a complete sequence, which
includes that specific sequence, but did not disclose the segment as an
independent invention. This fails to meet the support requirement for
patentability, and it is unlikely that a patent will be granted for this
specific sequence in the future.</p>
<p>Furthermore, even if Moderna were to attempt to obtain a patent right
for <span class="quote">"SEQ ID 11652,"</span> I believe it would be a
difficult process. Realistically, if Moderna were to pursue a patent
right for <span class="quote">"SEQ ID 11652,"</span> the specification
would need to explicitly describe the usage, function, and effects of
<span class="quote">"SEQ ID 11652"</span> in order to meet the
aforementioned support requirement. However, none of the specifications
in the Patents 1 to 5 mention <span class="quote">"SEQ ID 11652,"</span>
nor is there any disclosure suggesting it.</p>
<p>Additionally, Moderna does not address the complementary sequence of
<span class="quote">"CTACGTGCCCGCCGAGGAG,"</span> which is <span
class="quote">"CTCCTCGGCGGGCACGTAG,"</span> anywhere in the
specification. Therefore, the patent rights of the Patents 1 to 5 never
extend to the <span class="quote">'CTCCTCGGCGGGCACGTAG'</span> sequence
of the COVID-19 virus.</p>
</blockquote>
<p>Another addendum: I went to nucleotide BLAST:
<a href=https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn.><a
href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn"
class="uri">https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn</a>.</a>
I entered the accession of SARS-CoV-2 to the text field at the top
(NC_045512), and I changed the database to patent sequences. I set the
Entrez query to <code>0:2019[dp] Moderna</code> to return patent
sequences that were published at GenBank before 2020 and that matched
the word Moderna in any of the fields. Then under algorithm parameters I
reduced the word size to 16, I increased the expect threshold to 1000,
and I increased maximum target sequences to 1000.</p>
<p>There were a total of 495 results. The best matches with 100%
identity were 23 bases long, but even they had an expect value of about
2.3, which means that about 2.3 similarly close matches were expected to
occur by chance:</p>
<div>
<img src=i/modernagate-blast.png width=900>
</div>
<p>The expect values of the perfect 19-base matches were about 381,
which means that about 381 similarly close matches were expected to
occur by chance within the set of patent sequences that were published
before 2020 and that matched the word <span
class="quote">"Moderna"</span>.</p>
<p>Then I clicked <span class="quote">"Download"</span> and selected
<span class="quote">"Hit Table (text)"</span>. It showed that there were
176 perfect matches with a length of 19 bases, 55 with length 20, 6 with
length 21, 13 with length 22, and 6 with length 23:</p>
<pre>$ awk &#39;$3==100&amp;&amp;$4&gt;=19{a[$4]++}END{for(i in a)print i,a[i]}&#39; Downloads/XRM91BX9013-Alignment.txt
19 176
20 55
21 6
22 13
23 6</pre>
<p>There weren't even that many duplicate matches among the perfect
matches that were at least 19 bases long, because the matches started at
92 unique positions of the genome of SARS-CoV-2. There were also 256
unique patent sequences which had a perfect match at least 19 bases
long:</p>
<pre>$ awk &#39;$3==100&amp;&amp;$4&gt;=19&#39; Downloads/XRM91BX9013-Alignment.txt|cut -f7|sort -u|wc -l
92
$ awk &#39;$3==100&amp;&amp;$4&gt;=19&#39; Downloads/XRM91BX9013-Alignment.txt|cut -f2|sort -u|wc -l
256</pre>
<p>The perfect matches that were at least 19 bases long covered 1,532
bases of the genome of SARS-CoV-2, which is about 5% of the total length
of the genome:</p>
<pre>$ awk &#39;$3==100&amp;&amp;$4&gt;=19{for(i=$7;i&lt;=$8;i++)a[i]}END{print length(a)}&#39; Downloads/XRN2HBFD016-Alignment.txt
1532</pre>
<p>When I clicked <span class="quote">"Search Summary"</span>, it showed
that there were a total of about 700,000 sequences in the patent
database that matched the Entrez query <code>0:2019[dp] Moderna</code>,
and their total length was about a billion bases:</p>
<div>
<img src=i/modernagate-blast-2.png width=500>
</div>
<p>The total length of all sequences in the patent database in March
2025 was about 28.3 billion bases. So about 4% of the total length was
made up by sequences that were published before 2020 and that matched
the keyword <span class="quote">"Moderna"</span>.</p>
<p>This plot shows all matches with the same or lower expect value as
the match to Jikky's 19-base segment:</p>
<div>
<img src=i/modernagate-blast-3.png width=1000>
</div>
<div class="spoiler">
<button>
Show code
</button>
<pre>t=fread(&quot;http://sars2.net/f/modernagate-blast.txt&quot;)
p=unique(t[,.(start=qstart,end=qend,expect=evalue,mismatch=mismatch+gaps)])

gene=fread(text=&quot;name,start,end
1a,266,13468
1b,13468,21555
S,21563,25384
3,25393,26220
E,26245,26472
M,26523,27191
6,27202,27387
7,27394,27759
8,27894,28259
N,28274,29533
10,29558,29674&quot;)
bars=gene[,c(start[1],pmean(start[-1],end[-.N]),end[.N])]

exp=.16;ylim=10^(c(-1,1)*exp+log10(range(p$expect)))
ybreak=unique(p$expect);xend=29903;xbreak=c(seq(1,xend,2e3),xend)

color=c(&quot;black&quot;,&quot;gray60&quot;,hsv(c(20,12,6,3,0,31,27)/36,1,.7))

annox=23601+10;annoy=10^(log10(381)-exp);annoy2=10^mean(log10(ybreak[3:4]))

ggplot(p)+
annotate(&quot;rect&quot;,xmin=1,xmax=xend,ymin=ylim[1],ymax=ylim[2],linewidth=.4,lineend=&quot;square&quot;,color=&quot;gray75&quot;,fill=NA)+
geom_vline(xintercept=bars,color=&quot;gray75&quot;,linewidth=.4,lineend=&quot;square&quot;)+
geom_rect(aes(fill=factor(mismatch),xmin=start,xmax=end,ymin=10^(log10(expect)-exp),ymax=10^(log10(expect)+exp)))+
geom_point(aes(color=factor(mismatch),x=pmean(start,end),y=expect),size=1.5,stroke=0)+
geom_text(data=gene,aes(x=pmean(start,end),y=ylim[1],label=name),size=3.5,hjust=.5,vjust=-.7)+
annotate(&quot;segment&quot;,x=annox,xend=annox,y=annoy2*1.3,yend=annoy,arrow=arrow(type=&quot;closed&quot;,length=unit(4,&quot;pt&quot;)),lineend=&quot;butt&quot;,linejoin=&quot;mitre&quot;,color=&quot;red&quot;,size=.5)+
annotate(&quot;text&quot;,x=annox,y=annoy2,size=3.8,color=&quot;red&quot;,label=&quot;#CTCCTCGGCGGGCACGTAG&quot;)+
labs(x=NULL,y=&quot;Expect value&quot;,title=&quot;BLAST matches to SARS-CoV-2 in patent sequences published before 2020 matching keyword \&quot;Moderna\&quot;&quot;)+
scale_x_continuous(limits=c(1,xend),breaks=xbreak)+
scale_y_log10(breaks=ybreak,labels=ifelse(ybreak&gt;=10,round(ybreak),ybreak))+
scale_color_manual(values=color,name=&quot;Mismatches&quot;)+
scale_fill_manual(values=color,name=&quot;Mismatches&quot;)+
coord_cartesian(clip=&quot;off&quot;,expand=F)+
theme(axis.text=element_text(size=10,color=&quot;black&quot;),
  axis.text.x=element_text(margin=margin(3)),
  axis.ticks=element_line(size=.4,color=&quot;gray75&quot;),
  axis.ticks.length=unit(4,&quot;pt&quot;),
  legend.background=element_blank(),
  legend.key=element_blank(),
  legend.key.height=unit(13,&quot;pt&quot;),
  legend.key.width=unit(13,&quot;pt&quot;),
  legend.spacing.x=unit(3,&quot;pt&quot;),
  legend.text=element_text(size=11),
  panel.background=element_blank(),
  plot.margin=margin(5,5,5,5,&quot;pt&quot;),
  plot.title=element_text(size=11,face=2,margin=margin(1,,5)))
ggsave(&quot;1.png&quot;,width=8.5,height=2.4,dpi=300*4)

sub=&quot;The patent BLAST database was searched for matches to Wuhan-Hu-1. The Entrez filter `0:2019[dp] Moderna` was used to select sequences that were published before 2020 and that matched the keyword \&quot;Moderna\&quot;, which kept a total of about 700,000 sequences with a total length of about 1 billion bases. The match to #CTCCTCGGCGGGCACGTAG had an expect value of about 381, which means that about 381 similarly close matches were expected to occur by chance among the filtered set of patent sequences. There were a total of 495 matches to SARS-CoV-2 with the same or lower expect value, out of which 215 matches were unique. All unique matches are shown here. The length of the matches ranges from 19 to 40 bases. The matches cover 3,524 out of 29,903 bases of Wuhan-Hu-1. The blastn task was used with a word size of 16.&quot;
system(paste0(&quot;mogrify -trim 1.png;magick 1.png \\( -size `identify -format %w 1.png`x -font Arial -interline-spacing -3 -pointsize $[43*4] caption:&#39;&quot;,gsub(&quot;&#39;&quot;,&quot;&#39;\\\\&#39;&quot;,sub),&quot;&#39; -splice x80 \\) -append -resize 25% -bordercolor white -border 24 -dither none -colors 256 1.png;pngcrush -rem alla -reduce 1.png 1.png&quot;))</pre>
</div>
<p>When I searched the nucleotide database at GenBank for the query
<code>0:2019[dp] moderna patent</code>, there were a total of 671,931
results, which is close to the same as the number of sequences in the
patent BLAST database that matched the query
<code>0:2019[dp] moderna</code>.
[<a href=https://www.ncbi.nlm.nih.gov/nuccore/?term=0%3A2019%5Bdp%5D+moderna+patent>https://www.ncbi.nlm.nih.gov/nuccore/?term=0%3A2019%5Bdp%5D+moderna+patent</a>]</p>
<h3>
<a href=#Alex_Washburnes_endonuclease_fingerprint_paper id=Alex_Washburnes_endonuclease_fingerprint_paper>Alex
Washburne's endonuclease fingerprint paper</a>
</h3>
<p>Fleming's affidavit referred to Alex Washburne's paper titled <span
class="quote">"Endonuclease fingerprint indicates a synthetic origin of
SARS-CoV-2"</span>.</p>
<p>The paper said: <span class="quote">"Fisher's exact test was used to
assess if there was a higher rate of silent mutations within BsaI/BsmBI
recognition sites compared to the rest of the viral genome. Odds ratios
were computed as the ratio of silent mutations to all other nucleotides
within BsaI/BsmBI sites of either genome in a pairwise alignment
compared to the ratio of silent mutations outside BsaI/BsmBI sites to
all other nucleotides. There are 12 silent mutations found in 9 distinct
BsaI/BsmBI sites between RaTG13 and SARS-CoV-2, and 882 silent mutations
outside of BsaI/BsmBI sites."</span>
[<a href=https://europepmc.org/article/ppr/ppr560730>https://europepmc.org/article/ppr/ppr560730</a>]</p>
<p>The BsaI recognition sequence is <code>GGTCTC</code> and the BsmBI
recognition sequence is <code>CGTCTC</code>, so both of them are 6 bases
long. So the total length of the 9 recognition sites is <code>9*6</code>
which is 54 bases.</p>
<p>So the Fisher's test can be calculated like this:</p>
<pre>recognition_silent=12
recognition_total=54
other_silent=882
genome_length=29903

m=c(recognition_silent,recognition_total-recognition_silent)
m=c(m,other_silent,genome_length-other_silent-recognition_total)
fisher.test(matrix(m,2))$p
# 5.120702e-08</pre>
<p>However Guy Gadboit showed here why Washburne's logic of calculating
the p-values and odds ratios was wrong:
<a href=https://x.com/gadboit/status/1833191562961887679.><a
href="https://x.com/gadboit/status/1833191562961887679"
class="uri">https://x.com/gadboit/status/1833191562961887679</a>.</a>
After Gadboit corrected the calculation, he got the odds ratios to be
centered around 1 and not around 2 like in Washburne's code. See also
these threads:
<a href=https://x.com/gadboit/status/1748538944717750619>https://x.com/gadboit/status/1748538944717750619</a>,
<a href=https://x.com/gadboit/status/1702271854671462655.><a
href="https://x.com/gadboit/status/1702271854671462655"
class="uri">https://x.com/gadboit/status/1702271854671462655</a>.</a></p>
<p>There's 5 BsaI/BsmBI recognition sites in Wuhan-Hu-1 and 6 in RaTG13,
but 2 of them are included in both so there's 9 distinct sites:</p>
<pre>$ curl &#39;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&amp;rettype=fasta&amp;id=MN908947&#39;&gt;sars2.fa
$ seqkit locate -rip&#39;ggtctc|cgtctc&#39; sars2.fa|cut -f4-5,7|column -t
strand  start  matched
-       24102  GGTCTC
-       17972  GGTCTC # included in both
-       17329  CGTCTC # included in both
-       9751   CGTCTC
-       2193   CGTCTC
$ seqkit locate -rip&#39;ggtctc|cgtctc&#39; ratg13.fa|cut -f4-5,7|column -t
strand  start  matched
+       11645  ggtctc
+       22919  cgtctc
+       24494  cgtctc
-       17969  ggtctc # included in both
-       17326  cgtctc # included in both
-       10441  ggtctc</pre>
<p>People often say that the BsmBI and BsaI sites are ideally spaced for
a genetically engineered virus. But the output above shows that the two
sites that are shared by both RaTG13 and Wuhan-Hu-1 start at positions
17326 and 17969 in RaTG13. So why would two such nearby sites be kept?
But on the other hand the site 24494 would have been removed and 24102
would have been inserted in its place. And 10441 would be removed but
9751 would be inserted in its place.</p>
<p>In the scenario where a restriction map was built for Wuhan-Hu-1
using RaTG13 as a starting point, the engineers of the virus might have
introduced silent mutations to eliminate unwanted recognition sites and
to introduce wanted recognition sites, which is why Washburne counted
the number of recognition sites in both the source and target sequence.
His paper said: <span class="quote">"In 2013, researchers constructed a
recombinant MERS coronavirus (Scobey et al. 2013). The wildtype virus
had a few BglI sites at inconvenient locations, making it poorly
amenable to efficient assembly. To construct an idealized MERS-CoV
reverse genetic system for IVGA, the researchers removed the existing
BglI sites and inserted 6 more evenly spaced BglI sites. All
additions/removals were done via synonymous mutations, creating 7
fragments, the longest of which was 5721bp, or 19% the length of the
30kb MERS genome (Fig 2A)."</span></p>
<p>I don't know why the engineers would need to use both BsaI and BsmBI
and not just one of them, like in the case of the reverse genetics
system for MERS where only BglI was used. The draft version of the
DEFUSE proposal mentioned an order for BsmBI (R0580S and R0580L) but not
BsaI:
[<a href=https://x.com/NestCommander/status/1783112388000256308>https://x.com/NestCommander/status/1783112388000256308</a>]</p>
<div>
<img src=i/fleming-defuse-bsai.jpg width=500>
</div>
<p>If only BsmBI was used, it would produce only 4 fragments where the
longest fragment would be about 12,000 bases long:</p>
<pre>$ seqkit locate -rip&#39;cgtctc&#39; sars2.fa|cut -f4-5,7|column -t
strand  start  matched
-       17329  CGTCTC
-       9751   CGTCTC
-       2193   CGTCTC</pre>
<p>Washburne's paper says that <span class="quote">"the BsaI sites in
SARS-COV-2 flank the S1 gene and S1/S2 junction, and a similar design
has been used before for substitutions in this region"</span>. However
one BsaI site is about 500 away from the S1/S2 junction and the other
site is about 3600 bases away from the start of S1:</p>
<pre>$ seqkit locate -rip&#39;ggtctc,...cggcgg...&#39; sars2.fa|cut -f4-5,7|column -t
strand  start  matched
+       23603  CCTCGGCGGGCA # FCS
-       24102  GGTCTC # BsaI site (about 500 bases away from S1/S2 junction)
-       17972  GGTCTC # BsaI site (about 3600 bases away from start of S1)</pre>
<p>Restriction enzymes are not even needed to make synthetic
coronaviruses. Stuart Neil wrote: <span class="quote">"With all this
talk about restriction enzyme smoking guns, perhaps it's time to admit
(oh the shame) that my lab can make coronavirus genomes and mutants
thereof without using yeast, bacteria or restriction enzymes. And we can
do it over night and rescue the virus in a week."</span>
[<a href=https://twitter.com/stuartjdneil/status/1748367338590593116>https://twitter.com/stuartjdneil/status/1748367338590593116</a>]
And another Twitter user wrote: <span class="quote">"I hope I'm not
being irresponsible when I say that restriction enzymes have been passe
for at least a decade now. We reverse engineer viruses weekly using
Gibson assembly (fancy PCR). Restriction sites? Pish tosh!!!"</span>
[<a href=https://x.com/ruby_marchant/status/1748452331887452635>https://x.com/ruby_marchant/status/1748452331887452635</a>]
And Billy Bostickson wrote: <span class="quote">"Cloning of full genome
cDNA in a plasmid is bypassed using the ISA method relying on ex vivo
recombination &amp; transcription of viral RNA. Thus, no restriction
sites or other genomic modifications are required.
<a href=https://embopress.org/doi/full/10.15252/embr.202153820>https://embopress.org/doi/full/10.15252/embr.202153820</a>"</span>
<a href=https://x.com/BillyBostickson/status/1748632049010053569>https://x.com/BillyBostickson/status/1748632049010053569</a>]</p>
<h2>
<a href=#Crimes_Against_Humanity_Tour id=Crimes_Against_Humanity_Tour>Crimes
Against Humanity Tour</a>
</h2>
<h3>
<a href=#Matches_to_Corman_Drostens_primers_in_SHC014_MA15_and_Rs3367 id=Matches_to_Corman_Drostens_primers_in_SHC014_MA15_and_Rs3367>Matches
to Corman-Drosten's primers in SHC014, MA15, and Rs3367</a>
</h3>
<p>Fleming made this image which showed that Corman-Drosten's RdRp
primers matched MA15 and the SARS1-like bat viruses SHC014 and Rs3367:
[<a href=https://www.flemingmethod.com/gain-of-function>https://www.flemingmethod.com/gain-of-function</a>]</p>
<div>
<img src=i/fleming-primers-1.jpg width=1300>
</div>
<p>Corman-Drosten's RdRp primer set was designed to match both
SARS-CoV-2 and other sarbecoviruses including SARS-CoV. It has two
different probes, where the P2 probe is designed to be specific to
SARS-CoV-2 but the P1 probe is a so-called <span
class="quote">"pan-sarbecovirus"</span> probe that is also designed to
match other SARS-like viruses. So it's no surprise that the RdRp primers
actually match the MA15 strain of SARS1 and SHC014 and Rs3367 which are
SARS1-like bat viruses, because that's what they were designed to
do.</p>
<p>The paper about the Corman-Drosten protocol includes the figure below
which shows that their primers match various SARS-like viruses like
ZC45, SARS1, and BM48-31 (even though the E gene primers and probe each
have a single mismatch to BM48-31, and the N gene primers and probe have
even more mismatches to BM48-31, so the N gene primers and probe might
have too many mismatches to yield a positive result for BM48-31):
[<a href=https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2020.25.3.2000045>https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2020.25.3.2000045</a>]</p>
<div>
<img src=i/fleming-primers-2.png width=600>
</div>
<p>I also don't understand why Fleming claims that MA15 is a <span
class="quote">"chimera"</span>, because a chimeric virus has to be a
combination of two or more viruses. The paper about MA15 said: <span
class="quote">"We adapted the SARS-CoV (Urbani strain) by serial passage
in the respiratory tract of young BALB/c mice. Fifteen passages resulted
in a virus (MA15) that is lethal for mice following intranasal
inoculation."</span>
[<a href=https://pubmed.ncbi.nlm.nih.gov/17222058/>https://pubmed.ncbi.nlm.nih.gov/17222058/</a>]
There's a large number of different MA15 sequences at GenBank, but
Fleming's screenshot showed the MA15 isolate d30m5, which has only 23
nucleotide changes from the SARS-CoV Urbani sequence at GenBank:</p>
<pre>$ curl &#39;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&amp;rettype=fasta&amp;id=AY278741.1,JF292920.1&#39;|mafft --thread 4 --quiet -&gt;urbanima15.fa
$ seqkit seq -s urbanima15.fa|awk &#39;NR==1{split($0,a,&quot;&quot;);next}{split($0,b,&quot;&quot;);d=0;for(i=1;i&lt;=length;i++)d+=a[i]!=&quot;-&quot;&amp;&amp;b[i]!=&quot;-&quot;&amp;&amp;a[i]!=b[i];print d}&#39;
23</pre>
<p>SHC014 and Rs3367 are also not publicly known as either chimeric
viruses or gain-of-function viruses (even though the many recombination
events around the NTD and RBM in SARS1-like viruses are of course
suspicious, and it's also strange how Rs3367 is otherwise nearly
identical to WIV1 but Rs3367 is missing ORFx but WIV1 is not). Fleming
may have confused SHC014 with the chimeric virus described in Menachery
et al. 2015, where the spike protein of SHC014 was inserted to an MA15
backbone. In another version of his graphic I showed above he has in
fact inserted the SHC014-MA15 chimera in place of the regular SHC014.
However even in the SHC014-MA15 chimera the RdRp comes from SARS1, so
it's not unusual that the RdRp would match Corman-Drosten's primers
which were designed to match SARS1.</p>
<p>SHC014 and Rs3367 both have about 95-96% nucleotide identity to
SARS1. And in both of them the region of the RdRp that was shown in
Fleming's image is 100% identical to the Tor2 reference genome of
SARS1:</p>
<pre>$ curl -sL sars2.net/f/sarbe.fa.xz|xz -dc&gt;sarbe.fa
$ seqkit grep -nrpTor2,RsSHC014,Rs3367 sarbe.fa|seqkit seq -g|seqkit locate -ip attaagtgagatggtcatgtgtggcggctcactatatgttaaaccaggtggaacatcatccggtgatgctacaactgcttatgctaatagtgtctttaacatttgtcaag|cut -f1,4-6|column -t
seqID        strand  start  end
KC881006.1   +       15356  15465
KC881005.1   +       15356  15465
NC_004718.3  +       15356  15465</pre>
<p>In my FASTA file of 529 sarbecovirus sequences, Corman-Drosten's RdRp
forward primer has an exact match to 482 sequences:</p>
<pre>$ seqkit seq -g sarbe.fa|seqkit stat
file  format  type  num_seqs     sum_len  min_len   avg_len  max_len
-     FASTA   DNA        529  15,673,697   25,965  29,628.9   30,600
$ seqkit seq -g sarbe.fa|seqkit locate -dip GTGARATGGTCATGTGTGGCGG|sed 1d|wc -l
482</pre>
<p>And the RdRp reverse primer has an exact match to 444 out of 529
sequences in my file:</p>
<pre>$ seqkit seq -g sarbe.fa|seqkit locate -dip CARATGTTAAASACACTATTAGCATA|sed 1d|wc -l
444</pre>
<p>Fleming wrote:</p>
<blockquote>
<p>By Doing This You Will Find That
<a href=https://www.flemingmethod.com/_files/ugd/659775_60bcca8e5b5a4dcba167c56328434978.pdf>SARS-CoV-2
Wuhan Hu-1</a> Matches Three Gain-of-Function (GoF) Viruses Built by
Baric and Zhengli Using NIAID and Other Federal Money.<br> The Three GoF
Viruses are:<br>
<a href=https://www.flemingmethod.com/_files/ugd/659775_aa76331579184c83a4271ebccf9bbefe.pdf>CoV-RsSHC014</a><br>
(aka.
<a href=https://www.flemingmethod.com/_files/ugd/659775_203bd111f01844319ba86ae520e8d07e.pdf>SARS-CoV-Urbani
2003</a> &amp;
<a href=https://www.flemingmethod.com/_files/ugd/659775_026c11f0cef74b12844cdd832186611c.pdf>2016</a>)<br>
<a href=https://www.flemingmethod.com/_files/ugd/659775_7f8a6ee38f9c4ff085d74cc8c6138f1c.pdf>SARS-CoV-MA15</a><br>
<a href=https://www.flemingmethod.com/_files/ugd/659775_3f4d64c2c5b846d9986122bda564da49.pdf>SARS-CoV-Rs3367</a></p>
</blockquote>
<p>Fleming also made an image which said: <span class="quote">"Matching
PCR Primers for SARS-CoV-2 Wuhan-Hu-1 Using Snap Gene Analysis Reveal
Matches with SARS-CoV-Rs3367, CoV-RsSHC014 (Urbani) &amp;
SARS-CoV-MA15."</span>
[<a href=https://x.com/Doctor_I_am_The/status/1886410702937641471>https://x.com/Doctor_I_am_The/status/1886410702937641471</a>]
I believe he suggested that SHC014 was the same virus as SARS-CoV
Urbani. He may have been confused by how at GenBank Baric's MA15-SHC014
chimera is called <span
class="quote">"SARS-Urbani-MA_SHC014-spike"</span>.
[<a href=https://www.ncbi.nlm.nih.gov/nuccore/MT308984.1>https://www.ncbi.nlm.nih.gov/nuccore/MT308984.1</a>]
But MA15 is a mouse-adapted strain of SARS-CoV which is derived from the
Urbani strain, and in Baric's chimera the spike protein of the
SARS1-like bat virus SHC014 was inserted to MA15.</p>
<p>I also don't understand what evidence Fleming has that specifically
Rs3367 was <span class="quote">"built by Baric and Zhengli"</span> like
he wrote. It's one of about 50 published SARS1-like bat viruses that
have around 5-6% nucleotide distance to SARS1. I do consider some of the
SARS1-like bat viruses to have a laboratory origin, and it's suspicious
how Rs3367 is otherwise nearly identical to WIV1 except Rs3367 is
missing ORFx but WIV1 is not, and there are similarly other branches of
SARS1-like viruses where ORFx is included in some members of the branch
but not others, which is reminiscent of how an ACE2-binding RBM seems to
have mysteriously emerged in some closely related SARS1-like viruses but
not others.
[<a href=https://academic.oup.com/view-large/figure/232195290/veab007f2.tif>https://academic.oup.com/view-large/figure/232195290/veab007f2.tif</a>]
But I haven't seen Fleming explain why he has specifically focused on
Rs3367.</p>
<p>Fleming also wrote:</p>
<blockquote>
<p>All of which raises an interesting question.</p>
<p>We have been thinking about SARS- CoV-2 as a single virus because we
have been told it's a single virus; but</p>
<p>What if what we have been calling SARS-CoV-2 isn't a single
Gain-of-Function (GoF) Corona Virus leaked out of the Wuhan Lab?</p>
<p>What if it's MORE THAN ONE? Why should we believe that only one GoF
virus leaked?</p>
</blockquote>
<p>However if other sarbecoviruses would've been in widespread
circulation then they would've been detected by metagenomic sequencing,
where all genetic material in a sample is sequenced without having to
rely on primers to amplify any targeted organism:
<a href=nopandemic.html#Was_SARS_CoV_2_circulating_in_humans_before_2020_but_not_detected_in_the_same_way_that_HKU1_and_NL63_were_only_discovered_after_SARS1>nopandemic.html#Was_SARS_CoV_2_circulating_in_humans_before_2020_but_not_detected_in_the_same_way_that_HKU1_and_NL63_were_only_discovered_after_SARS1</a>.
The NCBI provides cloud services that make it possible to search the
Sequence Read Archive for sequencing runs with reads that have a k-mer
hit for a specific virus like SARS-CoV. A lot of people including me
have searched through sequencing runs trying to look for unusual
SARS-like viruses, and I have downloaded thousands of SRA runs with
k-mer hits for sarbecoviruse and I have aligned the reads of the runs
against SARS-like viruses, but I haven't found evidence that any other
sarbecovirus besides SARS-CoV-2 would've circulated in humans since
2020. In 2021 Chinese researchers found French influenza sequencing runs
from 2007-2012 that were contaminated with the lab-created ExoN1 and
wtic strains of SARS1 (so the runs were probably either contaminated in
a laboratory or there was an outbreak of SARS1 in France due a
laboratory leak).
[<a href=https://www.sciencedirect.com/science/article/pii/S2590053621001075>https://www.sciencedirect.com/science/article/pii/S2590053621001075</a>]
But if SARS1-like viruses would've been in any kind of widespread
circulation since 2020, then people would've found reads of SARS1 at the
SRA by using similar methodology that the Chinese researchers used to
find the French influenza A samples that were contaminated with SARS1.
And in general a huge amount of resources was directed to researching
sarbecoviruses during COVID, so if other sarbecoviruses woudl've been in
any kind of widespread circulation in humans, they would have certainly
been detected.</p>
<p>Fleming also posted a tweet that said: <span class="quote">"People
did not die because the PCR test wasn't conducted according the the
patent. They weren't maimed because of failure to conduct the PCR test
pursuant to the patent. They died because of SARS-CoV-Rs3367,
CoV-RsSHC014 and SARS-CoV-MA15."</span>
[<a href=https://x.com/Doctor_I_am_The/status/1843334021629030776>https://x.com/Doctor_I_am_The/status/1843334021629030776</a>]
But I don't know how he was confident enough that specifically Rs3367,
SHC014, and MA15 were circulating in humans that he was willing to state
it as a fact that people died because of those specific viruses. Apart
from SHC014 and Rs3367, there are approximately 50 other SARS1-like
viruses published at GenBank that have around 95% identity to SARS1, so
how did Fleming know it wasn't one of them circulating in humans
instead?</p>
<h3>
<a href=#Chromosomes_of_viruses id=Chromosomes_of_viruses>Chromosomes of
viruses</a>
</h3>
<p>Fleming said: <span class="quote">"In 2001, Ralph Baric at the
University of North Carolina gets a patent. It was paid for by National
Institutes of Health Research. And its specific goal was organizing
<span class="red">chromosomes of viruses</span>. He liked transmissible
gastroenteritis virus at the time - it was what he was working on -
infectious bronchitis, and coronaviruses."</span>
[<a href=https://www.flemingmethod.com/gain-of-function>https://www.flemingmethod.com/gain-of-function</a>,
presentation in Phoenix, time 15:27]</p>
<p>However viruses obviously don't have chromosomes (even though the
genomes of segmented viruses like influenza viruses are split into
multiple segments akin to chromosomes in eukaryotes and prokaryotes).
Fleming might have been confused by the title of the patent which was
<span class="quote">"Directional assembly of large viral genomes and
chromosomes"</span>, even though it obviously meant <span
class="quote">"large viral genomes"</span> and <span
class="quote">"chromosomes of other organisms"</span>.
[<a href=https://patents.google.com/patent/US4683195A/en>https://patents.google.com/patent/US4683195A/en</a>]</p>
<p>The authors of the patent wrote that their method of assembly can
also be applied to chromosomes of bacteria, plants, and animals: <span
class="quote">"The genomes of viruses, bacteria, plants and other
organisms (including humans) are being systematically cloned and
sequenced. Methods are needed to directionally assemble smaller DNA
subclones into full-length, functionally intact genomes or chromosomes
of these organisms. Such methods could allow for the precise genetic
manipulation of individual chromosomes in whole plants and animals and
the construction of artificial chromosomes for gene therapy."</span></p>
<h3>
<a href=#Description_of_genetic_sequencing_and_PCR_testing id=Description_of_genetic_sequencing_and_PCR_testing>Description
of genetic sequencing and PCR testing</a>
</h3>
<p>Fleming said: <span class="quote">"But once you know you have a new
virus, what do you do? Well you have to get cell tissue out, like those
bronchoalveolar lavages. You clean up the garbage, you get rid of the
other material, you get rid of the bacteria, all the mucus, all the
garbage that's in there, and you're left with genetic code
sequences."</span>
[<a href=https://www.flemingmethod.com/gain-of-function>https://www.flemingmethod.com/gain-of-function</a>,
part 2 of presentation in Tampa, time 11:09] However in the paper by Wu
et al. which described the first published genome of SARS-CoV-2, the
authors did metagenomic sequencing where they sequenced all genetic
material in a sample of bronchoalveolar fluid but they didn't remove
human or bacterial genetic material. In their STAT results only about
0.2% of their reads are identified as viral.
[<a href=https://trace.ncbi.nlm.nih.gov/Traces/?view=run_browser&page_size=10&acc=SRR10971381&display=analysis>https://trace.ncbi.nlm.nih.gov/Traces/?view=run_browser&amp;page_size=10&amp;acc=SRR10971381&amp;display=analysis</a>]
About half of their reads consist of only <code>N</code> bases because
human reads were masked with <code>N</code> bases for privacy reasons.
Supplementary Table 1 of the paper shows that 49 out of 50 of the most
abundant MEGAHIT contigs consisted of fragments of the genomes of
bacteria.
[<a href=https://www.nature.com/articles/s41586-020-2008-3#Sec13>https://www.nature.com/articles/s41586-020-2008-3#Sec13</a>]</p>
<p>Next Fleming said: <span class="quote">"Now frequently it's broken
into pieces and you have to put it together like a puzzle. But you can
do that just like a puzzle, and that's called Sanger sequencing."</span>
But the procedure of assembling the reads like a puzzle is not called
Sanger sequencing but it's called genome assembly. Wu et al. used
next-generation sequencing and not Sanger sequencing. Sanger sequencing
is used only rarely to sequence SARS-CoV-2. The NCBI's Sequence Read
Archive has raw reads of about 7 million sequencing runs where the
organism is listed as SARS-CoV-2, but I haven't found a single SRA run
where Sanger sequencing was used to sequence the entire genome, but I
have only found runs where fragments of the virus like a part of the
spike protein were sequenced with Sanger sequencing, and Sanger
sequencing is sometimes also used to sequence amplicons produced by PCR
tests.
[<a href=https://www.ncbi.nlm.nih.gov/sra/?term=sars-cov-2+%22sanger+sequencing%22>https://www.ncbi.nlm.nih.gov/sra/?term=sars-cov-2+%22sanger+sequencing%22</a>]</p>
<p>Next Fleming described PCR testing like this: <span
class="quote">"And so the spike protein was chosen. And you already saw
that we know where the spike protein is in this sequence. And so you
look for the beginning of it, and you find about 12 to 18 pieces - amino
acids - rather nucleotide bases - that start the spike protein, and
that's called the forward primer. And you look for the end of it and you
say, well, where's the end? And you look for a roughly 15 to 18 of
those, and that's called a reverse primer, which reads the opposite
direction."</span> But the spike protein of Wuhan-Hu-1 is 3,822 bases
long if you include the stop codon, but typically in PCR tests for
SARS-CoV-2 the distance between the forward and reverse primers is only
about 100 to 200 bases. The forward and reverse primers don't have to be
located at opposite ends of the same gene or even within the same gene.
Also PCR primers are almost always longer than 12 bases. In PCR tests
for SARS-CoV-2, the length of the primers ranges from about 18 to 26
bases.</p>
<h2>
<a href=#Book_Is_COVID_19_a_Bioweapon id=Book_Is_COVID_19_a_Bioweapon>Book
Is COVID-19 a Bioweapon</a>
</h2>
<p>This section deals with Fleming's book <span class="quote">"Is
COVID-19 a Bioweapon?: A Scientific and Forensic Investigation"</span>:
<a href=http://libgen.li/index.php?req=richard+fleming+bioweapon.><a
href="http://libgen.li/index.php?req=richard+fleming+bioweapon"
class="uri">http://libgen.li/index.php?req=richard+fleming+bioweapon</a>.</a></p>
<h3>
<a href=#Chimeric_sequence_from_2006_with_a_segment_labeled_SARS_CoV2 id=Chimeric_sequence_from_2006_with_a_segment_labeled_SARS_CoV2>Chimeric
sequence from 2006 with a segment labeled SARS-CoV2</a>
</h3>
<p>Fleming wrote:</p>
<blockquote>
<p>In 2006, using chimeric (Gain-of-Function) research, Chinese
scientists reported their ability to combine parts of four different
viruses into a single viral
genome.[<a href=https://pmc.ncbi.nlm.nih.gov/articles/PMC7542541/>16</a>]
This report raises a few serious questions in my mind.</p>
<p>First, why were these researchers combining parts of four dangerous
viruses - specifically, hepatitis C virus (HCV), human immunodeficiency
virus -1 (HIV-1), SARS-CoV-1 (identified as SARS-CoV-1 and not SARSCoV),
and SARS-CoV-2?</p>
<p>Second, if as we've been told, SARS-CoV-2 didn't appear until 2019
and there were no identified naturally occurring SARS-CoV-2 reported
between this 2006 publication and 2019, then doesn't this at least in
part suggest that SARS-CoV-2 is not naturally occurring but
man-made?</p>
<p>Third, if the answer to question number two is that the virus is
manmade, going as far back as 2006, then doesn't this add credence to
those who have cautioned that SARS-CoV-1 was a bioweapon and SARS-CoV-2
is an upgraded version of that bioweapon?</p>
</blockquote>
<p>However in the paper he referred to, the authors used <span
class="quote">"SARS-CoV1"</span> and <span
class="quote">"SARS-CoV2"</span> as names of two segments of SARS-CoV
they inserted into a 1,200-base chimeric RNA sequence. The paper said:
<span class="quote">"Because most commercial armored RNA preparations
contain exogenous sequences of &lt;500 nucleotides, separate armored RNA
species are often prepared for calibration of each target in multiple
virus assays. To reduce costs and simplify multivirus detection, we are
seeking to produce a single chimeric armored RNA species that might be
used as a positive control for multiple viral targets. We consider this
task to be feasible because the inventors of armored RNA predicted that,
theoretically, at least 2 kb of nonbacteriophage RNA sequence might be
encapsulated. As proof of this principle, we tried to directly package a
1200-nucleotide-long foreign RNA sequence containing gene fragments of
hepatitis C virus (HCV), HIV-1, severe acute respiratory syndrome
coronavirus 1 (SARS-CoV1), and SARS-CoV2 into the original armored RNA
production vector pAR-1."</span>
[<a href=https://pmc.ncbi.nlm.nih.gov/articles/PMC7542541/>https://pmc.ncbi.nlm.nih.gov/articles/PMC7542541/</a>]</p>
<p>The full chimeric sequence is shown in the supplementary document
file:</p>
<div>
<img src=i/fleming-sc2.png width=500>
</div>
<p>The formatting of the document was screwed up, but text at the top
says that the segment labeled <span class="quote">"SARS-CoV2"</span> was
190 bases long, so you can tell it refers to the segment that was
highlighted in gray near the end. It matches bases 18,153 to 18,342 of
the Tor2 reference genome of SARS-CoV:</p>
<pre>$ curl -s &#39;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&amp;rettype=fasta&amp;id=NC_004718.3&#39;&gt;sars1.fa
$ seqkit locate -pATGAATTACCAAGTCAATGGTTACCCTAATATGTTTATCACCCGCGAAGAAGCTATTCGTCACGTTCGTGCGTGGATTGGCTTTGATGTAGAGGGCTGTCATGCAACTAGAGATGCTGTGGGTACTAACCTACCTCTCCAGCTAGGATTTTCTACAGGTGTTAACTTAGTAGCTGTACCGACTGGTTATG sars1.fa|cut -f1,4-6|column -t
seqID        strand  start  end
NC_004718.3  +       18153  18342</pre>
<p>Fleming later referred to the same paper again here: <span
class="quote">"While most people believe that SARS-CoV-2 first appeared
in 2019, evidence shows the virus responsible for the InflammoThrombotic
disease known as COVID-19 was being manipulated two provinces from Wuhan
in 2006, and the work continued
forward.[<a href=https://pmc.ncbi.nlm.nih.gov/articles/PMC7542541/>34</a>]
Those initial genetic sequences are shown in the appendix."</span>
However I guess he didn't actually bother checking if the genetic
sequence in the appendix matched SARS-CoV-2 or the original
SARS-CoV.</p>
<p>There's also another paper from 2007 where the authors constructed a
2,248-base chimeric RNA sequence which included three different segments
from SARS-CoV, which they referred to as <span
class="quote">"SARS-CoV1"</span>, <span
class="quote">"SARS-CoV2"</span>, and <span
class="quote">"SARS-CoV3"</span>. The paper said: <span class=quote>"An
exogenous chimeric sequence 2,248 bp in length comprising the following
sequences was inserted into a pACYCDuet-1 plasmid (p15A-type replication
origin; Novagen): M-300 (nt 17∼373, 357 bp from avian influenza virus
matrix gene; GenBank accession no. DQ864720), SARS-CoV1 (nt 15224 to
15618, 395 bp from SARS-CoV; GenBank accession no. AY864806), SARS-CoV2
(nt 18038 to 18340, 303 bp from SARS-CoV; GenBank accession no.
AY864806), SARS-CoV3 (nt 328110 [sic; should probably be "nt
28110"</span>] to 28692, 583 bp from SARS-CoV; GenBank accession no.
AY864806), a pac site (19 bp), HCV (nt 18 to 310, 293 bp from HCV 5'UTR;
GenBank accession no. AF139594), and HA300 (nt 295 to 611, 317 bp from
H5N1 avian influenza virus; GenBank accession no. DQ864720)."
[<a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2395109/>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2395109/</a>]
The three segments of SARS-CoV were taken from a sequence for the
SARS-CoV isolate BJ202 that was submitted to GenBank in 2004.
[<a href=https://www.ncbi.nlm.nih.gov/nuccore/AY864806>https://www.ncbi.nlm.nih.gov/nuccore/AY864806</a>]</p>
<h3>
<a href=#Was_the_WIV_database_wiped_in_the_summer_of_2019 id=Was_the_WIV_database_wiped_in_the_summer_of_2019>Was
the WIV database wiped in the summer of 2019?</a>
</h3>
<p>Fleming wrote: <span class="quote">"During the summer of 2019, the
Wuhan Institute of Virology genetic databank records, including its
viral genomes and research, were wiped - months before the recognition
of the emergence of SARS-CoV-2."</span></p>
<p>However DRASTIC's paper about the database said that the website of
the database became inaccessible on September 12th 2019 in an
unspecified timezone (which would usually be considered fall and not
summer). And it doesn't mean that the entire database would've been
wiped, since later the website became intermittently accessible again:
<span class="quote">"Batvirus.whiov.ac.cn had been online for a few
years, saw a version 2 released in June 2019, went inactive for a week
<a href=https://drive.google.com/file/d/181PuoBwcno2rf9HNbSEK4yuabVkLGyGc/view>during
the second half of August 19</a>, before
<a href=https://archive.is/AGtFv>becoming definitely inaccessible</a>
(out of the WIV at least) on the 12th Sep 19. It was online
intermittently after this date <a href=https://archive.is/2A1cN>from
mid-December 2019</a>, and occasionally
<a href=https://archive.is/F5FPw>until February 2020</a>, but was not
accessed from outside of the WIV after 12 September 2019."</span>
[<a href=https://www.researchgate.net/publication/349073738_An_investigation_into_the_WIV_databases_that_were_taken_offline>https://www.researchgate.net/publication/349073738_An_investigation_into_the_WIV_databases_that_were_taken_offline</a>]</p>
<h3>
<a href=#Claim_that_Mullis_did_not_use_more_than_15_to_20_cycles_with_PCR id=Claim_that_Mullis_did_not_use_more_than_15_to_20_cycles_with_PCR>Claim
that Mullis did not use more than 15 to 20 cycles with PCR</a>
</h3>
<p>Fleming wrote:</p>
<blockquote>
<p>Patent number 4,683,195 was granted to Mullis and others for <span
class="quote">"a process for detecting the presence or absence of at
least one specific nucleic acid sequence in a sample containing a
nucleic acid or mixture of nucleic acids."</span></p>
<p>A clear review of this patent shows that Mullis did not exceed
fifteen to twenty cycles of PCR for the identification of genetic
material.</p>
<div>
<img src=i/fleming-mullis.png width=400>
</div>
</blockquote>
<p>However the table appears to simply demonstrate how the number of
copies produced by PCR doubles each cycle, so it's not an indication of
how many PCR cycles Mullis used himself during various workflows.</p>
<p>Right before the table Fleming showed, the patent by Mullis said
<span class="quote">"If the sample is human genomic DNA, preferably the
number of cycles is from about 10-30."</span>
[<a href=https://patents.google.com/patent/US4683195A/en>https://patents.google.com/patent/US4683195A/en</a>]</p>
<p>A paper from 1988 where Mullis was the corresponding author said they
used 60 cycles during one workflow: <span class="quote">"Fifteen 1-μg
samples of the 10<sup>-6</sup> dilution of MOLT4 DNA in GM2064 DNA
prepared previously (Fig. 2) were amplified for <span class="red">60
cycles</span> with 55°C annealing, and the primers RS79 and
RS80."</span>
[<a href=http://sci-hub.ee/https://www.science.org/doi/10.1126/science.2448875>http://sci-hub.ee/https://www.science.org/doi/10.1126/science.2448875</a>,
<a href=https://x.com/TakethatCt/status/1793390427594240368>https://x.com/TakethatCt/status/1793390427594240368</a>]
And another part of the paper said: <span class="quote">"Samples of
human genomic DNA were subjected to <span class="red">20 to 35
cycles</span> of PCR ampification with optimal amounts of either Klenow
or Tag DNA polymerase and analyzed by agarose gel electrophoresis (Fig.
1A) and Southern hybridization (Fig. 1B)."</span></p>
<h3>
<a href=#Claim_that_SARS_CoV_is_unable_to_bind_human_ACE2 id=Claim_that_SARS_CoV_is_unable_to_bind_human_ACE2>Claim
that SARS-CoV is unable to bind human ACE2</a>
</h3>
<p>Fleming wrote:</p>
<blockquote>
<p>In 2010, Shi Zhengli conducted chimeric research on SARS-CoV-1 (then
called SARS-CoV), including combining HIV-pseudovirus to look at the
binding capacity of this virus with human ACE2 receptors. Their work
specifically included altering (through mutagenesis) the spike proteins
to determine how to increase the spike protein binding to the ACE2
receptor.</p>
<p>The research showed no proline-arginine-arginine-alanine (PRRA)
insert critical to the binding of the SARS-CoV-2 to ACE2 receptors on
human cells. As noted in this published research (jointly done by
Dr. Zhengli at the Wuhan Institute of Virology, researchers in
Australia, and the University of Minnesota Medical School), not only is
the spike protein from horseshoe bats unable to bind to ACE2 receptors,
but also these differences along with differences in civets highlight a
critical missing piece to the zoonotic theory of the original of
SARS-CoV-2:</p>
<blockquote>
<p>However, although the genetically related SARS-like coronavirus
(SL-CoV) has been identified in horseshoe bats of the genus Rhinolophus
[5, 8, 12, 18], <strong>its spike protein was not able to use the human
ACE2 (hACE2) protein as a receptor</strong>. Close examination of the
crystal structure of human SARS-CoV RBD complexed with hACE2 suggests
that truncations in the receptor-binding motif (RBM) region of SL-CoV
spike protein abolish its hACE2-binding ability [7, 10], and hence the
SL-CoV found recently in horseshoe bats is unlikely to be the direct
ancestor of human SARS-CoV.</p>
<p>Also, it has been shown that <strong>the human SARS-CoV spike protein
and its closely related civet SARS-CoV spike protein were not able to
use a horseshoe bat</strong> (<em>R. pearsoni</em>) ACE2 as a receptor
[13], <strong>highlighting a critical missing link in the
bat-tocivet/human transmission</strong> chain of SARSCoV. [Emphasis
added [by Fleming].]</p>
</blockquote>
</blockquote>
<p>However the paper he quoted said that other studies had found that
various SARS-like bat viruses were not able to use human ACE2 as a
receptor, and next it said that SARS-CoV was not able to use bat ACE2 as
a receptor. But the paper he quoted did not say anywhere that SARS-CoV
was not able to use human ACE2 as a receptor.
[<a href=https://pmc.ncbi.nlm.nih.gov/articles/PMC7086629/>https://pmc.ncbi.nlm.nih.gov/articles/PMC7086629/</a>]</p>
<p>ACE2 binding ability is not even determined by the FCS insert like
Fleming suggested but by the RBD, because cleavage by furin occurs at a
later step after the virus has already bound to the ACE2 receptor.
ChatGPT said that the procedure of viral entry to a human cell consists
of these steps:</p>
<blockquote>
<ol type="1">
<li><strong>Spike protein on virus</strong> binds to the <strong>ACE2
receptor</strong> on human cells via the S1 subunit.</li>
<li><strong>Conformational change</strong> occurs in the spike protein
after ACE2 binding.</li>
<li><strong>Furin cleavage</strong> of the spike protein between the S1
and S2 subunits primes the virus for entry.</li>
<li><strong>Secondary cleavage</strong> by <strong>TMPRSS2</strong> or
<strong>cathepsins</strong> activates the spike protein, exposing the
fusion peptide.</li>
<li><strong>Membrane fusion</strong> is mediated by the S2 subunit,
allowing viral RNA to enter the host cell.</li>
<li><strong>Viral RNA release</strong> into the host cell cytoplasm
initiates infection.</li>
</ol>
</blockquote>
<h3>
<a href=#Claim_that_the_S1_subunit_is_more_stable_than_the_S2_subunit id=Claim_that_the_S1_subunit_is_more_stable_than_the_S2_subunit>Claim
that the S1 subunit is more stable than the S2 subunit</a>
</h3>
<p>Fleming wrote: <span class="quote">"The furin cleavage site lies in
the stable part of the spike protein - the S1 component. All of the
mutations being seen in SARSCoV-2 are occurring in the S2
component."</span> He has elsewhere also said that S2 is a more variable
region than S1, even though in reality it's of course the other way
around.</p>
<p>In Wuhan-Hu-1 S1 is generally considered to extend from amino acid 1
to 685 and S2 is considered to extend from amino acid 686 until the end
of the spike protein. The last <code>R</code> of <code>RRAR</code> is at
position 685.</p>
<p>In NextStrain's subsample of about 5,000 sequences of SARS-CoV-2,
about 87% of all amino acid changes in the spike protein were within the
range of 1 to 685:
[<a href=https://docs.nextstrain.org/projects/ncov/en/latest/reference/remote_inputs.html>https://docs.nextstrain.org/projects/ncov/en/latest/reference/remote_inputs.html</a>]</p>
<pre>$ curl https://data.nextstrain.org/files/ncov/open/global/metadata.tsv.xz|xz -dc&gt;global.tsv
$ sed 1d global.tsv|cut -f57|tr , \\n|grep ^S|sed &#39;s/...//;s/.$//&#39;|awk &#39;{s1+=($0&lt;=685)}END{print s1/NR}&#39;
0.873935</pre>
<h3>
<a href=#Critique_of_Pradhan_by_Zhang_et_al id=Critique_of_Pradhan_by_Zhang_et_al>Critique
of Pradhan by Zhang et al.</a>
</h3>
<p>Fleming wrote this about a paper by Zhang et al. which criticized
Pradhan's paper:</p>
<blockquote>
<p>The investigators concluded that three of the four inserts are
present in other coronaviruses:</p>
<blockquote>
<p>Among the 4 <span class="quote">"insertions"</span> (ISs) of the
2019-nCoV, IS1 has only 1 residue different from the bat coronavirus,
and 3 out of 7 residues are identical with MERS-CoV. IS2 and IS3 are all
identical to the bat coronavirus. For <strong>IS4</strong>, although the
local sequence alignment by BLAST <strong>did not hit the bat
coronavirus</strong> in Table 4, it has a close evolutionary relation
with the bat coronavirus in the MSA. In particular, the first 6 residues
in the IS4 fragment <span
class="quote">"QTQTNS<strong>PRRA</strong>"</span> from 2019-nCoV are
identical to the bat CoV, while <strong>the last 4 residues, which were
absent in the bat coronavirus or SARS-CoV</strong>, have at least 50%
identity to MERS-CoV and HCoV-HKU1 34 [Emphasis added [by Fleming].]
[<a href=https://www.biorxiv.org/content/10.1101/2020.02.04.933135v1>34</a>]</p>
</blockquote>
<p>Taken together, these statements from research paid for by NIAID
(AI134678) and the National Science Foundation (DBI1564756, 1IS1901191)
- both agencies involved in the funding of Daszak, Baric, and Zhengli
Gain-of-Function research - appear to confirm both Dr. Li-Meng Yan's
assertion that SARS-CoV-1 was the first bioweapon and that SARS-CoV-2 is
the advanced version noting the PRRA segment.</p>
</blockquote>
<p>(Fleming didn't mention that by <span class="quote">"the bat
coronavirus"</span> the authors he quoted referred to RaTG13.)</p>
<p>I don't understand how Fleming concluded based on the paper he quoted
that <span class="quote">"SARS-CoV-1 was the first bioweapon"</span>. If
motifs similar to Pradhan's inserts are missing from SARS-CoV but
present in RaTG13, MERS, or HKU1, then how does that show that SARS-CoV
was artificially engineered? Was SARS-CoV engineered to remove Pradhan's
inserts?</p>
<p>I wrote about Pradhan et al. here:
<a href=pradhan.html>pradhan.html</a>. Pradhan identified the locations
of the so-called inserts by doing a pairwise alignment of SARS-CoV-2
against SARS-CoV, so you couldn't tell if the gaps in Pradhan's pairwise
alignment were insertions in SARS-CoV-2 or deletions in SARS-CoV.
Pradhan's first so-called insert actually looks more like a deletion in
SARS-CoV. When I used a more accurate method to determine the positions
of the gaps by doing a multiple sequence alignment of SARS-CoV-2 against
a diverse set of sarbecoviruses, the gaps near the second and third
so-called inserts were placed at different spots than in Pradhan's
paper, so it looked like Pradhan's second and third inserts were not
inserts at all, but the real second insert was after the motif
identified by Pradhan and the real third insert was before the motif
identified by Pradhan.</p>
<p>For example in Pradhan's pairwise alignment it looked like SARS-CoV-2
had the insert <code>YYHK</code> relative to SARS-CoV:</p>
<div>
<img src=i/fleming-pradhan-insert2.png width=300>
</div>
<p>But the next image shows that two residues after the
<code>YYHK</code>, SARS-CoV-2 has the motif <code>KSW</code> which is
shared by European and African sarbecoviruses, but after the
<code>KSW</code> SARS-CoV-2 has the insert <code>MESEFR</code> that is
missing from the European and African viruses. So the real insert rather
seems like <code>MESEFR</code> and not <code>YYHK</code> (but before
<code>YYHK</code> SARS-CoV also seems to have an insertion of the
residues <code>SK</code>):</p>
<div>
<img src=i/origin-pradhan-7.jpg width=700>
</div>
<h3>
<a href=#Furin_cleavage_motif_in_HIV_is_not_PRRA id=Furin_cleavage_motif_in_HIV_is_not_PRRA>Furin
cleavage motif in HIV is not PRRA</a>
</h3>
<p>Fleming wrote: <span class="quote">"furin (PRRA) cleavage is
responsible not only for increasing the infectivity of SARS-CoV-2 but
also for converting HIV gp160 to gp120 and gp41"</span>.</p>
<p>However the motif cleaved by furin in Wuhan-Hu-1 is <code>RRAR</code>
but the <code>P</code> is just part of the 12-base insert which added
the furin cleavage site.</p>
<p>You can look up the coordinates of features in the envelope protein
of HIV-1 from this spreadsheet:
<a href=https://www.hiv.lanl.gov/content/sequence/HIV/MAP/Env_features.xlsx.><a
href="https://www.hiv.lanl.gov/content/sequence/HIV/MAP/Env_features.xlsx"
class="uri">https://www.hiv.lanl.gov/content/sequence/HIV/MAP/Env_features.xlsx</a>.</a>
It shows that in the HXB2 reference genome of HIV-1 the furin cleavage
motif is <code>REKR</code>:</p>
<div>
<img src=i/fleming-envfeatures.png width=500>
</div>
<p>The Los Alamos National Laboratory has published an HIV subtype
reference collection which contains up to 5 different sequences from
various subtypes of HIV. I downloded envelope protein sequences from the
subtype reference here:
<a href=https://www.hiv.lanl.gov/content/sequence/NEWALIGN/align.html.><a
href="https://www.hiv.lanl.gov/content/sequence/NEWALIGN/align.html"
class="uri">https://www.hiv.lanl.gov/content/sequence/NEWALIGN/align.html</a>.</a>
I switched the alignment type to <span class="quote">"Subtype
reference"</span>, changed <span class="quote">"DNA/Protein"</span> to
<span class="quote">"Protein"</span>, and clicked <span
class="quote">"Get Alignment"</span>. In 475 out of 555 sequences the
motif cleaved by furin was <code>REKR</code>, and the next most common
motifs at the same spot of the alignment were <code>RQKR</code> or
<code>RERR</code> (even though I don't know if some other sequences had
a furin cleavage motif at a different spot of the alignment):</p>
<pre>$ wget sars2.net/f/HIV1_REF_2023_env_PRO.fasta
$ seqkit seq -s HIV1_REF_2023_env_PRO.fasta|cut -c675-678|sort|uniq -c|sort -r
    475 REKR
     17 RQKR
     14 RERR
      9 RGKR
      8 RDKR
      7 KEKR
      5 RSKR
      4 RKKR
      3 RSRR
      3 IEKR
      2 RAKR
      1 RNKR
      1 RHKR
      1 RGER
      1 REK*
      1 KKKK
      1 GEKR
      1 -EKR
      1 ----</pre>
<h3>
<a href=#Perez_and_Montagniers_paper_about_BLAST_matches_to_HIV id=Perez_and_Montagniers_paper_about_BLAST_matches_to_HIV>Perez
and Montagnier's paper about BLAST matches to HIV</a>
</h3>
<p>Fleming wrote:</p>
<blockquote>
<p>Finally, we turn our attention to Luc Montagnier - the discoverer of
the Human Immunodeficiency Virus (HIV). Montagnier has published one
paper and has submitted another for consideration. In both of these
papers, Montagnier utilizes the same BLAST technology for analysis of
the genetic code of SARS-CoV-2.</p>
<p>He notes eighteen RNA fragments similar to HIV or simian (higher
primates) that have the potential to change the genetic expression of
COVID-19:</p>
<blockquote>
<p>18 RNA fragments of homology equal or more than 80% with human or
simian retroviruses have been found in the COVID-19 genome. These
fragments are 18 to 30 nucleotides long and therefore have the potential
to modify the gene expression of Covid19. We have named them external
Informative Elements or EIE. These EIE are not dispersed randomly, but
are concentrated in a small part of the genome.</p>
</blockquote>
<p>This is shown schematically in the figure found on page XXX in the
insert.</p>
<p>As stated so eloquently by Montagnier, <span class="red">the spike
protein not only has the PRRA insertion (twelve nucleotide bases) but
also a 1770 nucleotide base (590 amino acid) insertion matching
HIV-1</span>:</p>
<blockquote>
<p>We have studied the most recent genetic evolution of the COVID-19
strains involved in the world epidemic. We found a significant
occurrence of mutations and deletions in the 225 bases area.</p>
<p>On sampling genomes, we show that this 225 bases key region of each
genome, rich in EIE, and the 1770 bases SPIKE region evolve much faster
than the corresponding whole genome (cases of 44 patients' genomes from
WA Seattle state, original epicenter in USA).</p>
<p>In the comparative analysis of both SPIKES genes of COVID-19 and Bat
RaTG13 we note two abnormal facts:</p>
<ol type="1">
<li>The insertion of 4 contiguous PRRA amino acids in the middle of
SPIKE (we show that this site was already an optimal cleavage site
BEFORE this insertion).<br></li>
<li>An abnormal distribution of synonymous codons in the second half of
SPIKE." Finally we show the insertion in this 1770 bases SPIKE region of
a significant pair of EIEs from Plasmodium Yoelii and of a possible HIV1
EIE with a crucial Spike mutation.</li>
</ol>
</blockquote>
</blockquote>
<p>Fleming seems to have completely misunderstood what Perez meant by
the 1,770-base region of the spike protein. It wasn't an HIV-like
insertion in SARS-CoV-2 that was missing from other sarbecoviruses, but
it was a region where SARS-CoV-2 had high similarity to RaTG13:
[<a href=https://www.granthaalayahpublication.org/journals/granthaalayah/article/view/IJRG20_B07_3568>https://www.granthaalayahpublication.org/journals/granthaalayah/article/view/IJRG20_B07_3568</a>]</p>
<div>
<img src=i/fleming-perez.png width=700>
</div>
<p>Pezez's 1,770-base region started after the FCS insert and it
extended to the end of the spike protein, so it was identical to the S2
subunit of the spike protein. Perez seems to have calculated the length
of the region of the wrong, because it should actually be 1,767 or 1,764
nucleotides depending on whether you include the stop codon or not, but
Perez made many similar errors in his paper where he gave the wrong
coordinates or wrong length for various sequences. But anyway, the black
rectangle here shows that there's only 3 amino acid changes in the S2
region, because there's only 3 steps up in the line for nonsynonymous
mutations:</p>
<div>
<img src=i/fleming-perez-2.png width=600>
</div>
<p>In one of his presentations Fleming said: <span class="quote">"Luc
Montagnier - who was a friend of mine - passed away on the 8th of
February, he and Jean-Claude Perez did this work to show the HIV and SIV
inserts that they found. PRRA, four proteins, each one has three
nucleotide bases. That's 12 nucleotide bases. They found
1,770-nucleotide-base inserts. That's not naturally occurring. They
occur one nucleotide base at a time."</span>
[<a href=https://www.flemingmethod.com/gain-of-function>https://www.flemingmethod.com/gain-of-function</a>,
video from Phoenix, time 43:48] But again Perez's 1,770-base region is
not any kind of an insert, but it consists of the entire S2 subunit of
the spike protein (plus 3 or 6 extra ghost bases that were likely the
due to an error Perez made in calculating the length of the S2
subunit).</p>
<p>I have shown here why Perez's paper is complete junk:
<a href=pradhan.html#May_2020_preprint_by_Perez_and_Montagnier>pradhan.html#May_2020_preprint_by_Perez_and_Montagnier</a>.
He used nucleotide BLAST instead of protein BLAST like Pradhan, but he
forgot to exclude matches that were on the wrong strand or wrong frame,
which will cause the matches to be meaningless because they don't code
for the same amino acids in HIV as in SARS-CoV-2. His Table 1 contains
15 BLAST matches to HIV or SIV in SARS-CoV-2, but 10 of the matches are
on the wrong strand and 2 of the remaining 5 matches are in the wrong
frame:</p>
<pre>sseqid    evalue nident length pident  sstrand SC2_frame HIV_frame right_strand_and_frame
JN230738  1.3    16     16     100.000 plus    0         1         no
MF373163  0.003  21     21     100.000 minus                       no
JN863831  4.7    17     18     94.444  minus                       no
KR862351  0.11   20     21     95.238  minus                       no
EU875177  0.009  28     32     87.500  minus                       no
JF267434  0.11   18     18     100.000 plus    1         1         yes
KJ131112  0.38   19     20     95.000  minus                       no
AF003044  0.38   19     20     95.000  plus    1         1         yes
JN091691  0.38   20     22     90.909  minus                       no
HQ644953  0.009  25     28     89.286  plus    0         0         yes
L07625    1.3    22     26     84.615  plus    2         1         no
JF811228  1.3    16     16     100.000 minus                       no
KC187066  0.009  28     32     87.500  minus                       no
GU481454  0.009  32     39     82.051  minus                       no
LM999945  0.38   25     30     83.333  minus                       no

# sseqid is accession number of HIV or SIV sequence
# evalue is number of matches expected to occur by chance for given query and database
# nident is number of identical bases within a match
# length is length of match in bases
# pident is percentage of identical bases (nident/length*100)
# sstrand indicates if the subject is on the same strand as the query
# SC2_frame indicates coding frame of the match in SARS-CoV-2
# SC2_frame indicates coding frame of the match in HIV or SIV
# right_strand_and_frame indicates if sstrand is plus and SC2_frame is same as HIV_frame</pre>
<p>(The E-value depends on the size of the BLAST database, but the
E-values above were values in a tiny BLAST database that only contained
the 15 sequences listed here. In a bigger database the E-values would be
much higher, which indicates that all matches are either so short or
they have so many errors that they are likely to occur by chance.)</p>
<p>For example the second row of my table above shows that a 21-base
segment of SARS-CoV-2 has a perfect to a Swedish HIV sample from 2017.
[<a href=https://www.ncbi.nlm.nih.gov/nuccore/MF373163>https://www.ncbi.nlm.nih.gov/nuccore/MF373163</a>]
But the matches to SARS-CoV-2 and HIV are on opposite strands so the
amino acid translations of the matches are completely different:</p>
<pre>SARS-CoV-2:                HIV:
                            &lt;--------------------
 --------------------&gt;      TACGAAGTCTACTACTGCGTA # reversed version of segment matched in SARS-CoV-2
 ATGCGTCATCATCTGAAGCAT      ATGCTTCAGATGATGACGCAT # reversed version with complemented bases
AATGCGTCATCATCTGAAGCATTT   TATGCTTCAGATGATGACGCATAT
 N  A  S  S  S  E  A  F     Y  A  S  D  D  D  A  Y</pre>
<p>Perez also claimed to have identified so-called regions A and B in
the genome of SARS-CoV-2 that had an unusually high density of BLAST
matches to HIV/SIV. However he identified the regions based on the web
GUI for BLAST, which had a lot of duplicate results for matches to the
same region of SARS-CoV-2 from similar subtypes of HIV. In order to get
a more diverse representation of different subtypes of HIV, I tried to
instead do a BLAST search of SARS-CoV-2 against sequences from the LANL
HIV subtype reference, which consists of 555 sequences from various
subtypes of HIV along with a few HIV-1-like sequences of SIV. In the
next plot you can only look at the bottom panel where I only included
matches to the plus strand (because matches to the minus strand are
meaningless because they don't code for the same amino acids), but in
the bottom panel Perez's regions A and B didn't get a particularly high
density of matches:</p>
<div>
<img src=i/pradhan-perez-8.png width=1000>
</div>
<h2>
<a href=#Collaboration_with_Slovakian_commission id=Collaboration_with_Slovakian_commission>Collaboration
with Slovakian commission</a>
</h2>
<h3>
<a href=#So_a_Pekov_and_ALLATRA id=So_a_Pekov_and_ALLATRA>Soňa Peková
and ALLATRA</a>
</h3>
<p>In March 2025 Fleming published a PDF document by Soňa Peková which
summarized her analysis of DNA contamination in Pfizer and Moderna
vaccines.
[<a href=https://www.10letters.org/CzechResearch.pdf>https://www.10letters.org/CzechResearch.pdf</a>]
In May 2025 Fleming, Peter Kotlár, and Soňa Peková published a more
detailed paper about the analysis.
[<a href=https://www.fmtvdm.com/_files/ugd/659775_4f73f98b60ae4438bd343c21f2cfc79c.pdf>https://www.fmtvdm.com/_files/ugd/659775_4f73f98b60ae4438bd343c21f2cfc79c.pdf</a>]</p>
<p>In 2020 it was reported that Soňa Peková faced up to 10 years in
prison for medical insurance fraud.
[<a href=https://archiv.hn.cz/c1-66808150-lekarka-pekova-pujde-s-janouskem-pred-soud-kvuli-machinacim-s-pojistovnami-skoda-podle-zaloby-cini-240-milionu>https://archiv.hn.cz/c1-66808150-lekarka-pekova-pujde-s-janouskem-pred-soud-kvuli-machinacim-s-pojistovnami-skoda-podle-zaloby-cini-240-milionu</a>]
She was indicted in 2022 but apparently she did not serve time in jail.
[<a href=https://ct24.ceskatelevize.cz/clanek/domaci/soud-zprostil-obzaloby-lobbistu-janouska-i-dalsi-ctyri-lidi-v-kauze-spolecnosti-chambon-23287>https://ct24.ceskatelevize.cz/clanek/domaci/soud-zprostil-obzaloby-lobbistu-janouska-i-dalsi-ctyri-lidi-v-kauze-spolecnosti-chambon-23287</a>]</p>
<p>By 2023 she had become a member of a neo-Theosophical cult called
ALLATRA.
[<a href=https://x.com/search?q=So%C5%88a+Pekov%C3%A1+allatra+until%3A2024-7-1&f=live>https://x.com/search?q=So%C5%88a+Pekov%C3%A1+allatra+until%3A2024-7-1&amp;f=live</a>]
In 2023 Peková presented genetic evidence that Slavs are the oldest
ethnic group after the fall of Atlantis.
[<a href=https://cz24.news/video-molekularni-geneticka-mudr-pekova-o-genetice-nasich-predku-slovanu-jako-nejstarsim-etniku-po-padu-atlantidy-sile-a-vyjimecnosti-slovanu/>https://cz24.news/video-molekularni-geneticka-mudr-pekova-o-genetice-nasich-predku-slovanu-jako-nejstarsim-etniku-po-padu-atlantidy-sile-a-vyjimecnosti-slovanu/</a>]
In a video in 2024 Soňa Peková talked about Pleiadians and how contact
with aliens is a win-win, and she also said that Gaia has the capacity
to reach up to 9D reality and not only 5D, but it's possible that
vaccinated people cannot move to a higher dimension.
[<a href=https://www.podtatransky-kurier.sk/galaxia/sona-pekova-gaia-ma-byt-az-9d>https://www.podtatransky-kurier.sk/galaxia/sona-pekova-gaia-ma-byt-az-9d</a>]</p>
<p>ALLATRA say that the world will end in the year 2036, they promote
theories about ancient aliens and 5D ascension, and they say that
reality is contaminated by Draconian Reptilians who have a goal of
enslaving humanity.
[<a href=https://infosecurity.sk/zahranicne/sona-pekova-preslavila-sa-dezinformaciami-o-covid-19-v-sucasnosti-ma-blizko-k-ezoterickej-sekte/>https://infosecurity.sk/zahranicne/sona-pekova-preslavila-sa-dezinformaciami-o-covid-19-v-sucasnosti-ma-blizko-k-ezoterickej-sekte/</a>]
One of the members of the cult called Zhanna is allegedly an Anunnaki
who makes TikTok videos from a space station. The primary book of
ALLATRA is presented as a dialogue between its author and the ascended
master Rigden Djappo, who is a mythical character invented by the
Russian Theosophist Nikolas Roerich.
[<a href=https://handwiki.org/wiki/Organization:AllatRa>https://handwiki.org/wiki/Organization:AllatRa</a>]</p>
<p>ALLATRA is overtly based in Ukraine, but they run a multilingual
media operation that produces conspiracy content in various languages,
and much of their content is initially released in English but then
translated to various Slavic languages. It is reminiscent of the
multilingual empires operated by the Unification Church, Falun Gong, and
Miles Guo's cult. All of them are based in New York even though they
primarily target East Asian countries, so the leaders of all three cults
moved from an East Asian country to New York to run the cult.</p>
<p>ALLATRA's international representative is Egon Cholakian, who claims
that he is an intelligence expert who studied under Kissinger, and that
he had a close working relationship with the head of the CIA while he
resided in Iran, and that he was a founding member of the OSINT
Foundation.
[<a href=https://earthsavesciencecollaborative.com>https://earthsavesciencecollaborative.com</a>]
However his bio seems to be mostly fake and I didn't find it
corroborated by outside sources. His website used to have an
AI-generated photo where he posed together with the Nobel laureate
Robert Merton, but if you zoomed in on the background, there were photos
on display where people were missing faces, and the backs of books on a
bookshelf didn't have any legible text.
[<a href=http://web.archive.org/web/20240717153504/earthsavesciencecollaborative.com/assets/slider/13.jpg?t=1>http://web.archive.org/web/20240717153504/earthsavesciencecollaborative.com/assets/slider/13.jpg?t=1</a>]</p>
<p>ALLATRA has a YouTube channel that consists of AI-generated videos
that employ the avatar of Egon Cholakian:</p>
<div>
<img src=i/fleming-allatrayoutube.webp width=863>
</div>
<p>In one of the videos Egon Cholakian said that <span
class="quote">"The claim by Russian media that I am a deep fake is so
absurd it would be laughable if it weren't part of a larger coordinated
effort to mislead you."</span>
[<a href=https://www.youtube.com/watch?v=J_EM0Dp2Cmg&start=1m46s>https://www.youtube.com/watch?v=J_EM0Dp2Cmg&amp;start=1m46s</a>]
In another video he called himself a <span class="quote">"specialist in
intelligence and disinformation"</span>.
[<a href=https://www.youtube.com/watch?v=Q2GVN5Hj16Y&t=18m30s>https://www.youtube.com/watch?v=Q2GVN5Hj16Y&amp;t=18m30s</a>]
In the same video he also said that Scientology was an <span
class="quote">"international pro-democratic organization"</span> that
was unjustly persecuted in Russia. [time 1:32:47]</p>
<p>ALLATRA claims that they're not a cult, but they made an 8-hour
documentary about how the anti-cult movement is the root of all the
world's evils, and how the anti-cult movement is the power behind
Putin's throne, and the anti-cult movement did 9/11, and the anti-cult
movement is behind school shootings.
[<a href=https://www.youtube.com/watch?v=4l4CAqW1B2g&t=21060s>https://www.youtube.com/watch?v=4l4CAqW1B2g&amp;t=21060s</a>]</p>
<p>The secular arm of ALLATRA is known as the Creative Society, which
they aim to one day make the governing body of the world. Their emblem
looks similar to the emblem of the Theosophical Society, except the head
of the snake has been replaced with an arrowhead, the star of David has
been replaced with a triangle, the ankh has been replaced with a Moebius
strip, and the swastika has been removed.
[<a href=https://www.facebook.com/CreativeSociety.en/videos>https://www.facebook.com/CreativeSociety.en/videos</a>]</p>
<p>In the same way Scientology is allied with the Nation of Islam,
ALLATRA is also allied with an African American freemasonic lodge called
the World's Masonic United Nations. The Most Puissant Sovereign Grand
Commander of the World's Masonic United Nations said that the world's
strongest foundation is the Eight Pillars of Creative Society, and he
said that <span class="quote">"we support the Creative Society project
and we will use all legal and peaceful means to implement this ideal as
broadly and quickly as possible"</span>:
[<a href=https://www.facebook.com/watch/?v=264451399321129>https://www.facebook.com/watch/?v=264451399321129</a>]</p>
<div>
<img src=i/fleming-worldmasonic.webp width=1185>
</div>
<p>Soňa Peková also wrote that human have to find peace within
themselves <span class="quote">"So that we can start building a new
society as quickly as possible, based on the principles of humanity,
solidarity, generosity, equality and belonging."</span>
[<a href=https://www.facebook.com/story.php?story_fbid=122184364922255070&id=61557652117076>https://www.facebook.com/story.php?story_fbid=122184364922255070&amp;id=61557652117076</a>]
She used the same Czech word for <span class="quote">"society"</span>
that is used in the Czech name for Creative Society. So by building a
new society, she might have referred to the Creative Society, which is
the organization that ALLATRA aims to establish as the governing body of
the world, so that the world will one day be ruled by ALLATRA.
[<a href=https://creativesociety.com/about-the-project>https://creativesociety.com/about-the-project</a>]</p>
<p>ALLATRA is also concerned with the topic of autism similar to
Scientology and NOI, except ALLATRA says that autism is caused by
microplastics and not by vaccines.
[<a href=https://www.facebook.com/CreativeSociety.en/videos/1206675507639876>https://www.facebook.com/CreativeSociety.en/videos/1206675507639876</a>]</p>
<h3>
<a href=#Statement_by_Slovak_Academy_of_Sciences id=Statement_by_Slovak_Academy_of_Sciences>Statement
by Slovak Academy of Sciences</a>
</h3>
<p>The Slovak Academy of Sciences published the following statement
about the paper by Richard Fleming, Peter Kotlár, and Soňa Peková:
[<a href=https://www.sav.sk/?lang=sk&doc=services-news&source_no=20&news_no=12787>https://www.sav.sk/?lang=sk&amp;doc=services-news&amp;source_no=20&amp;news_no=12787</a>,
translated]</p>
<blockquote>
<p>The Slovak Academy of Sciences is responding to a publication
published in the HSOA Journal of Angiology and Vascular Surgery,
co-authored by the Slovak Government's Plenipotentiary for Review of the
Resource Governance and Management Process during the COVID-19 pandemic,
Peter Kotlár.</p>
<p>The Slovak Academy of Sciences is following with interest the
attempts to re-analyze vaccines that have been analyzed many times.
However, the published publication does not adequately describe the
methods, lacks controls, and does not clearly define the input
quantities of material. The published claims are not confronted with
existing knowledge or other methods, which is necessary given the
seriousness of the findings presented.</p>
<p>The title of the publication suggests a quantitative analysis, but
the results are not compared to the prescribed and strictly controlled
quantitative limits of residual DNA, and estimated numbers of molecules
are given, which can be misleading. Strong conclusions are not supported
by sufficiently strong data.</p>
<p>The scientific shortcomings of the aforementioned publication reflect
the fact that the journal in which it was published is classified as a
predatory journal. This particular journal does not meet even basic
quality criteria, is not registered in recognized databases, has a very
short review process and unclear review procedure, and does not have
defined basic rules for presenting results and methodologies.</p>
<p>The Slovak Academy of Sciences strictly condemns publishing in
predatory journals and warns that publications published in this way do
not bring value and are of no importance to the scientific community,
medical practice, or the public.</p>
</blockquote>
<p>The paper by Fleming, Kotlár, and Pekova was published in a journal
of the predatory publisher Herald Scholarly Open Access.
[<a href=https://www.heraldopenaccess.us/openaccess/quantitative-analysis-of-nucleic-acid-content-in-spikevax-moderna-and-bnt162b2-pfizer-covid-19-vaccine-lots>https://www.heraldopenaccess.us/openaccess/quantitative-analysis-of-nucleic-acid-content-in-spikevax-moderna-and-bnt162b2-pfizer-covid-19-vaccine-lots</a>]
One of their journals published a paper by Hellsing et al. about the
benefit of garlic in fighting vampires, which said the purpose of the
paper was <span class="quote">"improving the stature of vampire hunting
among the scientifically literate populace, and testing whether or not
AIMS journals are predatory"</span>.
[<a href=https://www.heraldopenaccess.us/openaccess/allium-argenti-et-aqua-sancta-transgressing-molecular-boundaries-in-hematology-post-alucard>https://www.heraldopenaccess.us/openaccess/allium-argenti-et-aqua-sancta-transgressing-molecular-boundaries-in-hematology-post-alucard</a>]</p>
<script>
document.querySelectorAll('pre').forEach(e=>e.innerHTML=e.innerHTML.replace(/((^| )# [^\n]*|^#$)/mg,"<span class=comment>$1</span>"))

const walker=document.createTreeWalker(document.body,NodeFilter.SHOW_TEXT,null,false)
const nodes=[]
while(walker.nextNode())nodes.push(walker.currentNode)
nodes.forEach(node=>{
  const text=node.textContent
  if(text.split(/\s+/).some(piece=>piece.length>=20)||/\w{60}/.test(text)){
    const frag=document.createDocumentFragment()
    text.split(/([A-Za-z0-9]{60}|[\041-\057\072-\100\133-\140\173-\176])/g).forEach(part=>{
      if(part){
        frag.appendChild(document.createTextNode(part))
        if(/^[A-Za-z0-9]{60}$/.test(part)||/^[\041-\057\072-\100\133-\140\173-\176]$/.test(part))
          frag.appendChild(document.createElement('wbr'))
      }
    })
    node.parentNode.replaceChild(frag,node)
  }
})

document.querySelectorAll('.spoiler button').forEach(x=>{x.addEventListener('click',function(){
  let block=this.nextElementSibling
  if(window.getComputedStyle(block).display=='none'){block.style.display='block';this.textContent='Hide code'}
  else{block.style.display='none';this.textContent='Show code'}
})})
</script>
