<!doctype html>
<meta charset=utf-8>
<meta name=viewport content=width=device-width,initial-scale=1.0>
<style>
body{background:#2f2f2f;color:white;text-align:center}
p,pre,ul,li,blockquote,table,dl{text-align:left}
body,td,th{font:14.5px/1.33 Arial,Helvetica,'Liberation Sans',sans-serif}
p{max-width:500px;margin:1em auto}
dl,pre{margin:1em auto;width:fit-content;max-width:min(100%, 1000px);box-sizing:border-box}
hr{border:none;border-top: 2px solid #bbb;margin:1em auto;max-width:500px}
ul,ol{width:min(100%, 500px);margin:1em auto;box-sizing:border-box}
ul ul,ul ol,ol ol,ol ul{margin-top:0;margin-bottom:0}
h1,h2,h3,h2 a,h3 a,h2 a:visited,h3 a:visited{color:white;font-weight:normal}
h1{font-size:2.1em;margin:.7em 0 .5em 0}
h2{font-size:1.75em}
h3{font-size:1.35em}
blockquote h2,blockquote h3{margin:.5em 0}
blockquote ul,blockquote ol{margin:.4em 0}
a,a .quote{color:hsl(220,100%,75%)}
a:visited,a .quote{color:hsl(270,100%,75%)}
pre,code{white-space:pre-wrap;word-wrap:break-word;font:13.7px Menlo,'DejaVu Sans Mono','Liberation Mono',Consolas,monospace;background:#4a4a4a;color:white}
pre{padding:4px}
code{padding:1px}
table{border-collapse:collapse;margin:0 auto}
td,th{border:1px solid #555;padding:1px 3px}
th{background:#444;font-weight:bold}
img{max-width:100%;margin:0;vertical-align:middle}
div:has(img){width:fit-content;margin:0 auto}
dt{font-size:1.3em;margin:.4em 0}
body>blockquote{margin:1em auto;padding-left:45px;width:465px}
blockquote blockquote{width:calc(100% - 45px);margin-left:45px;box-sizing:border-box}
blockquote>:first-child{margin-top:0}
blockquote>:last-child{margin-bottom:0}
blockquote,.quote,em{color:hsl(100,100%,88%)}
blockquote blockquote,blockquote .quote,.quote .quote,blockquote em,.quote em,em .quote{color:hsl(100,50%,67%)}
blockquote blockquote blockquote,blockquote blockquote .quote,blockquote .quote .quote,.quote .quote .quote,blockquote blockquote em,blockquote .quote em,.quote .quote em,blockquote em .quote{color:hsl(100,38%,45%)}
blockquote blockquote blockquote blockquote,blockquote blockquote blockquote .quote,blockquote blockquote .quote .quote,blockquote blockquote blockquote em,blockquote blockquote .quote em,blockquote .quote .quote em,blockquote blockquote em .quote{color:hsl(100,28%,35%)}
blockquote h3{color:inherit}
.spoiler{max-width:1100px;margin:0 auto}
.spoiler button{width:90px;border:0;font-family:inherit;font-size:inherit;color:inherit;background:#4a4a4a;padding:4px 0px;outline:none;cursor:pointer;margin:0 0 -.3em 0}
.spoiler pre{display:none;margin-top:0;text-align:left}
.red,.red .quote{color:hsl(0,100%,77%)!important}
.comment{color:hsl(0,100%,80%)!important}
.dates{color:#ccc}
.toc{width:calc(50% + 250px);margin:1em 0 1em max(0px, 50% - 250px)}
.toc ul{width:min(100%, 50% + 250px - 40px);margin:0}
.maxwidth{max-width:100%}
.wide{width:min(100%, 1000px)}
@media print{
  body{background:white;color:black;margin:0px;zoom:.9;print-color-adjust:exact;-webkit-print-color-adjust:exact}
  a,a:visited{color:#0000EE}
  h1,h2,h2 a,h3 a,h2 a:visited,h3 a:visited{color:black}
  pre,code{background:#d6d6d6;color:black}
  th{background:#ddd}
  td,th{border:1px solid #ccc}
  blockquote,.quote{color:hsl(100,65%,28%)}
  blockquote .quote,blockquote blockquote,.quote .quote{color:hsl(100,50%,53%)}
  blockquote blockquote blockquote,blockquote .quote .quote,.quote .quote .quote{color:hsl(100,70%,72%)}
  .spoiler{display:none}
  .red,.red .quote{color:hsl(0,50%,60%)!important}
  .dates{color:#777}
}
</style>
<title>Uncanny HIV inserts of Pradhan and Perez</title>
<h1>Uncanny HIV inserts of Pradhan and Perez - <a href=index.html>sars2.net</a></h1><div class=dates>First published 2024-05-27 UTC, last modified 2025-10-24 UTC</div><h2 id=Contents><a href=#Contents>Contents</a></h2>
<ul class=toc>
<li><a href="#Pradhan_et_al_s_HIV_inserts">Pradhan et al.&#39;s HIV inserts</a></li>
<ul class=toc>
<li><a href="#Introduction">Introduction</a></li>
<li><a href="#BLAST_searches">BLAST searches</a></li>
<li><a href="#Matches_to_sequences_from_HIV_and_influenza_databases">Matches to sequences from HIV and influenza databases</a></li>
<li><a href="#First_insert">First insert</a></li>
<li><a href="#Second_insert">Second insert</a></li>
<li><a href="#Third_insert">Third insert</a></li>
<li><a href="#Unhighlighted_insert_between_second_and_third_inserts">Unhighlighted insert between second and third inserts</a></li>
<li><a href="#Fourth_insert">Fourth insert</a></li>
<li><a href="#Shell_code_for_making_colorized_amino_acid_aligments">Shell code for making colorized amino acid aligments</a></li>
<li><a href="#Arkmedics_BLAST_searches_for_Pradhan_et_al_s_inserts">Arkmedic&#39;s BLAST searches for Pradhan et al.&#39;s inserts</a></li>
<li><a href="#Matches_to_env_in_LANLs_HIV_subtype_reference">Matches to env in LANL&#39;s HIV subtype reference</a></li>
<li><a href="#E_values_of_matches_to_Pradhans_first_two_inserts">E values of matches to Pradhan&#39;s first two inserts</a></li>
<li><a href="#Are_Pradhans_inserts_involved_in_entry_to_CD4_T_helper_cells">Are Pradhan&#39;s inserts involved in entry to CD4 T helper cells?</a></li>
</ul>
<li><a href="#Perez_and_Montagniers_paper_about_BLAST_matches_to_HIV_and_SIV">Perez and Montagnier&#39;s paper about BLAST matches to HIV and SIV</a></li>
<ul class=toc>
<li><a href="#Location_of_regions_A_and_B">Location of regions A and B</a></li>
<li><a href="#Table_of_15_BLAST_matches_to_HIV_and_SIV">Table of 15 BLAST matches to HIV and SIV</a></li>
<li><a href="#Expect_values_of_Perezs_matches_at_web_BLAST">Expect values of Perez&#39;s matches at web BLAST</a></li>
<li><a href="#Version_of_Perezs_graphic_with_E_value_and_strand_added">Version of Perez&#39;s graphic with E-value and strand added</a></li>
<li><a href="#Additional_matches_to_HIV_and_SIV_outside_regions_A_and_B">Additional matches to HIV and SIV outside regions A and B</a></li>
<li><a href="#Matches_to_the_parasite_Plasmodium_yoelii">Matches to the parasite Plasmodium yoelii</a></li>
<li><a href="#Histogram_of_matches_per_block_in_LANLs_HIV_subtype_reference">Histogram of matches per block in LANL&#39;s HIV subtype reference</a></li>
<li><a href="#Tweets_by_Jean_Claude_Perez">Tweets by Jean-Claude Perez</a></li>
</ul>
</ul>


<h2>
<a href=#Pradhan_et_al_s_HIV_inserts id=Pradhan_et_al_s_HIV_inserts>Pradhan
et al.'s HIV inserts</a>
</h2>
<h3>
<a href=#Introduction id=Introduction>Introduction</a>
</h3>
<p>Pradhan et al. did a pairwise alignment of the spike protein of SARS2
against SARS1, and they identified 4 regions where SARS2 had an insert
relative to SARS1:
[<a href=https://www.researchgate.net/publication/338957445_Uncanny_similarity_of_unique_inserts_in_the_2019-nCoV_spike_protein_to_HIV-1_gp120_and_Gag>https://www.researchgate.net/publication/338957445_Uncanny_similarity_of_unique_inserts_in_the_2019-nCoV_spike_protein_to_HIV-1_gp120_and_Gag</a>]</p>
<div>
<img src=i/origin-pradhan-1.webp width=700>
</div>
<p>When Pradhan et al. did BLAST searches for the inserts and the
surrounding amino acids, they found that for example the last 6 aa of
the first 7-aa insert had a perfect match to an HIV sequence from
Thailand, and the last 2 aa of the second insert along with the next 4
aa had a perfect match to an HIV sequence from Kenya:</p>
<div>
<img src=i/origin-pradhan-2.webp width=600>
</div>
<h3>
<a href=#BLAST_searches id=BLAST_searches>BLAST searches</a>
</h3>
<p>I tried searching for the first 7-aa insert in protein BLAST:
<a href=https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp&PAGE_TYPE=BlastSearch.><a
href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp&amp;PAGE_TYPE=BlastSearch"
class="uri">https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp&amp;PAGE_TYPE=BlastSearch</a>.</a>
I entered <code>GTNGTKR</code> to the text field at the top, I set the
database to refseq_protein, I set the organism as <span
class="quote">"Viruses (taxid:10239)"</span>, and I clicked BLAST.</p>
<p>There were many different bacteriopages and a few non-phage viruses
which matched 6 out of 7 aa so that one aa at either end was clipped by
the local alignment, so that there was 100% identity but 85% query
coverage:</p>
<div>
<img src=i/origin-pradhan-3.jpg width=800>
</div>
<p>HIV-1 was not included in the results above, because I searched in a
database which only contained refseq proteins. The standard reference
genome of HIV-1 is called HXB2, and its envelope protein matches only 3
out of 7 aa of Pradhan's first insert:</p>
<pre>$ curl &#39;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&amp;rettype=fasta_cds_aa&amp;id=NC_001802&#39;&gt;hiv.aa
$ seqkit locate -p GTNGTKR -m3 &lt;(seqkit grep -nrp gene=env hiv.aa)|cut -f3,5-|column -t # no matches for max mismatch 3
pattern  start  end  matched
$ seqkit locate -p GTNGTKR -m4 &lt;(seqkit grep -nrp gene=env hiv.aa)|cut -f3,5-|column -t # 7 matches for max mismatch 4
pattern  start  end  matched
GTNGTKR  232    238  TFNGTGP
GTNGTKR  300    306  NNNTRKR
GTNGTKR  321    327  GKIGNMR
GTNGTKR  354    360  GNNKTII
GTNGTKR  404    410  GSNNTEG
GTNGTKR  495    501  GVAPTKA
GTNGTKR  822    828  VAEGTDR</pre>
<p>Next I tried doing another BLAST search for Pradhan's first insert,
but I set the <span class="quote">"Non-redundant protein sequences
(nr)"</span> database and I set the organism to HIV-1. There were 3
different spots of the envelope protein which all matched 6/7 aa of the
insert so that one aa at either end was cut off:</p>
<div>
<img src=i/origin-pradhan-4.jpg width=1200>
</div>
<p>There's more than a million sequences of HIV-1 in GenBank:
<a href=https://www.ncbi.nlm.nih.gov/nuccore/?term=%22Human+immunodeficiency+virus+1%22%5Bporgn%3A%5f%5ftxid11676%5D.><a
href="https://www.ncbi.nlm.nih.gov/nuccore/?term=%22Human+immunodeficiency+virus+1%22%5Bporgn%3A%5f%5ftxid11676%5D"
class="uri">https://www.ncbi.nlm.nih.gov/nuccore/?term=%22Human+immunodeficiency+virus+1%22%5Bporgn%3A%5f%5ftxid11676%5D</a>.</a>
I don't know how many of them are included at BLAST, but in any case
it's not that unusual that a handful of sequences would randomly happen
to match 6 out of 7 aa of Pradhan's first insert.</p>
<h3>
<a href=#Matches_to_sequences_from_HIV_and_influenza_databases id=Matches_to_sequences_from_HIV_and_influenza_databases>Matches
to sequences from HIV and influenza databases</a>
</h3>
<p>The env protein of HIV is divided into the gp120 and gp41
polyproteins, but Pradhan et al.'s first insert matched the gp120 part.
I downloaded 125,046 gp120 sequences from the HIV sequence database of
the Los Alamos National Laboratory:
<a href=https://www.hiv.lanl.gov/components/sequence/HIV/search/search.html.><a
href="https://www.hiv.lanl.gov/components/sequence/HIV/search/search.html"
class="uri">https://www.hiv.lanl.gov/components/sequence/HIV/search/search.html</a>.</a>
None of them matched the full <code>GTNGTKR</code> insert, but there
were 15 sequences that matched the first 6 aa and there were 12
sequences which matched the last 6 aa, and the matches were at 3
different spots:</p>
<pre>$ seqkit stat HIV-1_env.fasta|column -t
file             format  type     num_seqs  sum_len      min_len  avg_len  max_len
HIV-1_env.fasta  FASTA   Protein  125,046   105,184,828  479      841.2    1,469
$ seqkit locate -p GTNGTK HIV-1_env.fasta|cut -f1,5-7|column -t
seqID                                            start  end  matched
C.IN.-.VB49.EF694035                             135    140  GTNGTK
C.ZA.2009.CAP177_39mo_30.KC833437                452    457  GTNGTK
C.ZA.2009.CAP177_4260_186wpi_plasma_12.MK205618  452    457  GTNGTK
C.ZA.2009.CAP177_4260_186wpi_cvl_73.MK205633     452    457  GTNGTK
B.US.1990.1580m.11a.MT861903                     136    141  GTNGTK
B.US.1990.1580m.13s.MT861906                     136    141  GTNGTK
B.US.1990.1580m.15.MT861908                      136    141  GTNGTK
B.US.1990.1580m.16a.MT861910                     136    141  GTNGTK
B.US.1990.1580m.17.MT861911                      136    141  GTNGTK
B.US.1990.1580m.18.MT861913                      136    141  GTNGTK
B.US.1990.1580m.22.MT861919                      136    141  GTNGTK
B.US.1990.1580m.3a.MT861922                      136    141  GTNGTK
B.US.1990.1580m.5.MT861925                       136    141  GTNGTK
B.US.1990.1580m.7.MT861927                       136    141  GTNGTK
A1CD.TZ.2012.30196v23_env12.OM825465             460    465  GTNGTK
$ seqkit locate -p TNGTKR HIV-1_env.fasta|cut -f1,5-7|column -t
seqID                                start  end  matched
01_AE.TH.2006.AA042a12R.JX447199     364    369  TNGTKR
01_AE.TH.2006.AA042a13R.JX447200     399    404  TNGTKR
01_AE.TH.2006.AA042a09R.JX447201     398    403  TNGTKR
01_AE.TH.2006.AA042a08R.JX447202     398    403  TNGTKR
01_AE.TH.2006.AA042a10R.JX447203     404    409  TNGTKR
01_AE.TH.2006.AA042a11R.JX447204     404    409  TNGTKR
01_AE.TH.2006.AA042a02R.JX447205     398    403  TNGTKR
01_AE.TH.2006.AA042a05R.JX447206     398    403  TNGTKR
01_AE.TH.2006.AA042a07R.JX447207     398    403  TNGTKR
01_AE.TH.2006.AA042a03R.JX447208     399    404  TNGTKR
01_AE.TH.2006.AA042a06R.JX447209     404    409  TNGTKR
01_AE.TH.2007.AA042d_ENV24.MZ347079  408    413  TNGTKR</pre>
<p>So again it's not that unusual that 29 out of over a 100,000
sequences would happen to match 6 out of 7 aa. The reason why all four
of Pradhan's inserts happened to match HIV and not some other virus
might simply be that BLAST has a very large number of HIV sequences
relative to other viruses.</p>
<p>Pradhan's second so-called insert matches the last 2 aa of the
<code>YYHK</code> insert here and the next 4 aa:</p>
<div>
<img src=i/origin-pradhan-5.jpg width=300>
</div>
<p>But there's a total of 7 possible 6 aa segments which include at
least 2 aa of the 4 aa insert, so again it's not that surprising that
one of them would happen to match one out of hundreds of thousands or
millions of HIV sequences at BLAST. The gp120 sequences I downloaded had
a perfect match for only one out of the 7 segments:</p>
<pre>$ x=FLGVYYHKNNKS;for i in {0..6};do seqkit locate -ip ${x:i:6} HIV-1_env.fasta;done
seqID   patternName pattern strand  start   end matched
seqID   patternName pattern strand  start   end matched
seqID   patternName pattern strand  start   end matched
seqID   patternName pattern strand  start   end matched
seqID   patternName pattern strand  start   end matched
seqID   patternName pattern strand  start   end matched
seqID   patternName pattern strand  start   end matched
G.KE.2006.06KE275457V6.KT022379 HKNNKS  hknnks  +   462 467 hknnks</pre>
<p>I also tried downloading all protein sequences of influenza A from
the NCBI's influenza virus database. I simply kept the default settings
here and clicked <span class="quote">"add query"</span>:
<a href=https://www.ncbi.nlm.nih.gov/genomes/FLU/Database/nph-select.cgi#mainform.><a
href="https://www.ncbi.nlm.nih.gov/genomes/FLU/Database/nph-select.cgi#mainform"
class="uri">https://www.ncbi.nlm.nih.gov/genomes/FLU/Database/nph-select.cgi#mainform</a>.</a>
There were a bit over a million sequences:</p>
<pre>$ seqkit stat influenzamega.fa
file              format  type      num_seqs      sum_len  min_len  avg_len  max_len
influenzamega.fa  FASTA   Protein  1,234,340  497,828,616        1    403.3      775</pre>
<p>The spike protein of Wuhan-Hu-1 is 1273 aa long so it has 1268
possible 6 aa subsegments. However 269 of them had an exact match to one
or more of the protein sequences of influenza A:</p>
<pre class=maxwidth>$ curl &#39;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&amp;rettype=fasta_cds_aa&amp;id=MN908947&#39;|seqkit grep -nrp gene=S&gt;sars2spike.aa
$ seqkit seq -s sars2spike.aa|awk &#39;{for(i=1;i&lt;length-5;i++)print substr($0,i,6)}&#39;&gt;kmer
$ grep -f kmer &lt;(seqkit seq -s influenzamega.fa)&gt;temp
$ for x in `&lt;kmer`;do grep -m1 $l temp;done|wc -l
269
</pre>
<p>And in a FASTA file with 125,046 gp120 sequences of HIV, 53 out of
1268 6-mers had a perfect match (but the number would've been bigger if
I was looking at all proteins of HIV and not just gp120):</p>
<pre>$ grep -fkmer &lt;(seqkit seq -s HIV-1_env.fasta)&gt;temp2
$ for x in `&lt;kmer`;do grep -m1 $x temp2;done|wc -l
53</pre>
<h3>
<a href=#First_insert id=First_insert>First insert</a>
</h3>
<p>Pradhan's first insert was <code>GTNGTKR</code>, but Wuhan-Hu-1 only
has one or 2 inserted aa in that region relative to European and African
sarbecoviruses, so it rather looks like SARS1 has deletions in that
region. The <code>TKR</code> at the end of the first so-called insert is
also matched by ZC45 and RpYN06:</p>
<div>
<img src=i/origin-pradhan-8.jpg width=600>
</div>
<p>One of the major flaws in Pradhan's paper was that he didn't report
the E-values of the matches, which would've shown how likely similarly
close matches were to occur by chance.</p>
<p>I tried doing a BLAST search for Pradhan's first so-called insert
together with the 5 surrounding amino acids on either side, so that kept
the default nr database but I restricted the species to HIV-1:
<a href=https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE=Proteins.><a
href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE=Proteins"
class="uri">https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE=Proteins</a>.</a></p>
<div>
<img src=i/pradhan-evalue-1.webp width=600>
</div>
<p>The matches to the Thai sequences that were listed in Pradhan's paper
had an E-value of about 507, which means that about 507 similarly close
matches were expected to occur by chance:</p>
<div>
<img src=i/pradhan-evalue-2.webp width=600>
</div>
<p>There were 97 results which had a better match to the query than
Pradhan's Thai sequences. But the lowest E-value among the matches was
still 11, so none of the matches were even close to reaching the
significance level of 0.05:</p>
<div>
<img src=i/pradhan-evalue-3.webp width=800>
</div>
<p>One of the terms which is used to calculate the E-value is the length
of the database in letters, so the E-value depends linearly on the
length of the database. My database of HIV-1 sequences was about 300
million letters long. If I would've broadened the search to all viruses
except SARS-CoV-2, then the length of the database would've been about 7
times bigger, so the E-values would've also been about 7 times
higher.</p>
<h3>
<a href=#Second_insert id=Second_insert>Second insert</a>
</h3>
<p>In Pradhan's pairwise alignment SARS2 had the insert
<code>YYHK</code> relative to Tor2. Pradhan's second so-called insert
was <code>HKNNKS</code>, which consisted of the last two aa of
<code>YYHK</code> and the next 4 aa. However the <code>YYHKNNK</code>
without the <code>K</code> in the middle is also included in ZC45 and
ZXC21 (so it's weird that Pradhan didn't mention them since they were
the closest published relatives of SARS2 in January 2020):</p>
<div>
<img src=i/origin-pradhan-7.jpg width=600>
</div>
<p>Also from my alignment above it looks that relative to SARS1, SARS2
has the deletion <code>SK</code> and the insertion <code>MESEFR</code>.
Pradhan et al. identified the locations of their inserts by doing a
pairwise alignment of SARS2 and SARS1, but they would've been able to
identify the locations more accurately if they would've done a multiple
sequence alignment that also included other sarbecoviruses. Trevor
Bedford also pointed out that the <code>YYHK</code> wasn't really even
an insert relative to other sarbecoviruses, but from his alignment it
looks like SARS2 has the insertion <code>SEFR</code> relative to SARS1:
[<a href=https://x.com/trvrb/status/1223667730911354880>https://x.com/trvrb/status/1223667730911354880</a>]</p>
<div>
<img src=i/origin-pradhan-6.jpg width=600>
</div>
<h3>
<a href=#Third_insert id=Third_insert>Third insert</a>
</h3>
<p>The third insert that was highlighted in Pradhan's alignment
consisted of the amino acids <code>SSG</code>, even though a bit before
it there was another 3 aa insert that was not highlighted. But the third
insert might have been placed in the wrong spot because Pradhan et
al. only did a pairwise alignment of SARS2 and SARS1. When I did an
alignment of more viruses, I got a single 6 aa insert in SARS2 relative
to SARS1 instead of two different 3 aa inserts. And my alignment also
shows that the <code>G</code> at the end of <code>SSG</code> is also
included in ZC45 and RpYN06 (and the middle <code>S</code> in
<code>SSG</code> is even included in European and African
sarbecoviruses, even though it might be a coincidence):</p>
<div>
<img src=i/origin-pradhan-14.jpg width=600>
</div>
<h3>
<a href=#Unhighlighted_insert_between_second_and_third_inserts id=Unhighlighted_insert_between_second_and_third_inserts>Unhighlighted
insert between second and third inserts</a>
</h3>
<p>The insert between inserts 2 and 3 that wasn't highlighted by Pradhan
consisted of the amino acids <code>LHR</code>. When I tried searching
for the insert and the surrounding amino acids on BLAST, I found that
the <code>LHR</code> and the next 4 amino acids were matched by a
bacteriophage from the class <em>Caudoviricetes</em> (so it's even
better than Pradhan's first two inserts which only matched 6 amino
acids):</p>
<div>
<img src=i/origin-pradhan-16.jpg width=600>
</div>
<h3>
<a href=#Fourth_insert id=Fourth_insert>Fourth insert</a>
</h3>
<p>Pradhan's fourth insert consisted of <code>QTNS</code> followed by 11
aa with no match in SARS2 followed by <code>PRRA</code>, which was found
in a single random subtype B sequence of HIV from India. But if the
designers of SARS2 were inspired to borrow the <code>PRRA</code> motif
from HIV, how did they know to look at a random HIV sequence from India
out of hundreds of thousands of published sequences? Here the Indian
sequence is compared to the HXB2 refseq, which contains
<code>QVTNS</code> instead of <code>QTNS</code>, but which only matches
1 out of 4 aa of the <code>PRRA</code>:</p>
<pre class=maxwidth>$ curl &#39;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=protein&amp;rettype=fasta&amp;id=AKR75206,NP_057850&#39;|mafft --clustalout -
[...]
AKR75206.1  RVLAEAMSQ-TNS-SILMQRSNFKGPRRAVKCFNCGREGHIAKNCRAPRKKGCWKCGKEG
NP_057850.1 RVLAEAMSQVTNSATIMMQRGNFRNQRKIVKCFNCGKEGHTARNCRAPRKKGCWKCGKEG
            ********* *** :*:***.**:. *: *******:*** *:*****************
                    ^^^^^^^^^^^^^^^^^^^^^
                    Pradhan insert 4
[...]
</pre>
<h3>
<a href=#Shell_code_for_making_colorized_amino_acid_aligments id=Shell_code_for_making_colorized_amino_acid_aligments>Shell
code for making colorized amino acid aligments</a>
</h3>
<p>Here's code for making amino acid alignments similar to the previous
sections of this file:</p>
<pre># download whole genome sequences of sarbecoviruses
curl -sL sars2.net/f/sarbe.fa.xz|xz -dc&gt;sarbe.fa

# efetch multiple sequences with accessions from STDIN
emu()(curl -sd &quot;id=$(paste -sd,)&quot; &quot;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=${1:-nuccore}&amp;rettype=${2:-fasta}&amp;retmode=$3&quot;)

# download spike protein sequences of sarbecoviruses and rename them to have same the defline as the whole genome sequences
seqkit seq -ni sarbe.fa|emu &#39;&#39; fasta_cds_aa|seqkit grep -nrp gene=S,spike,surface|sed &#39;s/_prot_.*//;s/lcl|//&#39;|seqkit fx2tab|awk -F\\t &#39;NR==FNR{a[$1]=$2;next}{print&quot;&gt;&quot;$1&quot; &quot;a[$1]&quot;\n&quot;$2}&#39; &lt;(seqkit seq -n sarbe.fa|sed $&#39;s/ /\t/&#39;) -&gt;spike.aa

# make a TSV matrix for percentage identity between spike protein sequences
aln()(\mafft --thread 3 --quiet &quot;${@--}&quot;)
curl -Ls sars2.net/f/pid.cpp&gt;pid.cpp;g++ pid.cpp -O3 -o pid
seqkit replace -isp &#39;[^X]&#39; -r &#39;&#39; spike.aa|seqkit seq -m 5|seqkit seq -ni|seqkit grep -vf- spike.aa|aln|./pid&gt;spike.pid

# thin out a TSV percentage identity matrix by removing rows with over n% identity to any previous row (default 99%)
thin()(awk -F\\t &#39;NR&gt;1{for(i=2;i&lt;NR;i++)if($i&gt;x)next;print$1}&#39; x=&quot;${1-99}&quot; &quot;${@:2}&quot;)

# print color scheme
aacolor()(printf %s\\n A:242:121:121 C:242:182:121 D:242:242:121 G:182:242:121 F:121:242:151 E:121:242:222 T:121:182:242 I:121:121:242 K:182:121:242 L:242:121:242 M:255:191:191 N:255:223:191 P:255:255:191 Q:223:255:191 R:191:255:207 S:191:255:244 H:191:223:255 V:191:191:255 W:223:191:255 Y:255:191:255 -:140:140:140 X:255:255:255 \*:255:255:255|tr : \ )

# display alignment with colorized amino acids
seqcol()(seqkit seq -u|seqkit fx2tab|gawk -F\\t &#39;NR==FNR{a[$1]=$2;next}{name[FNR]=$1;split($2,z,&quot;&quot;);for(i in z)seq[FNR][i]=z[i];if(length($1)&gt;max1)max1=length($1);if(length($2)&gt;max2)max2=length($2)}END{for(i=1;i&lt;=max2;i+=width){printf(&quot;%&quot;(max1+1)&quot;s&quot;,&quot;&quot;);for(pos=i;pos&lt;i+width&amp;&amp;pos&lt;=max2;pos+=10)printf(pos-i&gt;width-10?&quot;%s&quot;:&quot;%-10s&quot;,pos);print&quot;&quot;;for(j in seq){printf(&quot;%&quot;max1&quot;s &quot;,name[j]);for(k=i;k&lt;=max2&amp;&amp;k&lt;i+width;k++)printf(&quot;%s&quot;,&quot;\033[38;2;0;0;0m\033[48;2;&quot;a[seq[j][k]]&quot;m&quot;seq[j][k]&quot;\033[0m&quot;);print&quot;&quot;}}}&#39; &quot;width=${1-60}&quot; &lt;(aacolor|sed $&#39;s/ /\t/;s/ /;/g&#39;) -)

# print an alignment of selected viruses
thin 92 &lt;spike.pid|cut -d\  -f1|seqkit grep -f- spike.aa|seqkit grep -nrvp &#39;SARS coronavirus&#39;|cat - &lt;(seqkit grep -nrip db159,hu-1,tor2,zc45,rp3\\b,rs7896,banal-20-52,ratg,lyra11,btsy2,shc014,wiv1\\b,gx_p2v,gx-p1e spike.aa)|seqkit rmdup -s|aln --reorder -|cut -d, -f1|sed &#39;s/&gt;[^ ]*/&gt;/;s/ ORF.*//;s/ surface .*//;s/Severe acute respiratory syndrome/SARS/;s/Khosta-2 strain //;s/ isolate / /;s/ strain / /;s/Bat SARS-like coronavirus //;s/Betacoronavirus sp. //;s/ RNA$//&#39;|seqcol</pre>
<h3>
<a href=#Arkmedics_BLAST_searches_for_Pradhan_et_al_s_inserts id=Arkmedics_BLAST_searches_for_Pradhan_et_al_s_inserts>Arkmedic's
BLAST searches for Pradhan et al.'s inserts</a>
</h3>
<p>Arkmedic did a BLAST search for the nucleotide sequence of Pradhan's
first insert, but he used the coding sequence of the insert in SARS2 and
not HIV. He restricted the search to viruses published in 2019 or
earlier, so many of the top matches were sequences of HIV:
[<a href=https://www.arkmedic.info/p/absolute-proof-the-gp-120-sequences>https://www.arkmedic.info/p/absolute-proof-the-gp-120-sequences</a>]</p>
<div>
<img src=i/origin-pradhan-9.webp width=1400>
</div>
<p>However his top matches got only 83% query coverage because they only
matched 15 out of 18 bases of the query, similar to my matches here:</p>
<div>
<img src=i/origin-pradhan-10.jpg width=600>
</div>
<p>The first match above was missing the first 3 bases of the query and
the second match was missing the first base, but both matches were still
in frame so that the first match coded for 5 out of 6 aa of
<code>TNGTKR</code> and the second match coded for 4 out of 6 aa:</p>
<pre class=maxwidth>$ emu()(curl -sd &quot;id=$(paste -sd,)&quot; &quot;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=${1:-nuccore}&amp;rettype=${2:-fasta}&amp;retmode=$3&quot;)
$ emu protein fasta_cds_na&lt;&lt;&lt;AVN55366|seqkit translate|seqkit subseq -r 400:409
&gt;lcl|MH012801.1_cds_AVN55366.1_1 [gene=env] [protein=envelope glycoprotein] [protein_id=AVN55366.1] [location=1..2577] [gbkey=CDS]
WENGTKRSNE
$ emu &#39;&#39; fasta_cds_na&lt;&lt;&lt;JX071072|seqkit translate|seqkit subseq -r 259:266
&gt;lcl|JX071072.1_cds_AFV81607.1_1 [gene=env] [protein=envelope glycoprotein] [protein_id=AFV81607.1] [location=&lt;1..&gt;1116] [gbkey=CDS]
NPNGTKIY
</pre>
<p>The main text of Pradhan's paper doesn't include the accession
numbers of the HIV sequences which matched their inserts, but you have
to dig them up from the supplementary Excel file which was only posted
at bioRxiv but not ResearchGate:
<a href=https://www.biorxiv.org/content/10.1101/2020.01.30.927871v1.supplementary-material?versioned=true.><a
href="https://www.biorxiv.org/content/10.1101/2020.01.30.927871v1.supplementary-material?versioned=true"
class="uri">https://www.biorxiv.org/content/10.1101/2020.01.30.927871v1.supplementary-material?versioned=true</a>.</a>
It shows that the first insert was matched by 10 different sequences
from Thailand:</p>
<table>
<tr>
<th>
country
</th>
<th>
accession
</th>
<th>
subtype
</th>
<th>
insert
</th>
</tr>
<tr>
<td>
Thailand
</td>
<td>
AFU28737.1
</td>
<td>
CRF01_AE
</td>
<td>
Insert 1
</td>
</tr>
<tr>
<td>
Thailand
</td>
<td>
AFU28711.1
</td>
<td>
CRF01_AE
</td>
<td>
Insert 1
</td>
</tr>
<tr>
<td>
Thailand
</td>
<td>
AFU28717.1
</td>
<td>
CRF01_AE
</td>
<td>
Insert 1
</td>
</tr>
<tr>
<td>
Thailand
</td>
<td>
AFU28733.1
</td>
<td>
CRF01_AE
</td>
<td>
Insert 1
</td>
</tr>
<tr>
<td>
Thailand
</td>
<td>
AFU28693.1
</td>
<td>
CRF01_AE
</td>
<td>
Insert 1
</td>
</tr>
<tr>
<td>
Thailand
</td>
<td>
AFU28721.1
</td>
<td>
CRF01_AE
</td>
<td>
Insert 1
</td>
</tr>
<tr>
<td>
Thailand
</td>
<td>
AFU28699.1
</td>
<td>
CRF01_AE
</td>
<td>
Insert 1
</td>
</tr>
<tr>
<td>
Thailand
</td>
<td>
AFU28729.1
</td>
<td>
CRF01_AE
</td>
<td>
Insert 1
</td>
</tr>
<tr>
<td>
Thailand
</td>
<td>
AFU28705.1
</td>
<td>
CRF01_AE
</td>
<td>
Insert 1
</td>
</tr>
<tr>
<td>
Thailand
</td>
<td>
AFU28725.1
</td>
<td>
CRF01_AE
</td>
<td>
Insert 1
</td>
</tr>
<tr>
<td>
Kenya
</td>
<td>
ALB06757.1
</td>
<td>
G
</td>
<td>
Insert 2
</td>
</tr>
<tr>
<td>
India
</td>
<td>
ACL98861.1
</td>
<td>
C
</td>
<td>
Insert 3
</td>
</tr>
<tr>
<td>
India
</td>
<td>
ACL98864.1
</td>
<td>
C
</td>
<td>
Insert 3
</td>
</tr>
<tr>
<td>
India
</td>
<td>
ACL98860.1
</td>
<td>
C
</td>
<td>
Insert 3
</td>
</tr>
<tr>
<td>
India
</td>
<td>
ACL98859.1
</td>
<td>
C
</td>
<td>
Insert 3
</td>
</tr>
<tr>
<td>
India
</td>
<td>
AKR75206.1
</td>
<td>
C
</td>
<td>
Insert 4
</td>
</tr>
</table>
<p>However in all Thai sequences the first insert was coded by
<code>ACA AAT GGA ACC AAG AGG</code>, so it's actually 3 bases different
from the nucleotide sequence in Wuhan-Hu-1 that was used by Arkmedic
(<code>ACC AAT GGT ACT AAG AGG</code>):</p>
<pre>$ emu()(curl -sd &quot;id=$(paste -sd,)&quot; &quot;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=${1:-nuccore}&amp;rettype=${2:-fasta}&amp;retmode=$3&quot;)
$ emu protein fasta_cds_na&lt;&lt;&lt;AFU28737,AFU28711,AFU28717,AFU28733,AFU28693,AFU28721,AFU28699,AFU28729,AFU28705,AFU28725 &gt;insert1.fa
$ seqkit translate insert1.fa|seqkit locate -p TNGTKR|sed 1d|cut -f1,5|while read l m;do seqkit grep -p $l insert1.fa|seqkit subseq -r $[m*3-2]:$[m*3-2+17];done|seqkit seq -s|sort -u|sed &#39;s/.../&amp; /g&#39;
ACA AAT GGA ACC AAG AGG</pre>
<p>There were about 3.2 million results at GenBank when I searched for
<code>0:2019[dp] viruses[organism]</code>, but about 0.9 million or about
27% of all results were classified under HIV-1.
[<a href=https://www.ncbi.nlm.nih.gov/nuccore/?term=0%3A2019%5Bdp%5D+viruses%5Borganism%5D>https://www.ncbi.nlm.nih.gov/nuccore/?term=0%3A2019%5Bdp%5D+viruses%5Borganism%5D</a>]
So the reason why many of Arkmedic's top hits at BLAST were sequences of
HIV-1 might be because before 2020 HIV-1 was probably the species of
virus with the most sequences at GenBank:</p>
<div>
<img src=i/origin-pradhan-11.jpg width=600>
</div>
<p>Arkmedic also wrote that he didn't find any 100% match when he
searched for the nucleotide sequence of Pradhan's second insert on
BLAST, but that's because he again used the nucleotide sequence in
Wuhan-Hu-1 (<code>CAC AAA AAC AAC AAA AGT</code>), which is 3 bases
different from Pradhan's HIV sequences:</p>
<pre>$ emu protein fasta_cds_na&lt;&lt;&lt;ALB06757 &gt;insert2.fa
$ x=$(seqkit translate insert2.fa|seqkit locate -p HKNNKS|sed 1d|cut -f5);seqkit subseq -r $[x*3-2]:$[(x+5)*3] insert2.fa|seqkit seq -s|sed &#39;s/.../&amp; /g&#39;
CAT AAA AAT AAT AAA AGT</pre>
<p>When I did a BLAST search for the nucleotide sequence in HIV, it had
18/18 matching bases only with Pradhan's Kenyan HIV-1 sequence and with
some other random virus where the match was on the minus strand (but in
the screenshot below the other results with 100% identity and 100% query
coverage were noncontiguous matches where the match was split into two
different segments, as you can see from the max score being lower than
on the first two rows):</p>
<div>
<img src=i/origin-pradhan-12.jpg width=800>
</div>
<p>Arkmedic also pointed out that he found no matches for Pradhan's
third insert on BLAST, but that's because he searched for the region
around the insert in SARS-CoV-2 and not HIV (<code>RSYLTPGDSSSG</code>
and not <code>RTYLFNETRGNSSSG</code>). The nucleotide sequence of the
version in HIV is found in all 4 HIV sequences that were listed in
Pradhan's supplementary Excel file:</p>
<pre>$ emu protein fasta_cds_na&lt;&lt;&lt;ACL98861,ACL98864,ACL98860,ACL98859 &gt;insert3.fa
$ x=RTYLFNETRGNSSSG;seqkit translate insert3.fa|seqkit locate -p $x|sed 1d|cut -f1,5|while read l m;do seqkit grep -p $l insert3.fa|seqkit subseq -r $[m*3-2]:$[(m+${#x})*3];done|seqkit seq -s|sort -u|sed &#39;s/.../&amp; /g&#39;
AGG ACA TAC CTG TTT AAT GAG ACA AGA GGT AAT TCA AGC TCA GGT AAT</pre>
<div>
<img src=i/origin-pradhan-13.jpg width=800>
</div>
<p>Arkmedic wrote: <span class="quote">"TLDR: In order to get the 3
inserts of Gp-120 to exist in SARS-Cov-2, the genomic sequences that
coded for them had to have got there by recombination from another
organism or in a lab. Because they don't exist anywhere in nature it is
not possible to have come from another organism."</span></p>
<p>However as I showed earlier, Pradhan's first insert actually looks
like a deletion in SARS1 and not an insertion in SARS2. And Pradhan's
second and third inserts were placed at the wrong spot because he only
did a pairwise alignment of SARS1 and SARS2 and he didn't include more
sarbecoviruses in his alignment.</p>
<p>Arkmedic was also wrong that the inserts <span class="quote">"don't
exist anywhere in nature"</span>, because the first two so-called
inserts are only 6 aa long, so if you do a BLAST search for the
nucleotide sequences of the inserts in Wuhan-Hu-1, they get perfect
matches to many organisms but just not to virus sequences published
before 2020, like for example this shows that the second insert has
perfect matches to mollusks, otters, fish, and so on:</p>
<div>
<img src=i/origin-pradhan-15.jpg width=800>
</div>
<p>Arkmedic wrote: <span class="quote">"I mean, sure, the old virus can
undergo some mutations (GATTTCA...) but these are evolutionarily very slow
and can result in deletions and changes. In order to get inserts for
these to happen by chance is so rare that it would take millions of
years to develop functional inserts by chance so you really need a
genome donor."</span> However for example the nucleotide sequence of the
second insert is <code>CACAAAAACAACAAAAGT</code> in Wuhan-Hu-1, which is
identical in RaTG13, differs by one base in BANAL-52 and BtSY2, and
differs by two bases in Rp22DB159. And even if you consider all
sequences released since 2020 to be fake, even in ZC45 which was
published in 2018, the region of the so-called insert differs by 3
nucleotide changes and 3 gaps:</p>
<div>
<img src=i/pradhan-insert2nt.webp width=700>
</div>
<p>Similarly the nucleotide sequence of the first insert TNGTKR differs
by 2 bases in RaTG13, 3 bases in BANAL-52, BtSY2, and Rp22DB159, and 7
bases in ZC45 (but it's located within a variable region of the genome
that is expected to be mutating fast):</p>
<div>
<img src=i/pradhan-insert1nt.webp width=700>
</div>
<h3>
<a href=#Matches_to_env_in_LANLs_HIV_subtype_reference id=Matches_to_env_in_LANLs_HIV_subtype_reference>Matches
to env in LANL's HIV subtype reference</a>
</h3>
<p>From the HIV sequence database of the Los Alamos National Laboratory,
you can download a FASTA file which contains 2-5 sequences from
different subtypes of HIV-1 along with 27 chimpanzee and gorilla
sequences of SIV that are similar to HIV-1. Set <span
class="quote">"Alignment type"</span> to <span class="quote">"Web (all
complete sequences)"</span>, set <span
class="quote">"DNA/Protein"</span> to <span
class="quote">"Protein"</span>, and click <span class="quote">"Get
alignment"</span>:
<a href=https://www.hiv.lanl.gov/content/sequence/NEWALIGN/align.html.><a
href="https://www.hiv.lanl.gov/content/sequence/NEWALIGN/align.html"
class="uri">https://www.hiv.lanl.gov/content/sequence/NEWALIGN/align.html</a>.</a>
You can keep region as <span class="quote">"env"</span> to download only
the envelope protein, which covers about 20% of the genome of HIV-1.</p>
<p>The 2023 edition of LANL's subtype reference has a total of 555
sequences, but zero of them have a perfect match to Pradhan's first
motif <code>TNGTKR</code>. Two sequences have one mismatch, 215 have 2
mismatches, and the rest have 3 mismatches:</p>
<pre>$ wget sars2.net/f/HIV1_REF_2023_env_DNA.fasta
$ x=TNGTKR;for((i=0;i&lt;=${#x};i++));do seqkit seq -g HIV1_REF_2023_env_PRO.fasta|seqkit replace -sp \# -r X|seqkit locate -iPp $x -m$i|sed 1d|awk &#39;!a[$1]++&#39;|echo &quot;$i `wc -l`&quot;;done
0 0
1 2
2 215
3 555
4 555
5 555
6 555</pre>
<p>For the second motif <code>HKNNKS</code>, there's zero perfect
matches and only sequence with a single mismatch:</p>
<pre>$ x=HKNNKS;for((i=0;i&lt;=${#x};i++));do seqkit seq -g HIV1_REF_2023_env_PRO.fasta|seqkit replace -sp \# -r X|seqkit locate -iPp $x -m$i|sed 1d|awk &#39;!a[$1]++&#39;|echo &quot;$i `wc -l`&quot;;done
0 0
1 1
2 41
3 551
4 555
5 555
6 555</pre>
<p>All of the matches with one mismatch were located in the first half
of gp120:</p>
<pre>$ seqkit seq -g HIV1_REF_2023_env_PRO.fasta|seqkit replace -sp\# -rX|seqkit locate -ip TNGTKR -m1|cut -f1,3,5-|column -t
seqID                               pattern  start  end  matched
Ref.H.GB.00.00GBAC4001.FJ711703     tngtkr   190    195  tngtnr
Ref.L.CD.01.L_CG_0018a_01.MN271384  tngtkr   132    137  tngttr
$ seqkit seq -g HIV1_REF_2023_env_PRO.fasta|seqkit replace -sp\# -rX|seqkit locate -ip HKNNKS -m1|cut -f1,3,5-|column -t
seqID                               pattern  start  end  matched
Ref.101_01B.CN.13.YNKM250.MK158946  hknnks   366    371  hfnnks</pre>
<p>The two matches to <code>TNGTKR</code> were both located within
variable loops but the match to <code>HKNNKS</code> wasn't:</p>
<pre>seqkit grep -nrp FJ711703,MN271384,YNKM250,HXB2 HIV1_REF_2023_env_PRO.fasta|seqkit replace -p&#39;^Ref\.(.*)\..*?$&#39; -r\$1|mafft --quiet -|seqkit fx2tab|gawk -F\\t &#39;ARGIND==1{a[$1]=$2}ARGIND==2{for(i=$2;i&lt;=$3;i++)loop[i]=$1}ARGIND==3{if($1~/HXB2/){gap=0;split($2,z,&quot;&quot;);for(i in z){if(z[i]==&quot;-&quot;)gap++;gaps[i]=gap}}name[FNR]=$1;split($2,z,&quot;&quot;);for(i in z)seq[FNR][i]=z[i];if(length($1)&gt;max1)max1=length($1);if(length($2)&gt;max2)max2=length($2)}END{width=60;for(i=1;i&lt;=max2;i+=width){print&quot;&quot;;printf(&quot;%*s&quot;,max1+1,&quot;&quot;);for(pos=i;pos&lt;i+width&amp;&amp;pos&lt;=max2;pos+=10)printf(pos-i&gt;width-10?&quot;%s&quot;:&quot;%-10s&quot;,pos-gaps[pos]);print&quot;&quot;;printf(&quot;%*s&quot;,max1+1,&quot;&quot;);for(pos=i;pos&lt;i+width;pos++){l=loop[pos-gaps[pos]];printf&quot;%s&quot;,l?l:&quot; &quot;};print&quot;&quot;;for(j in seq){printf(&quot;%&quot;max1&quot;s &quot;,name[j]);for(k=i;k&lt;=max2&amp;&amp;k&lt;i+width;k++)printf(&quot;%s&quot;,&quot;\033[38;2;0;0;0m\033[48;2;&quot;a[seq[j][k]in a?seq[j][k]:&quot;X&quot;]&quot;m&quot;seq[j][k]&quot;\033[0m&quot;);print&quot;&quot;}}}&#39; &lt;(aacolor|sed $&#39;s/ /\t/;s/ /;/g&#39;) &lt;(printf %s\\n 1:131:157 2:158:196 3:296:331 4:385:418 5:460:469|tr : \\t) -</pre>
<div>
<img src=i/pradhan-subref.webp width=700>
</div>
<p>The numbers in the image above show the position in HXB2. I used
these as the coordinates of the variable loops in the env protein of
HXB2:</p>
<pre>loop start end
V1   131   157
V2   158   196
V3   296   331
V4   385   418
V5   460   469</pre>
<h3>
<a href=#E_values_of_matches_to_Pradhans_first_two_inserts id=E_values_of_matches_to_Pradhans_first_two_inserts>E
values of matches to Pradhan's first two inserts</a>
</h3>
<p>The E value or expect value indicates how many similarly close
matches are expected to occur by chance for the given query sequence and
pool of searched sequences.</p>
<p>You can try to go to the web GUI for blastp:
<a href=https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp.><a
href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp"
class="uri">https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp</a>.</a>
Then set query to <code>TNGTKR</code>, organism to <span
class="quote">"Viruses (taxid:10239)"</span>, click <span
class="quote">"Add organism"</span>, set the second organism to <span
class="quote">"SARS-CoV-2 (taxid:2697049)"</span> and click the exclude
checkbox next to it. It showed me that the matches to HIV got an E value
of about 372, which means that about 372 similar matches with 6 out of 6
identical residues are expected to occur by chance. There were a total
of 76 perfect matches with 6 out of 6 identical residues, so the expect
score was about 5 times higher than the actual number of perfect
matches.</p>
<p>When I did another search in the whole nr database so that I didn't
include the organism filters, then the E scores of perfect matches
increased to about 54093. So they were now about 145 times higher than
with the filters because now the pool of subject sequences was much
larger, and the E score is directly proportional to the effective length
of the pool of subject sequences.</p>
<p>The E score is calculated by the formula
<code>E = m * n * 2^(-S)</code>, where <code>m</code> and <code>n</code>
are the effective length of the query and database sequences and
<code>S</code> is the bitscore. I don't know how to calculate the
effective length, because it is calculated by subtracting a parameter
called the length adjustment from the actual length to account for
duplicate sequences and repetitive elements within sequences, so I
believe you need to download the entire database to calculate the length
adjustment. However since the E values are directly proportional to the
effective length of the pool of subject sequences, from the ratio of my
E scores you can calculate that the effective length of the whole nr
database is about 145 times higher than the effective length of the
subset of the nr database that contains viruses apart from
SARS-CoV-2.</p>
<p>Pradhan's second insert <code>HKNNKS</code> was 6 residues long like
the first insert, but when I searched for the second insert among
viruses other than SARS-CoV-2 in the nr database, the E value was now
about 185 for the match to HIV. The results included 36 perfect matches,
so the expect score was again about 5 times higher than the number of
perfect matches. I believe the reason why both queries don't get the
same E value even though they're the same length is that the algorithm
assigns different scores to common amino acids and rare amino acids.</p>
<p>The nr database contains a massive number of protein sequences for a
few species of widely studied human viruses like SARS-CoV-2, HIV-1, and
influenza A. But in order to restrict the search to a database that only
has one or a few representative samples for each species of virus, you
can search in the refseq_protein database instead and restrict the
organism to viruses. It gave me an E value of about 33 for results that
had a perfect match to Pradhan's first insert <code>TNGTKR</code> (but
the results didn't include any HIV sequences, because the envelope
protein of the HIV-1 refseq matches only 2 of 6 residues of
<code>TNGTKR</code> at the spot where Pradhan's HIV sequences from
Thailand match <code>TNGTKR</code>):</p>
<div>
<img src=i/pradhan-escore.jpg width=800>
</div>
<p>So even within a fairly small pool of proteins from virus refseqs,
the E values for perfect matches to the 6 aa insert were still very
high.</p>
<p>In the following code I created my own BLAST database for proteins
from virus refseqs. For some reason it failed to download coding
sequences for about half of all virus refseqs even though refseqs should
usually have annotations for proteins. But anyway, I now got only a
single perfect match to <code>TNGTKR</code> which was in the refseq for
SARS-CoV-2, but the E value of the match was still 289 which is very
high:</p>
<pre class=maxwidth>$ wget -q https://ftp.ncbi.nlm.nih.gov/refseq/release/viral/viral.1.1.genomic.fna.gz
$ seqkit seq -ni viral.fa|curl -sd &quot;id=$(paste -sd, -)&quot; &#39;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&amp;rettype=fasta_cds_na&#39;&gt;viral.aa
$ makeblastdb -dbtype prot -in viral.aa
[...]
$ echo $&#39;&gt;a\nTNGTKR&#39;|blastp -db viral.aa -evalue 1000
[...]
&gt;lcl|NC_045512.2_prot_YP_009724390.1_3 [gene=S] [locus_tag=GU280_gp02] # spike protein of refseq for SARS-CoV-2
[db_xref=GeneID:43740568] [protein=surface glycoprotein]
[protein_id=YP_009724390.1] [location=21563..25384] [gbkey=CDS]
Length=1273
</pre>
<pre> Score = 16.5 bits (31),  Expect = 289, Method: Composition-based stats.
 Identities = 6/6 (100%), Positives = 6/6 (100%), Gaps = 0/6 (0%)

Query  1   TNGTKR  6
           TNGTKR
Sbjct  73  TNGTKR  78
[...]</pre>
<h3>
<a href=#Are_Pradhans_inserts_involved_in_entry_to_CD4_T_helper_cells id=Are_Pradhans_inserts_involved_in_entry_to_CD4_T_helper_cells>Are
Pradhan's inserts involved in entry to CD4 T helper cells?</a>
</h3>
<p>Tony VanDongen posted this tweet:
[<a href=https://x.com/tony_vandongen/status/1887707580148941305>https://x.com/tony_vandongen/status/1887707580148941305</a>]</p>
<div>
<img src=i/pradhan-tcell.webp width=500>
</div>
<p>Brian Gadd posted this reply: <span class="quote">"AFAIK there is no
evidence that these sequences are necessary or sufficient for CD4
binding; they are absent from most gp120 sequences, no two homology
sequences are found in the same molecule of gp120, they barely overlap
with the CD4 binding site &amp; not at the most important contact
points."</span>
[<a href=https://x.com/Brian_Gadd/status/1887724003604746578>https://x.com/Brian_Gadd/status/1887724003604746578</a>]
He also replied: <span class="quote">"Brunetti et al. did look at
spike-CD4 interaction and their binding data suggests that this
interaction is almost exclusively mediated by the RBD, not the NTD:
<a href=https://elifesciences.org/articles/84790.><a
href="https://elifesciences.org/articles/84790"
class="uri">https://elifesciences.org/articles/84790</a>.</a>"</span>
Another user responded: <span class="quote">"CD209 (DC-SIGN) is a lectin
that binds HIV gp120 and modulates CD4+ T-cell infection. Lectin-gp120
interactions impact HIV entry—critical for viral transmission and immune
evasion."</span> But Gadd answered: <span class="quote">"Yes. And Amraei
et al. found that L-SIGN and DC-SIGN do indeed bind to spike, and that
this interaction is mediated by the RBD:
<a href=https://pubs.acs.org/doi/10.1021/acscentsci.0c01537.><a
href="https://pubs.acs.org/doi/10.1021/acscentsci.0c01537"
class="uri">https://pubs.acs.org/doi/10.1021/acscentsci.0c01537</a>.</a>"</span></p>
<h2>
<a href=#Perez_and_Montagniers_paper_about_BLAST_matches_to_HIV_and_SIV id=Perez_and_Montagniers_paper_about_BLAST_matches_to_HIV_and_SIV>Perez
and Montagnier's paper about BLAST matches to HIV and SIV</a>
</h2>
<p>In April 2020 Jean-Claude Perez and Luc Montagnier published a
preprint at OSF titled <span class="quote">"COVID-19, SARS and Bats
Coronaviruses Genomes Unexpected Exogenous RNA Sequences"</span>.
[<a href=https://osf.io/preprints/osf/d9e5g_v1>https://osf.io/preprints/osf/d9e5g_v1</a>]
A slightly revised version of the preprint was published in May 2020 at
ResearchGate.
[<a href=https://www.researchgate.net/publication/341756383_COVID-19_SARS_and_Bats_Coronaviruses_Genomes_Unexpected_Exogenous_RNA_Sequences>https://www.researchgate.net/publication/341756383_COVID-19_SARS_and_Bats_Coronaviruses_Genomes_Unexpected_Exogenous_RNA_Sequences</a>]
The paper was published in August 2020 in an Indian junk journal.
[<a href=https://www.granthaalayahpublication.org/journals/granthaalayah/article/view/IJRG20_B07_3568>https://www.granthaalayahpublication.org/journals/granthaalayah/article/view/IJRG20_B07_3568</a>]
I refer to the author of the paper as simply Perez, who seems to have
done most of the work on the paper.</p>
<h3>
<a href=#Location_of_regions_A_and_B id=Location_of_regions_A_and_B>Location
of regions A and B</a>
</h3>
<p>Perez confusingly did not use Wuhan-Hu-1 as his reference genome but
another early sequence from Wuhan.
[<a href=https://www.ncbi.nlm.nih.gov/nucleotide/LR757998>https://www.ncbi.nlm.nih.gov/nucleotide/LR757998</a>]
It's missing the first 25 bases from the start of Wuhan-Hu-1, so the
genome coordinates given by Perez are off by 25 relative to Wuhan-Hu-1
coordinates.</p>
<p>Perez identified two contiguous regions with a high number of matches
to HIV or SIV, which he called region A and region B:</p>
<div>
<img src=i/pradhan-perezab.webp width=600>
</div>
<p>Perez indicated that region A was 600 bases long but it matched bases
21072 to 21672 of the reference genome, which would mean that region A
would be 601 bases long if both ends of the range would be inclusive.
But the end of the range is exclusive and Perez started numbering the
bases from zero and not one, so the region matches bases 21073 to 21672
in regular one-based numbering where both ends of the range are
inclusive:</p>
<pre class=maxwidth>$ emu()(curl -sd &quot;id=$(paste -sd,)&quot; &quot;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=${1:-nuccore}&amp;rettype=${2:-fasta}&amp;retmode=$3&quot;)
$ emu&lt;&lt;&lt;LR757998&gt;perezref.fa
$ seqkit locate -p AGGGTTTTTTCACTTACATTTGTGGGTTTATACAACAAAAGCTAGCTCTTGGAGGTTCCGTGGCTATAAAGATAACAGAACATTCTTGGAATGCTGATCTTTATAAGCTCATGGGACACTTCGCATGGTGGACAGCCTTTGTTACTAATGTGAATGCGTCATCATCTGAAGCATTTTTAATTGGATGTAATTATCTTGGCAAACCACGCGAACAAATAGATGGTTATGTCATGCATGCAAATTACATATTTTGGAGGAATACAAATCCAATTCAGTTGTCTTCCTATTCTTTATTTGACATGAGTAAATTTCCCCTTAAATTAAGGGGTACTGCTGTTATGTCTTTAAAAGAAGGTCAAATCAATGATATGATTTTATCTCTTCTTAGTAAAGGTAGACTTATAATTAGAGAAAACAACAGAGTTGTTATTTCTAGTGATGTTCTTGTTAACAACTAAACGAACAATGTTTGTTTTTCTTGTTTTATTGCCACTAGTCTCTAGTCAGTGTGTTAATCTTACAACCAGAACTCAATTACCCCCTGCATACACTAATTCTTTCACACGTGGTGTTTATTACCCTGACAAAGTTTTCAGATCC perezref.fa|cut -f5,6|column -t
start  end
21073  21672
</pre>
<p>Regions A and B combined consist of bases 21098 to 22027 of
Wuhan-Hu-1. Region A consists of the last 458 bases of ORF1ab, the
7-base intergenic region between ORF1ab and the spike protein, and the
first 135 bases of the spike protein. Region B consists of bases 136 to
465 of the spike protein.</p>
<h3>
<a href=#Table_of_15_BLAST_matches_to_HIV_and_SIV id=Table_of_15_BLAST_matches_to_HIV_and_SIV>Table
of 15 BLAST matches to HIV and SIV</a>
</h3>
<p>Table 1 of Perez's paper showed matches to regions A and B in various
sequences of HIV-1, HIV-2, and SIV, where matches shorter than 18 bases
were indicated as not significant (even though in reality all matches
were either so short or they had so many errors that their E values were
far above 0.05).</p>
<div>
<img src=i/pradhan-perez.webp width=600>
</div>
<p>The table has several errors:</p>
<ul>
<li>Perez wrote that the table contains 14 matches out of which 12 are
significant, even though actually the table contains 15 matches out of
which 13 are indicated to be so-called significant matches.</li>
<li>The SIV isolate TAN5 from Tanzania should have the accession number
JN091691, but it was accidentally given the same accession number as the
sequence on the next row.</li>
<li>The ranges in the last column were accidentally swapped for AF003044
and JN091691.</li>
<li>On the last row the length of the match is indicated to be 30 bases
even though the given starting and ending positions would result in a
length of only 20 bases, but it's because the starting position is too
high by 10.</li>
</ul>
<p>It's also confusing that the starting and ending positions in the
last column of the table use zero-based indexing and not regular
one-based indexing.</p>
<p>When I downloaded all 15 sequences in the table and I did a BLAST
search for the sequences against SARS-CoV-2, there were no matches
returned when I ran BLAST with the default settings:</p>
<pre>$ cat perez # fixed errors mentioned above and converted to 1-based indexing
JN230738.1 21138 21153
MF373163.1 21226 21246
JN863831.1 21308 21325
KR862351.1 21438 21458
EU875177.1 21543 21573
JF267434.1 21584 21601
KJ131112.1 21695 21714
AF003044.1 21749 21768 # the starting and ending positions on these
JN091691.1 21701 21722 # two rows were accidentally swapped
HQ644953.1 21757 21784
L07625.1 21804 21829
JF811228.1 21851 21866
KC187066.1 21884 21915
GU481454.1 21914 21952
LM999945.1 21941 21970 # the starting position of this row was too high by 10
$ curl &quot;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&amp;rettype=fasta&amp;id=$(cut -d\  -f1 perez|paste -sd, -)&quot;&gt;perez.fa
$ brew install blast
[...]
$ makeblastdb -dbtype nucl -in perez.fa
[...]
$ fmt=&#39;sseqid evalue nident length pident sstrand qstart qend&#39;
$ &lt;perezref.fa blastn -db perez.fa -outfmt &quot;6 $fmt&quot;
$ # no results</pre>
<p>So therefore in order to use less strict matching criteria, I added
the options <code>-word_size 5 -evalue 100</code> which gave me one or
more matches for all 15 sequences. At first the starting position of one
of my matches wasn't the same as in Perez's table, but I got the
starting positions to match when I added the option
<code>-task blastn</code>:</p>
<pre>$ &lt;perezref.fa blastn -db perez.fa -outfmt &quot;6 $fmt&quot; -task blastn|awk &#39;$7&gt;=21073&amp;&amp;$8&lt;=22027&#39;|awk &#39;NR==FNR{a[$1]=$2&quot; &quot;$3;pos[$1]=NR;next}{print pos[$1],$0,a[$1]}&#39; perez -|awk &#39;{print$0,$8==$10&amp;&amp;$9==$11?&quot;true&quot;:&quot;false&quot;}&#39;|sort -rk12|sort -sgk3|awk &#39;!a[$1]++&#39;|sort -n|cut -d\  -f2-|(echo $fmt perezstart perezend matchesperez;cat)|column -t
sseqid      evalue  nident  length  pident   sstrand  qstart  qend   perezstart  perezend  matchesperez
JN230738.1  1.3     16      16      100.000  plus     21138   21153  21138       21153     true
MF373163.1  0.003   21      21      100.000  minus    21226   21246  21226       21246     true
JN863831.1  4.7     17      18      94.444   minus    21308   21325  21308       21325     true
KR862351.1  0.11    20      21      95.238   minus    21438   21458  21438       21458     true
EU875177.1  0.009   28      32      87.500   minus    21543   21573  21543       21573     true
JF267434.1  0.11    18      18      100.000  plus     21584   21601  21584       21601     true
KJ131112.1  0.38    19      20      95.000   minus    21695   21714  21695       21714     true
AF003044.1  0.38    19      20      95.000   plus     21749   21768  21749       21768     true
JN091691.1  0.38    20      22      90.909   minus    21701   21722  21701       21722     true
HQ644953.1  0.009   25      28      89.286   plus     21757   21784  21757       21784     true
L07625.1    1.3     22      26      84.615   plus     21804   21829  21804       21829     true
JF811228.1  1.3     16      16      100.000  minus    21851   21866  21851       21866     true
KC187066.1  0.009   28      32      87.500   minus    21884   21915  21884       21915     true
GU481454.1  0.009   32      39      82.051   minus    21914   21952  21914       21952     true
LM999945.1  0.38    25      30      83.333   minus    21941   21970  21941       21970     true</pre>
<p>In the table above the E value indicates how many matches are
expected to occur by chance for a given query in a given database. A
bigger database size means higher E values. But even though in the code
above I had a tiny database that consisted of only 15 sequences of HIV
and SIV, 4 out of 15 matches above got an E value above 1, which means
that more than 1 similar matches are expected to occur by chance. And
even the lowest E value was only about 0.003.</p>
<p>The output above shows that 10 out of 15 matches are on the minus
strand, which means that HIV/SIV matched the reverse complement of the
segment in SARS-CoV-2, so the matched segment does not even code for the
same amino acids in SARS-CoV-2 and HIV/SIV. For example the second
sequence matches a Swedish HIV sequence from 2017, but it matches
SARS-CoV-2 on the opposite strand so the translation is completely
different:
[<a href=https://www.ncbi.nlm.nih.gov/nuccore/MF373163>https://www.ncbi.nlm.nih.gov/nuccore/MF373163</a>]</p>
<pre>SARS-CoV-2:                HIV:
                            &lt;--------------------
 --------------------&gt;      TACGAAGTCTACTACTGCGTA # reversed version of segment matched in SARS-CoV-2
 ATGCGTCATCATCTGAAGCAT      ATGCTTCAGATGATGACGCAT # reversed version with complemented bases
AATGCGTCATCATCTGAAGCATTT   TATGCTTCAGATGATGACGCATAT
 N  A  S  S  S  E  A  F     Y  A  S  D  D  D  A  Y</pre>
<p>So after the 10 matches on the wrong strand are eliminated, only 5
out of 15 matches remain. But 2 out of the 5 matches are in a different
frame in SARS-CoV-2 and HIV/SIV, so their amino acid translation is not
the same, which means that only 3 out of the 15 matches in Perez's table
are both on the right strand and in the right frame:</p>
<table>
<tr>
<th>
accession
</th>
<th>
SC2 start
</th>
<th>
SC2 CDS start
</th>
<th>
SC2 frame
</th>
<th>
HIV start
</th>
<th>
HIV CDS start
</th>
<th>
HIV frame
</th>
<th>
same frame
</th>
</tr>
<tr>
<td>
JN230738
</td>
<td>
21138
</td>
<td>
20899
</td>
<td>
1
</td>
<td>
77
</td>
<td>
77
</td>
<td>
2
</td>
<td>
false
</td>
</tr>
<tr>
<td>
JF267434
</td>
<td>
21584
</td>
<td>
47
</td>
<td>
2
</td>
<td>
284
</td>
<td>
284
</td>
<td>
2
</td>
<td>
true
</td>
</tr>
<tr>
<td>
AF003044
</td>
<td>
21749
</td>
<td>
212
</td>
<td>
2
</td>
<td>
845
</td>
<td>
845
</td>
<td>
2
</td>
<td>
true
</td>
</tr>
<tr>
<td>
HQ644953
</td>
<td>
21757
</td>
<td>
220
</td>
<td>
1
</td>
<td>
967
</td>
<td>
967
</td>
<td>
1
</td>
<td>
true
</td>
</tr>
<tr>
<td>
L07625
</td>
<td>
21804
</td>
<td>
267
</td>
<td>
3
</td>
<td>
6701
</td>
<td>
23
</td>
<td>
2
</td>
<td>
false
</td>
</tr>
</table>
<p>In the table above the <span class="quote">"CDS start"</span> columns
show the starting position of the match within the protein coding
sequence it is part of. For example the match on the first row starts at
base 20899 of ORF1ab in SARS-CoV-2. By calculating
<code>(20899-1)%3+1</code>, you get the frame number which is 1.</p>
<h3>
<a href=#Expect_values_of_Perezs_matches_at_web_BLAST id=Expect_values_of_Perezs_matches_at_web_BLAST>Expect
values of Perez's matches at web BLAST</a>
</h3>
<p>In order to see how high the E values of Perez's matches were in the
full nucleotide database, I copied this sequence which consists of
Perez's regions A and B:</p>
<pre>$ seqkit subseq -r 21073:22027 sars2.fa|seqkit seq -s
GTTACAAAAGAAAATGACTCTAAAGAGGGTTTTTTCACTTACATTTGTGGGTTTATACAACAAAAGCTAGCTCTTGGAGGTTCCGTGGCTATAAAGATAACAGAACATTCTTGGAATGCTGATCTTTATAAGCTCATGGGACACTTCGCATGGTGGACAGCCTTTGTTACTAATGTGAATGCGTCATCATCTGAAGCATTTTTAATTGGATGTAATTATCTTGGCAAACCACGCGAACAAATAGATGGTTATGTCATGCATGCAAATTACATATTTTGGAGGAATACAAATCCAATTCAGTTGTCTTCCTATTCTTTATTTGACATGAGTAAATTTCCCCTTAAATTAAGGGGTACTGCTGTTATGTCTTTAAAAGAAGGTCAAATCAATGATATGATTTTATCTCTTCTTAGTAAAGGTAGACTTATAATTAGAGAAAACAACAGAGTTGTTATTTCTAGTGATGTTCTTGTTAACAACTAAACGAACAATGTTTGTTTTTCTTGTTTTATTGCCACTAGTCTCTAGTCAGTGTGTTAATCTTACAACCAGAACTCAATTACCCCCTGCATACACTAATTCTTTCACACGTGGTGTTTATTACCCTGACAAAGTTTTCAGATCCTCAGTTTTACATTCAACTCAGGACTTGTTCTTACCTTTCTTTTCCAATGTTACTTGGTTCCATGCTATACATGTCTCTGGGACCAATGGTACTAAGAGGTTTGATAACCCTGTCCTACCATTTAATGATGGTGTTTATTTTGCTTCCACTGAGAAGTCTAACATAATAAGAGGCTGGATTTTTGGTACTACTTTAGATTCGAAGACCCAGTCCCTACTTATTGTTAATAACGCTACTAATGTTGTTATTAAAGTCTGTGAATTTCAATTTTGTAATGATCCATTTTTGGGTGTTTATTACCACAAAAACAACAAAAGTTGGATGGAAAGT</pre>
<p>Then I pasted the sequence to the text box here:
<a href=https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn.><a
href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn"
class="uri">https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn</a>.</a>
I changed the database type to <span class="quote">"Nucleotide
collection (nt/nr)"</span>, which used to be the default database back
when Perez wrote his paper, and which has slightly more HIV sequences
than the new core_nt database. I entered <span class="quote">"HIV-1
(taxid:11676)"</span> in the organism field, clicked <span
class="quote">"Add Organism"</span>, entered <span class="quote">"HIV-2
(taxid:11709)"</span>, clicked <span class="quote">"Add
Organism"</span>, and entered <span class="quote">"Simian
immunodeficiency virus (taxid:11723)"</span>. I typed <span
class="quote">"0:2019[dp]"</span> under <span class="quote">"Entrez
Query"</span> to only select sequences with date of publication before
2020. And then under <span class="quote">"Program selection"</span>
change the algorithm from <span class="quote">"megablast"</span> to
<span class="quote">"blastn"</span>, because otherwise short matches
would not be returned. And under <span class="quote">"Algorithm
parameters"</span> I increased <span class="quote">"Expect
threshold"</span> from 0.05 to 10000, so that even matches that have
more than 5% likelihood to be due to chance will be returned, and I also
increased <span class="quote">"Max target sequences"</span> from 100 to
5000. Then I clicked <span class="quote">"BLAST"</span>.</p>
<p>There was a single match which had an E value of about 1.4, which was
the Swedish HIV sequence that was shown on the second row of Perez's
Table 1, and which has 21 out of 21 matching bases. But the next best
match had an E value of about 5.0, which means that about 5.0 similarly
close matches were expected to occur by chance:</p>
<div>
<img src=i/pradhan-pereze.webp width=1000>
</div>
<p>If I would've omitted the step where I increased the expect threshold
from 0.05 to 1000, BLAST would have returned zero results and said <span
class="quote">"No significant similarity found"</span>, because none of
the matches had lower than 5% likelihood of occuring by chance.</p>
<p>Next I selected <span class="quote">"Hit Table (text)"</span> from
the <span class="quote">"Download"</span> menu and kept only results
which were included in Perez's Table 1. The expect value was over 100
for 9 out of 15 results:</p>
<table>
<tr>
<th>
subject
</th>
<th>
expect
</th>
<th>
pident
</th>
<th>
length
</th>
<th>
mismatch
</th>
<th>
qstart
</th>
<th>
qend
</th>
<th>
sstart
</th>
<th>
send
</th>
<th>
sstrand
</th>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/JN230738.1>JN230738.1</a>
HIV-2 isolate 56 from France envelope glycoprotein (env) gene, partial
cds
</td>
<td>
746
</td>
<td>
100.000
</td>
<td>
16
</td>
<td>
0
</td>
<td>
91
</td>
<td>
106
</td>
<td>
77
</td>
<td>
92
</td>
<td>
+
</td>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/MF373163.1>MF373163.1</a>
HIV-1 isolate 060SE from Sweden, partial genome
</td>
<td>
1.4
</td>
<td>
100.000
</td>
<td>
21
</td>
<td>
0
</td>
<td>
179
</td>
<td>
199
</td>
<td>
5634
</td>
<td>
5614
</td>
<td>
-
</td>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/JN863831.1>JN863831.1</a>
HIV-2 isolate CA65410.13 from Guinea-Bissau envelope gene, partial cds
</td>
<td>
2604
</td>
<td>
94.444
</td>
<td>
18
</td>
<td>
1
</td>
<td>
261
</td>
<td>
278
</td>
<td>
436
</td>
<td>
419
</td>
<td>
-
</td>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/KR862351.1>KR862351.1</a>
Simian immunodeficiency virus isolate VSAA2001, complete genome
</td>
<td>
61
</td>
<td>
95.238
</td>
<td>
21
</td>
<td>
1
</td>
<td>
391
</td>
<td>
411
</td>
<td>
8635
</td>
<td>
8615
</td>
<td>
-
</td>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/EU875177.1>EU875177.1</a>
HIV-1 clone ML1592n from Kenya nonfunctional vpu protein (vpu) gene,
complete sequence; and nonfunctional envelope glycoprotein (env) gene,
partial sequence
</td>
<td>
5.0
</td>
<td>
87.500
</td>
<td>
32
</td>
<td>
4
</td>
<td>
496
</td>
<td>
526
</td>
<td>
270
</td>
<td>
239
</td>
<td>
-
</td>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/JF267434.1>JF267434.1</a>
HIV-2 isolate 05HANCV37 from Cape Verde envelope glycoprotein (env)
gene, partial cds
</td>
<td>
61
</td>
<td>
100.000
</td>
<td>
18
</td>
<td>
0
</td>
<td>
537
</td>
<td>
554
</td>
<td>
284
</td>
<td>
301
</td>
<td>
+
</td>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/KJ131112.1>KJ131112.1</a>
HIV-2 isolate 106CP_RT from Cote d'Ivoire reverse transcriptase gene,
partial cds
</td>
<td>
214
</td>
<td>
95.000
</td>
<td>
20
</td>
<td>
1
</td>
<td>
648
</td>
<td>
667
</td>
<td>
85
</td>
<td>
66
</td>
<td>
-
</td>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/AF003044.1>AF003044.1</a>
Simian immunodeficiency virus isolate P18 patient P1, gp120 (env) gene,
partial cds
</td>
<td>
214
</td>
<td>
95.000
</td>
<td>
20
</td>
<td>
1
</td>
<td>
702
</td>
<td>
721
</td>
<td>
845
</td>
<td>
864
</td>
<td>
+
</td>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/JN091691.1>JN091691.1</a>
Simian immunodeficiency virus isolate TAN5 from Tanzania, complete
genome
</td>
<td>
214
</td>
<td>
90.909
</td>
<td>
22
</td>
<td>
2
</td>
<td>
654
</td>
<td>
675
</td>
<td>
2801
</td>
<td>
2780
</td>
<td>
-
</td>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/HQ644953.1>HQ644953.1</a>
HIV-1 isolate 19828.PPH11 from Netherlands envelope glycoprotein (env)
gene, partial cds
</td>
<td>
5.0
</td>
<td>
89.286
</td>
<td>
28
</td>
<td>
3
</td>
<td>
710
</td>
<td>
737
</td>
<td>
967
</td>
<td>
994
</td>
<td>
+
</td>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/L07625.1>L07625.1</a> Human
immunodeficiency virus type 2 complete genome from strain HIV-2UC1
</td>
<td>
746
</td>
<td>
84.615
</td>
<td>
26
</td>
<td>
4
</td>
<td>
757
</td>
<td>
782
</td>
<td>
6701
</td>
<td>
6726
</td>
<td>
+
</td>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/JF811228.1>JF811228.1</a>
HIV-2 isolate H2A62_111808_CINT_WBC_25 from Senegal pol gene, partial
sequence
</td>
<td>
746
</td>
<td>
100.000
</td>
<td>
16
</td>
<td>
0
</td>
<td>
804
</td>
<td>
819
</td>
<td>
803
</td>
<td>
788
</td>
<td>
-
</td>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/KC187066.1>KC187066.1</a>
HIV-1 isolate 4045_Plasma_Visit1_amplicon9 from Malawi envelope
glycoprotein (env) gene, complete cds
</td>
<td>
2604
</td>
<td>
90.000
</td>
<td>
20
</td>
<td>
2
</td>
<td>
849
</td>
<td>
868
</td>
<td>
431
</td>
<td>
412
</td>
<td>
-
</td>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/GU481454.1>GU481454.1</a>
HIV-1 isolate 07.RU.SP-R497.VI.F5 from Russia envelope glycoprotein
(env) gene, complete cds
</td>
<td>
5.0
</td>
<td>
82.051
</td>
<td>
39
</td>
<td>
5
</td>
<td>
867
</td>
<td>
905
</td>
<td>
1064
</td>
<td>
1029
</td>
<td>
-
</td>
</tr>
<tr>
<td>
<a href=https://www.ncbi.nlm.nih.gov/nuccore/LM999945.1>LM999945.1</a>
Simian immunodeficiency virus partial pol gene for Pol, isolate
SIVagmTAN-CM545-pol
</td>
<td>
214
</td>
<td>
83.333
</td>
<td>
30
</td>
<td>
5
</td>
<td>
894
</td>
<td>
923
</td>
<td>
1098
</td>
<td>
1069
</td>
<td>
-
</td>
</tr>
</table>
<h3>
<a href=#Version_of_Perezs_graphic_with_E_value_and_strand_added id=Version_of_Perezs_graphic_with_E_value_and_strand_added>Version
of Perez's graphic with E-value and strand added</a>
</h3>
<p>Perez and other people have posted the image below on Twitter.
[<a href=https://x.com/JCPEREZCODEX/status/1920908991925776671>https://x.com/JCPEREZCODEX/status/1920908991925776671</a>]
But the image misleadingly doesn't show the expect value or strand of
the matches, so I added the green and red pieces of text to the image
which show the expect value and strand. The matches with the right
strand and frame are shown in green, and matches with the wrong strand
or frame are shown in red. There's only two green matches, but both of
them have a very high E-value.</p>
<p>I took the E-values from the table in the previous section, which
shows the E-values when I searched for regions A and B of Wuhan-Hu-1
against sequences in the nt database with the species HIV-1, HIV-2, or
SIV. Regions A and B account for about one fiftieth part of the length
of the whole genome of SARS-CoV-2, so if I would've searched for the
whole genome instead of only regions A and B, then the E-values would've
been about 50 times bigger.</p>
<div>
<img src=i/pradhan-perez-fixed.webp width=600>
</div>
<p>In Perez's graphic it's also misleading that he drew the arrows of
all 8 matches from left to right, even though 5 out of 8 matches were on
the minus strand so the arrows should've been drawn from right to
left.</p>
<h3>
<a href=#Additional_matches_to_HIV_and_SIV_outside_regions_A_and_B id=Additional_matches_to_HIV_and_SIV_outside_regions_A_and_B>Additional
matches to HIV and SIV outside regions A and B</a>
</h3>
<p>In Table 2 of the paper by Perez and Montagnier, there's a list of 4
additional matches to HIV and SIV that are not located in the regions A
or B:</p>
<div>
<img src=i/pradhan-perez-2.webp width=600>
</div>
<p>However 3 out of 4 matches in the table are on the minus strand:</p>
<pre>$ curl &#39;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&amp;rettype=fasta&amp;id=LR757998&#39;&gt;perezref.fa
$ printf %s\\n &#39;KM378564.1 8752 8770&#39; &#39;EU184986.1 14341 14378&#39; &#39;AY516986.1 20374 20401&#39; &#39;HQ217329.1 20401 20430&#39;&gt;perez2
$ cut -d\  -f1 perez2|curl -sd &quot;id=$(paste -sd,)&quot; &#39;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&amp;rettype=fasta&#39;&gt;perez2.fa
$ makeblastdb -dbtype nucl -in perez2.fa
[...]
$ fmt=&#39;sseqid evalue nident length pident sstrand sstart send qstart qend&#39;
$ &lt;perezref.fa blastn -db perez2.fa -outfmt &quot;6 $fmt&quot; -task blastn|awk &#39;NR==FNR{a[$1]=$2&quot; &quot;$3;pos[$1]=NR;next}{print pos[$1],$0,a[$1]}&#39; perez2 -|awk &#39;{print$0,$8==$10&amp;&amp;$9==$11?&quot;true&quot;:&quot;false&quot;}&#39;|sort -rk12|awk &#39;!a[$1]++&#39;|sort -n|cut -d\  -f2-|(echo $fmt perezstart perezend matchesperez;cat)|column -t
sseqid      evalue    nident  length  pident  sstrand  qstart  qend   perezstart  perezend  matchesperez
KM378564.1  3.8       22      27      81.481  minus    4696    4722   8752        8770      false
EU184986.1  4.95e-05  33      38      86.842  minus    14341   14378  14341       14378     true
AY516986.1  4.95e-05  26      28      92.857  plus     20374   20401  20374       20401     true
HQ217329.1  4.95e-05  28      30      93.333  minus    20401   20430  20401       20430     true</pre>
<p>The first row of Perez's table is shown to match positions 8751 to
8770 of his reference genome (i.e. positions 8752 to 8770 using
one-based indexing and an inclusive end of the range). But that region
of his reference genome does not actually match the HIV sequence on the
first row even though the region 8736 to 8755 shown in my output above
does, so Perez may have accidentally entered the wrong starting and
ending positions on the first row.</p>
<p>Only one out of the 4 matches in Table 2 is on the plus strand, but
it doesn't even match HIV, because it matches a part of a human genome
that is included in an HIV integration site sequence, where the first 69
bases come from an HIV long terminal repeat but the rest of the sequence
comes from a human genome:
[<a href=https://www.ncbi.nlm.nih.gov/nuccore/AY516986>https://www.ncbi.nlm.nih.gov/nuccore/AY516986</a>]</p>
<div>
<img src=i/pradhan-perez-3.jpg width=500>
</div>
<p>Perez wrote that in the match to the integration site sequence <span
class="quote">"addresses 20373 to 20401 comes from an HIV1 Integrase
from a USA virus from 2004"</span>. However he may have confused the
term <span class="quote">"integration site"</span> with the HIV
integrase enzyme which is contained within the pol gene. The integrase
is located at positions 4236 to 5086 in HXB2, but the first 69 bases of
the integration site sequence are identical to bases 112 to 180 of HXB2
(which is contained within the 5' long terminal repeat):</p>
<pre>$ emu&lt;&lt;&lt;NC_001802&gt;hiv1.fa
$ emu&lt;&lt;&lt;AY516986|seqkit subseq -r1:69|seqkit locate -if- hiv1.fa|cut -f4-7|column -t
strand  start  end  matched
+       112    180  gtctgttgtgtgactctggtaactagagatccctcagacccttttagtcagtgtggaaaatctctagca</pre>
<p>Another big fail by Perez was that he thought that one of his BLAST
matches was located at the start of the spike protein in RaTG13 but that
it spanned both ORF1ab and the spike protein in SARS-CoV-2. But that's
because his nonstandard reference genome was missing the first 25 bases
of Wuhan-Hu-1, so he should've subtracted 25 from Wuhan-Hu-1 coordinates
when he identified the start of the spike protein:</p>
<div>
<img src=i/pradhan-perez-4.webp width=600>
</div>
<h3>
<a href=#Matches_to_the_parasite_Plasmodium_yoelii id=Matches_to_the_parasite_Plasmodium_yoelii>Matches
to the parasite Plasmodium yoelii</a>
</h3>
<p>On page 36 of the paper Perez presented this match to the rodent
parasite <em>Plasmodium yoelii</em>:</p>
<div>
<img src=i/pradhan-perez-5.jpg width=600>
</div>
<p>However the matching sequence is in the frame 2 in SARS-CoV-2 but
frame 1 in <em>Plasmodium yoelii</em>, so Perez's amino acid translation
of the match in SARS-CoV-2 is wrong:</p>
<pre>$ url=&#39;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&amp;rettype=fasta_cds_na&#39;
$ curl &quot;$url&amp;id=MN908947&quot;&gt;sars2.na
$ curl &quot;$url&amp;id=LM993664&quot;&gt;yoelii10.na
$ seqkit locate -pCACAAGTCAAACAAATTTACAAAACACCACCAATTAAAGAT sars2.na|cut -f4-6|column -t
strand  start  end
+       2348   2388 # (2348-1)%3+1 is frame 2
$ seqkit locate -pCACAAATCAAACAAATTTACAAAACACAAACCAAAAAAAAAT yoelii10.na|cut -f4-6|column -t
strand  start  end
+       106    147 # (106-1)%3+1 is frame 1
+       106    147</pre>
<p>Next Perez presented a second match to the same rodent parasite which
they initially said appeared to be located downstream in the same
protein as the first match, even though actually the match was located
in a different chromosome and a different protein, and in fact Perez
later corrected his own initial impression and wrote that <span
class="quote">"it is therefore clear that this second region of Yoelii
does not coincide with the extension downstream of the sequence <span
class="quote">'Fam a'</span>"</span>. However for some reason Perez
still decided to concatenate the two unrelated matches together. The
second match to <em>Plasmodium yoelii</em> has two gaps near the start
in SARS-CoV-2 which means that the segments before and after the gaps
are in different frames:</p>
<div>
<img src=i/pradhan-perez-6.jpg width=600>
</div>
<p>The screenshot above shows that the match to the whole chromosome in
<em>Plasmodium yoelii</em> is on the minus strand, but the match to the
gene in <em>Plasmodium yoelii</em> is still on the plus strand because
the gene is on the reverse strand. And the part of the match after the
gaps happens to be in the +2 frame in both <em>Plasmodium yoelii</em>
and SARS-CoV-2. But the match is still not impressive at all because it
has only 36 out of 46 identical bases so the percentage identity is only
about 78%:</p>
<pre>$ url=&#39;https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&amp;rettype=fasta_cds_na&#39;
$ curl &quot;$url&amp;id=MN908947&quot;&gt;sars2.na
$ curl &quot;$url&amp;id=LK934642&quot;&gt;yoelii14.na # protein coding sequences within chromosome 14
$ seqkit locate -pCAAAACACCACCAATTAAAGATTTTGGTGGTTTTAATTTTTCACAA sars2.na|cut -f4-6|column -t
strand  start  end
+       2367   2412 # (2367-1)%3+1 is frame 3
$ seqkit locate -pCAAAATAAAACCAATTATATATTTTGATCATATTAATTTTTCAAAA yoelii14.na|cut -f4-6|column -t
strand  start  end
+       6903   6948 # (6903-1)%3+1 is frame 3</pre>
<h3>
<a href=#Histogram_of_matches_per_block_in_LANLs_HIV_subtype_reference id=Histogram_of_matches_per_block_in_LANLs_HIV_subtype_reference>Histogram
of matches per block in LANL's HIV subtype reference</a>
</h3>
<p>In the plot below I used web BLAST to search for Wuhan-Hu-1 within
the nr/nt database, where I set species to HIV, task to blastn, expect
threshold to 1000, and maximum results to 1000. I then kept scrolling
the alignment view for a couple of minutes to fetch all results, I saved
the source code of the web page, and I made a list of the starting and
ending positions of each match, and I counted how many positions were
covered by at least one match. The E values of the matches ranged
between 5.2 and 776. When I divided Wuhan-Hu-1 to 500-base blocks, the
three blocks with the highest covered percentage were the blocks that
started at positions 21001, 11001, and 21501, where two of the blocks
roughly coincide to Perez's regions A and B which cover positions 21098
to 22027. However there was also a block in ORF3a which had almost as
high coverage percentage as the block at the end of ORF1b that covered
the first half of regions A and B. And when I included only matches on
the plus strand, the block at the end of ORF1b got 0% coverage, and the
block at the start of the spike protein got 14% coverage which was about
as high as three random blocks in ORF1ab:</p>
<div>
<img src=i/pradhan-perez-8.webp width=1000>
</div>
<p>Even though I included the top 1000 BLAST results in the plot above,
there were many duplicate matches where different sequences of HIV
matched the same segment, so there were only 82 unique starting
positions among the matches. But next in order to search within a
diverse set of HIV subtypes with only a few results by subtype, I did a
BLAST search within the HIV subtype reference published by the Los
Alamos National Laboratory:
<a href=https://www.hiv.lanl.gov/content/sequence/NEWALIGN/align.html.><a
href="https://www.hiv.lanl.gov/content/sequence/NEWALIGN/align.html"
class="uri">https://www.hiv.lanl.gov/content/sequence/NEWALIGN/align.html</a>.</a>
It contains 555 sequences that represent 165 different subtypes of
HIV-1, but it also contains a few HIV-1 like SIV sequences from chimps
and gorillas. I got zero results when I searched for Wuhan-Hu-1 against
the LANL subtype reference with the default options, but my number of
results increased to 7 when I added the option <code>-task blastn</code>
and to 609 when I also added the option <code>-evalue 1000</code>. There
were 245 unique starting positions among the results, so now the matches
were distributed more evenly because there weren't as many duplicate
matches for the same region. And now even in the top panel which
includes both strands, the two bars which roughly correspond to Perez's
regions A and B only had the 4th and 11th highest percentage of covered
bases:</p>
<div>
<img src=i/pradhan-perez-9.webp width=1000>
</div>
<h3>
<a href=#Tweets_by_Jean_Claude_Perez id=Tweets_by_Jean_Claude_Perez>Tweets
by Jean-Claude Perez</a>
</h3>
<p>Perez might have written his paper as a form of deliberate
disinformation, because he has also said that vaccinated people are
harboring nanopathogens that can be activated by 5G, that vaccinated
people have MAC addresses, that vaccines contain graphene oxide, and
that the pandemic was fake:
[<a href=https://x.com/JCPEREZCODEX/status/1875105196294201653>https://x.com/JCPEREZCODEX/status/1875105196294201653</a>,
<a href=https://x.com/JCPEREZCODEX/status/1768971965178548510>https://x.com/JCPEREZCODEX/status/1768971965178548510</a>,
<a href=https://x.com/JCPEREZCODEX/status/1686762168606167040>https://x.com/JCPEREZCODEX/status/1686762168606167040</a>,
<a href=https://x.com/JCPEREZCODEX/status/1655898773774647303>https://x.com/JCPEREZCODEX/status/1655898773774647303</a>,
<a href=https://x.com/JCPEREZCODEX/status/1671902292402995200>https://x.com/JCPEREZCODEX/status/1671902292402995200</a>]</p>
<div>
<img src=i/pradhan-perez-tweets.webp width=550>
</div>
<script>
document.querySelectorAll('pre').forEach(e=>e.innerHTML=e.innerHTML.replace(/((^| )# [^\n]*|^#$)/mg,"<span class=comment>$1</span>"))

const walker=document.createTreeWalker(document.body,NodeFilter.SHOW_TEXT,null,false)
const nodes=[]
while(walker.nextNode())nodes.push(walker.currentNode)
nodes.forEach(node=>{
  const text=node.textContent
  if(text.split(/\s+/).some(piece=>piece.length>=20)||/\w{60}/.test(text)){
    const frag=document.createDocumentFragment()
    text.split(/([A-Za-z0-9]{60}|[\041-\057\072-\100\133-\140\173-\176])/g).forEach(part=>{
      if(part){
        frag.appendChild(document.createTextNode(part))
        if(/^[A-Za-z0-9]{60}$/.test(part)||/^[\041-\057\072-\100\133-\140\173-\176]$/.test(part))
          frag.appendChild(document.createElement('wbr'))
      }
    })
    node.parentNode.replaceChild(frag,node)
  }
})

document.querySelectorAll('.spoiler button').forEach(x=>{x.addEventListener('click',function(){
  let block=this.nextElementSibling
  if(window.getComputedStyle(block).display=='none'){block.style.display='block';this.textContent='Hide code'}
  else{block.style.display='none';this.textContent='Show code'}
})})
</script>
